\chapter*{Abstract}

While somatic evolution is widely accepted as fundamental to cancer development, significant gaps in understanding persist. These gaps concern the role of cellular interactions within the \acf{TME} and the impact of spatial constraints on cancer evolution. Emerging spatial omics technologies offer the potential to address these gaps, although their application in spatial genomics remains limited. This is particularly crucial because genetic alterations not only drive cancer evolution but also serve as an archive of its history.

To address this research gap, this thesis aims to develop computational methods for analysing spatial genomics data, specifically on data generated by \acf{BaSISS}, a method that enables high-resolution mapping of diverse somatic mutations across large tumour tissue sections. Bayesian algorithms designed in the study generate quantitative clonal maps, effectively tracing cancer evolution in tissue space while accommodating multiple forms of biological and technical variability. Despite inherent assumptions, the algorithm demonstrates robustness and quantitative accuracy.

Complementary data types contribute to the findings by applying the Bayesian model to two multifocal breast cancers at different stages of progression. Integration of histology, \acf{IHC}, and targeted \textit{in situ} gene expression allows for the phenotypic characterisation of clones in distinct microanatomical niches. Subsequent analyses across various stages of breast cancer, including \acl{CIS}, invasive cancer, and lymph node metastasis reveal clone-specific variations in proliferation, morphology, stroma, hypoxia, and immune microenvironments. In one instance involving \acl{DCIS}, polyclonal neoplastic expansions manifest on a macroscopic scale but remain segregated within microanatomical structures. 

In summary, this thesis establishes a robust computational framework for extracting clonal architecture from spatial genomics data. It provides a proof-of-concept that such maps, when integrated with tissue morphology and spatial phenotype data, can offer vital insights into the mechanisms driving both cancer evolution and tissue ecology.



