\chapter{Spatial Evolution and Ecology of Breast Cancer: From Ducts to Lymph Nodes}
\label{sec:chapter-basiss-applications}

\section*{Contributions}
This chapter is largely based on the main analysis from:
\\~\\
 \fullauthcite{Lomakin2022-ks}. 
\\~\\
The work I present here is primarily my own contribution. I focused on developing and implementing the core mathematical model for the \acs{BaSISS} data under the supervision of \ac{moritz}, with valuable inputs from \ac{artem1} and \ac{vitalii}. I analysed and interpreted the data, and drafted the original article and figures together with and under the supervision of \ac{moritz} and \ac{lucy}. You can find all the code on \href{https://github.com/gerstung-lab/BaSISS}{Github}.

Another first author of this paper, \ac{jessica}, in collaboration with \ac{peter}, \ac{mats}, and \ac{lucy}, designed the initial study of \acs{BaSISS}. \ac{jessica}, \ac{mats}, and \ac{carina} conducted the experiments and provided the raw \acs{BaSISS} data. \ac{junsung}, \ac{vasyl}, \ac{tong}, and \ac{milana} preprocessed and decoded this data. \ac{lucy} performed the \acs{LCM} cuts, and \ac{stefan} conducted the \acs{WGS} subclonality analyses. I had access to the decoded \acs{BaSISS} and \acs{LCM}-\acs{WGS} data.

The Introductory \cref{sec:applications-breast-intro} is original. I borrowed the rest of the sections from the original manuscript with only minor stylistic and structural alterations. All the main and margin figures in the introduction are original. Figures \cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2,fig:applications-maps-DCIS,fig:applications-LN-genome,fig:applications-maps-LN} and the margin figure in \cref{sec:applications-DICS-growth} are borrowed from the original paper with minor stylistic and compositional adjustments.

\section{A Primer on Breast Cancer}
\label{sec:applications-breast-intro}

\explain{}{\marginfig{side-plot-cancer-cases-2020.pdf}
Breast cancer is the most prevalent type of cancer world wide, with approximately 2.2 million cases registered in 2020.}

In 2020, breast cancer surpassed lung cancer to become the most diagnosed cancer type in the world \parencite{Sung2021-xv}. It accounted for approximately 11.7\% of all diagnosed cancer cases. This trend occurred irrespective of a country's development index, predominantly targeting the female population. Within this demographic, breast cancer constituted nearly a quarter of all diagnoses, and 15.5\% of all cancer-related deaths. An estimated 2.2 million cases were registered in 2020, and this number is expected to grow, partly due to obesity, the ageing population and the fact that females constitute the majority of the older population. Despite advances in treatment and early detection, the death rate is not declining rapidly enough to counteract the consistent 0.5\% increase in incidence rate observed over the past two decades. \parencite{Cronin2022-mc}. This motivates the need for a better understanding of the disease's biology and evolution.

\subsection{Normal breast tissue structure and development}
\label{sec:applications-normal-breast-development}
Before exploring the biology of breast cancer, it is crucial to understand the biology of the affected organ - the mammary gland. Development of the mammary gland begins in the embryo through interactions between epithelial and mesenchymal cells. At first, mammary \explain{placodes}{thickened areas of ectodermal tissue in embryonic development that give rise to specialised structures.} form and subsequently invaginate to create mammary buds. After that, the ectoderm proliferates and branches into the fat pad to form a rudimentary ductal tree. Throughout the entire process, paracrine signalling between the ectoderm and surrounding mesenchyme is crucial, as epithelial cells induce specialised mesenchymal cells formation, that in turn instruct the ectoderm to commit to mammary identity \parencite{Macias2012-su,Spina2021-ej}.

\figuretextwidth{applications-breast-structure.pdf}{applications-breast-structure}
    {Mammary gland structure and breast cancer progression}
    {Breast cancer originates from the bilayer epithelial cells lining the branched system of the mammary gland, often specifically within the milk-producing \acf{TDLU}. Initially, these cancer cells divide and fill the lumen of the ducts and lobules, a phase termed carcinoma in situ. Subsequently, these cells may penetrate the basement membrane, invading adjacent tissues in a stage called invasive carcinoma. Eventually, the cancer cells gain the potential to enter the bloodstream and metastasise to distant body sites.}

The mature ductal epithelium consists of two distinct cell types: luminal cells, which express \protein{KRT7}, \protein{KRT8}, and \protein{KRT18}, and basal cells, marked by \protein{KRT5} and \protein{KRT14} expression. Luminal cells constitute the inner layer and play a critical role in milk production and duct formation. In contrast, basal cells form the outer layer and adhere to the basement membrane. They execute the contractile function essential for milk release. During embryogenesis, mammary epithelial cells undergo lineage restriction. Early bipotent progenitors express both basal and luminal markers \parencite{Stingl2001-cb,Prater2014-qd,Rios2014-jj}. As development progresses, unipotent basal and luminal progenitors emerge \parencite{Van_Keymeulen2011-um,Rios2014-jj,Tao2014-ol}. Regulators such as the \gene{TP63} gene are suspected to stabilise basal cell identity, while the \gene{NOTCH1} gene probably solidifies luminal identity, at least in mice models  \parencite{Spina2021-ej}. This hierarchical pattern, commonly observed in various tissues, tends to mitigate the risk of cancer development as discussed in \cref{sec:intro-tissue-organisation}.

During puberty, sex hormones such as oestrogen drive ductal elongation, branching in the mammary gland, and the formation of milk-producing alveolar expansions grouped in \acf{TDLU} \pcref{fig:applications-breast-structure}. Although only a subset of luminal cells express oestrogen receptor (ER+), these cells effectively transmit signals across the entire epithelium. This signalling occurs in a paracrine manner and involves interactions with breast stroma.
In murine models, oestrogen stimulates the release of epidermal growth factors (\protein{EGF}s). These \protein{EGF}s bind to their corresponding receptors on stromal cells, initiating the expression of fibroblast growth factors (\protein{FGF}s). \protein{FGF}s, in turn, promote the proliferation of luminal cells. Additionally, stromal cells secrete insulin-like growth factor 1 (\protein{IGF1}), further stimulating mammary gland morphogenesis \parencite{Macias2012-su}.

Breast development extends beyond puberty, undergoing substantial remodelling and increased branching during pregnancy. This transformation, particularly in epithelial and alveolar structures, is triggered by hormones such as progesterone and prolactin, which prepare the gland for lactation. Following weaning, many glandular structures involute, leading to the apoptosis of secretory cells \parencite{Macias2012-su}.

These observations imply the existence of persistent progenitor cells within the tissue or a general capacity for cell proliferation. Such mechanisms are essential, as a significant portion of mammary tissue must regenerate to accommodate subsequent pregnancies.

\explain{}{\marginfig{side-plot-breast-hierarchy.pdf} Continuous lineage hierarchy from \textcite{Nguyen2018-jl}.}

Research by \textcite{Prater2014-qd} indicates that up to 65\% of basal cells can form ductal-lobular structures \emph{in vivo}. Single-cell transcriptional trajectory analysis reveals that a substantial fraction of \protein{KRT14}$^+$ basal cells act as a transcriptional pivot point between luminal and specialised myoepithelial cells \parencite{Nguyen2018-jl}. The equipotency of basal cells in the mammary gland is noteworthy, especially given its implication in elevated cancer risk, as discussed in \cref{sec:intro-tissue-organisation}.

One plausible explanation for this characteristic could be the evolutionary necessity for rapid mammary gland remodelling with each successive pregnancy. As mammals evolved, the imperative for tissue renewal may have outweighed the need for cancer protection. This evolutionary trade-off might now manifest as an elevated risk of breast cancer as mentioned in \cref{sec:applications-breast-intro}.

\subsection{Breast cancer staging and survival}

Most breast cancers are adenocarcinomas originating from the epithelial cells in the breast's ductal system, frequently from \acp{TDLU}.\footnote{Surprisingly, many scientific papers and even the Dana-Farber Cancer Institute's \href{https://www.dana-farber.org/health-library/articles/what-is-lobular-breast-cancer-}{website} identify the \ac{TDLU} as the  \textbf{sole site of origin for all breast cancers}. This assertion appears unlikely, given that any dividing cell has the potential to become cancerous. The study by \textcite{Tabar2014-ea} suggest that ~ 25\% of cases actually originate in the duct and show no link to \ac{TDLU}.} In the early stages of cancer development, genetically altered precancerous cells grow uncontrollably, a process known as hyperplasia. These cells may also display atypical features. Such aberrant growth often fills the lumens of ducts and lobules, a condition described as \acf{CIS}. While \ac{CIS} can remain confined to the ducts throughout an individual's lifetime, in some cases these cells breach the basement membrane, invading the surrounding stroma. This invasive behaviour marks the transition to invasive carcinoma. At this stage, some cancers can develop the capability to enter the bloodstream or lymphatic system, and form metastatic deposits outside of the breast. Once distant metastases are established, a breast cancer is invariably incurable \pcref{fig:applications-breast-structure}.

\subsubsection{Anatomical staging}

In managing a disease with such diverse clinical outcomes, clinicians use classification systems to select the most appropriate treatment strategies. The vast majority of primary breast cancers will be completely excised by surgery. The threat for most breast cancer patients is therefore an invisible one – the risk that cancer cells have already spread to distant sites and will reawaken to cause metastasis at a later date. The major challenges for an oncologist are therefore to establish a) whether microscopic metastatic disease is likely to be present and b) which treatment strategies are most likely to eliminate or control it as opposed to simply causing unnecessary side-effects. The most widely used and well validated strategies for achieving this aim draw on both pathological features and anatomical staging of the cancer.

The TNM staging system is used worldwide and helps to classify breast cancers based on the anatomical extent of the disease: the size and reach of the primary tumour (T), lymph node involvement (N), and the occurrence of distant organ metastases (M). This anatomical staging is defined at first diagnosis `clinically' through physical examination and radiological imaging. Following primary surgery, macro and microscopic analysis of the excised tumour permits detailed `pathological' staging.  In general, larger cancers and/or those with extensive spread to the regional lymph nodes have a worse prognosis and this reflects the higher incidence of micrometastatic disease at diagnosis. 

\subsubsection{Histological and molecular classification}

In global clinical practice, the TNM classification system serves as a cornerstone. However, the system has limitations; specifically, it inadequately captures the biological heterogeneity of breast cancers across different patients. Therefore, additional methods are employed to complement the TNM system to facilitate a personalised approach to cancer management and prognostication.

Histology provides valuable insights into the level of dedifferentiation and proliferative activity of breast cancer, similar to other cancer types, as discussed in Section \cref{sec:intro-tissue-organisation}. Tumour grade, ranging from 1 to 3, encapsulates this information. Grade 3 indicates the most aggressive tumours, often comprising less differentiated cells, while grade 1 represents the least aggressive forms \parencite{Greenough1925-wg, Elston1991-md}. Features like tumour grade and the presence of lymphovascular invasion help to stratify those who are most likely to benefit from chemotherapy. 

The majority of breast cancer cases are histologically classified as either No Special Type (NST) ($\sim$80\%) or `lobular' ($\sim$10\%).

In molecular diagnostics for breast cancer, the standard protocol includes \ac{IHC} classification for three key markers: oestrogen receptor (\protein{ER}), progesterone receptor (\protein{PgR}), and human epidermal growth factor receptor 2 (\protein{HER2}).

Treatments aimed at lowering oestrogen exposure are effective for all \textbf{Hormone Receptor (HR)} positive breast cancers, which express either \protein{ER} or \protein{PgR}. These treatments, such as Tamoxifen, aromatase inhibitors, and ovarian suppression, have been proven to improve overall survival, disease free survival and reduce the both local and distant metastasis \parencite{Early_Breast_Cancer_Trialists_Collaborative_Group_EBCTCG2005-mk}.

\textbf{HER2} is another important marker, which is a receptor tyrosine kinase and a common breast cancer oncogene \pcref{fig:applications-driver-pathways}. The high level of expression of \protein{HER2} is associated with a higher levels of dedifferentiation and proliferation. Development of HER2-targeted therapies (e.g. Trastuzumab) has significantly improved the survival of patients with HER2-positive tumours \parencite{Slamon2001-qs}. 

Based on the aforementioned molecular markers, cancers are categorised into three distinct clinical subtypes : HR+/HER2- (luminal); HER2+ (HER2) and HR-/HER2- (\acs{TNBC}). The HR+/HER2- subtype, long recognised as biologically heterogeneous, presents a clinical dilemma in balancing treatment benefit while avoiding unnecessary toxicity. While many HR+/HER2- cancers are cured with surgery and endocrine therapy, others follow a highly aggressive course that may require a more intensive chemotherapeutic strategy.

\explain{}{\marginfig{side-plot-ihc2pam-sankey.pdf} Although gene expression and protein-based systems contain similar clusters, the alignment between them is not precise. Data from \textcite{Kim2019-mz}.}

In routine clinical practice, one common approach for stratifying HR+/HER2- cases involves evaluating their expression of the proliferative marker Ki67, measured via \ac{IHC}. These classifications roughly correspond to the Luminal A (low Ki67) and Luminal B (high Ki67) intrinsic subtypes as defined by modern RNA-based assays \pcref{tab:bc-subtypes}.

{
\footnotesize
\begin{longtable}{llp{7.6cm}}
    \tabcap{bc-subtypes}{Summary of breast cancer bioloigcal subtypes}
           {Overview of the general characteristics of common breast cancer subtypes defined my biomarkers.} \\
    \toprule
    Subtype      &  \ac{IHC} Status   &   General Characteristics \\
    \midrule
    Luminal A    &  HR+ HER2- Ki67$\downarrow$ &   Generally lower-grade cancers, have a better prognosis compared to other subtypes. \\
    Luminal B    &  HR+ HER2- Ki67$\uparrow$&   Often higher-grade than Luminal A, generally more aggressive and less responsive to hormone therapy. \\
    HER2         &  HR+/- HER2+ &  Generally more aggressive and treated with targeted therapies. \\
    \acs{TNBC}   &  HR- HER2- &   Generally higher grade, more aggressive, and poor prognosis. \\
    \bottomrule
\end{longtable}
}

The advent of high-throughput technologies, including microarrays and next-generation sequencing, has facilitated the creation of classification systems like PAM50 \parencite{Parker2009-um}, MammaPrint \parencite{Van_t_Veer2002-jd}, Oncotype Dx \parencite{Paik2004-aq}, and IntClust \parencite{Curtis2012-hu}. These systems can offer a more granular view of breast cancer's biological heterogeneity. While largely confirming existing subtype clusters, they further divide them into subclasses with distinct biological characteristics and survival outcomes. Despite their analytical potential, these advanced systems have not been universally adopted in clinical practice for various reasons including economic cost and the scarcity of validation data from prospective clinical trials at the time of writing. 

Future improvements likely require a deeper understanding of the disease, including an appreciation for both intra- and inter-tumour heterogeneity. A complexity which arises from the interplay between the evolving genomic landscape of cancer cells and the tissue microenvironment that influences how the tumour develops \pcref{sec:chapter-introduction}.

\subsection{Genomic landscape} 

Pioneering studies by \textcite{Stehelin1976-jr, Nowell1960-eh}, established a critical link between genomic aberrations and cancer, thereby integrating evolutionary theory with our understanding of cancer as a disease \parencite{Cairns1975-oz,Nowell1976-sm}. In the following decades, many cancer genes such as \gene{TP53} \parencite{Lane1979-hx}, \gene{MYC} \parencite{Kohl1984-ri}, \gene{BRCA1/2}\explain{}{\marginfig{side-plot-brca-road.pdf} Cycling path in Cambridge\footnotemark with the entire sequence of \gene{BRCA2}.} (on 13q12-13) \parencite{Wooster1994-xa, Hall1990-mg} \parencite{Wooster1994-xa, Hall1990-mg} , and \gene{PIK3CA} \parencite{Chang1997-sa} have been identified. Post 2000, high throughput genomic sequencing enabled large-scale, systematic genomics analyses of thousands of tumours. These studies have revealed complex patterns of genetic alterations.

\footnotetext{My processing speed is approximately 60 bp/s, as I can slide along 10,257 base pairs of \gene{BRCA2} sequence in 2 minutes and 47 seconds. For comparison, \emph{Taq} polymerase operates at a rate of 150 bp/s, while the high-fidelity \emph{Pfu} polymerase works at a considerably slower pace of 15 bp/s. I deeply thank Lukas Weilguny for consistently providing slipstream support along the way.}

\figuretextwidth{applications-driver-pathways.pdf}{applications-driver-pathways}{Main driver genes and signalling pathways in breast cancer}{Comprehensive early genomics studies have identified key driver genes associated with breast cancer. These genes primarily target four main pathways: Receptor Tyrosine Kinases, PI(3)K/Akt, p53 signalling, and cell cycle checkpoint regulators. Mutations in genes contolling these pathways results in: cell survival through inhibition of apoptosis, proliferation, dedifferentiation and faults in DNA reparation. On the figure red indicates gain-of-function mutations or amplifications; blue represents loss-of-function mutations or deletions.}

\subsubsection*{Breast cancer driver genes}
Several genomics studies of breast cancer identified thousands of likely driver mutations, spanning hundreds of cancer-related genes and several non-coding regions \parencite{Shah2009-xz, Cancer_Genome_Atlas_Network2012-gx,Shah2012-xz, Nik-Zainal2016-ek,Banerji2012-as,Ciriello2015-ey,Stephens2012-vj,Pereira2016-ov, Curtis2012-hu}. The most commonly mutated breast cancer genes and gene regions as found in \textcite{Cancer_Genome_Atlas_Network2012-gx} are \gene{TP53} (43\%), \gene{PIK3CA} (32\%), \gene{MYC} (20\%), \gene{CCND1} (16\%), \gene{PTEN} (16\%), \gene{ERBB2} (14\%), Chr8:(\gene{ZNF703}/\gene{FGFR1}) (12\%), \gene{GATA3} (10\%), \gene{RB1} (9\%), \gene{MAP3K1} (8\%). These genes predominantly participate in cell cycle regulation, reparation and growth factor signalling \pcref{fig:applications-driver-pathways}. However, most mutations in cancer-associated genes are infrequent, leading to significant genetic diversity among patients. 

The mechanisms through which these driver genes are generated vary significantly. While some mutations are inherited, most arise somatically \pcref{sec:chapter-introduction}, frequently as a result of both mutagenic factors and imperfections of the repair systems. High-throughput genome sequencing, especially \ac{WGS}, has not only facilitated the identification of driver genes but also elucidated the complex mutational processes responsible for their occurrence.

\subsubsection*{Mutational processes}

While DNA microarrays and exome sequencing have identified broad copy number profiles and mutational landscapes in genes, their coverage remains limited and biased. The advent of massively parallel \ac{WGS} overcame these limitations by providing comprehensive data, allowing for detailed analysis across entire genomes. This advance enabled the decomposition of the mutational landscape into additive sets of mutational signatures, many of which have clear biological implications \parencite{Nik-Zainal2012-vo}. 

Breast cancer mutagenesis involves a variety of mechanisms that generate a spectrum of mutation types. Chromosomal rearrangements frequently occur and can be linked to periods of chromosomal instability \parencite{Curtis2012-hu, Gerstung2020-sg, Nik-Zainal2012-vo}. When homologous recombination-based DNA damage repair is defective, often due to inactivating mutations in \gene{BRCA1} or \gene{BRCA2}, error-prone non-homologous DNA end-joining activity emerges. This leads to unique mutational signatures, including both point mutations and a high number of tandem duplications and deletions, particularly in regions of microhomology.

These rearrangements are often associated with mutational signatures related to \protein{BRCA1/2} deficiency and other DNA double-strand break repair pathways \parencite{Nik-Zainal2012-vo}.

\explain{}{\marginfig{side-plot-apobec.pdf} \protein{APOBEC} is a cytidine deaminase. Under normal conditions it serves as a cellular defence against viral infections by mutating viral ssRNA and ssDNA. In cancer, dysregulated \protein{APOBEC} mutates ssDNA in the nucleus, causing a high frequency of C>T mutations.}

Another common cause of hypermutation in around half of breast cancers is \protein{APOBEC} enzyme mutagenesis \parencite{Nik-Zainal2016-ek, Banerji2012-as, Nik-Zainal2012-vo}. The \protein{APOBEC} signature consists of a high frequency of C>T and C>G mutations in the Tp\underline{C} context, sometimes exhibiting focal hypermutation patterns, known as kataegis, that frequently coincide with structural variant breakpoints. Given the diversity of driver genes and the extended period of genome instability, breast cancer appears to be driven by a cumulative effect of multiple accumulated drivers.

\subsubsection*{Somatic mutations in breast cancer subtypes}

Notably, the frequency of driver mutations and mutation signatures varies among breast tumour types. The \textbf{Luminal class} commonly features somatic mutations that activate the \protein{PI3K-AKT} signalling pathway (\gene{PIK3CA}, \gene{PTEN}) and inactivate the \protein{GATA3} and \protein{JUN} kinase (\gene{MAP3K1}) pathways. Tumour suppressors \gene{TP53} and \gene{RB1} are less frequently implicated in Luminal A cancers but are often inactivated in the more aggressive Luminal B subtype \parencite{Cancer_Genome_Atlas_Network2012-gx}.

The \textbf{\ac{TNBC} class} aligns closely with the basal-like intrinsic expression subtype, with approximately 75\% of \ac{TNBC}s being basal-like \parencite{Shah2012-xz}. This class exhibits a high frequency of \gene{TP53} mutations, especially among basal-like types, as well as a high level of \acp{CNA} \parencite{Shah2012-xz, Cancer_Genome_Atlas_Network2012-gx}. This coincides with frequent inactivation of genes involved in double-strand break reparation such as \gene{BRCA1} and \gene{BRCA2}, more so than in other subtypes. However, the mutation landscape of \ac{TNBC} is highly diverse overall \parencite{Shah2012-xz}.

In the \textbf{HER2+ class}, there is frequent high copy number amplification of \gene{ERBB2}, which accounts for elevated levels of \protein{HER2} protein expression. This amplification often co-occurs with amplification of the nearby gene, \gene{GRB7}. Activating hotspot mutations in the \gene{ERBB2} gene can contribute to the overstimulation of \protein{HER2} signalling. A systematic analysis of this group revealed subclusters that resemble luminal and basal types in both gene expression and mutation distribution. This suggests that rather than constituting an entirely independent intrinsic type, this group represents a composite of the broader breast cancer spectrum, further complicated by \gene{ERBB2} amplification \parencite{Ferrari2016-qj}.

While histological subtypes generally exert limited influence on treatment stratification, certain `special' subtypes exhibit distinct molecular aberrations. These aberrations can account for unique clinical presentations.

\explain{}{\marginfig{side-plot-ductal-lobular.pdf} Ductal and lobular breast cancer are two histological subtypes often differentiated by the presence of a single mutation in the \gene{CDH1} gene.}

Invasive \textbf{lobular} breast cancer, the most prevalent `special' histological subtype, frequently harbours \gene{CDH1} mutations, observed in 45-95\% of cases depending on the methodology and cohort \parencite{Ciriello2015-ey, McCart_Reed2021-aa}. This gene encodes \protein{E-cadherin}, an essential component of adherens junctions that facilitate cell-cell adhesion. The compromised function or loss of \protein{E-cadherin} is considered a driving factor for the lobular phenotype. Recent studies reveal that this phenotype correlates with unique cancer behaviours. Although most lobular cancers belong to the Luminal type, they demonstrate a poorer long-term prognosis compared to other Luminal cancers. Moreover, these cancers metastasise to unconventional sites, including gastrointestinal and gynaecological regions, beyond the common locations of liver, lung, and bone \parencite{McCart_Reed2021-aa}.

Overall, bulk genomic studies indicate that the breast cancer genome exhibits significant inter-tumour diversity. However, while some evolutionary patterns emerge across patients, the presence of mixed phenotypes and observed subclonal diversity within a single tumour complicates our understanding of tumour evolution \parencite{Ciriello2015-ey,Pereira2016-ov,Shah2012-xz}. Initial bulk studies on large cohorts yielded basic insights into intra-tumour heterogeneity, but these were largely confined to broad statements of diversity. A nuanced understanding of evolutionary history at the subclonal level within a single tumour is crucial for comprehending both the biology and therapeutic responses of patients.

\subsection{Clonal evolution}

Progress in \ac{WGS}, multi-regional tissue sampling techniques, and the development of advanced computational tools collectively enable more intricate studies of breast cancer evolution \pcref{box:technologies,box:phylogeny-reconstruction}. The first study by \textcite{Nik-Zainal2012-zz} used \ac{WGS} data from single samples complemented with bespoke computational methods to reconstruct the evolutionary history of 21 breast tumours in detail. The study traced the tumour's lineage back to the zygote, delineating multiple clonal branches and establishing the temporal sequence of chromosomal rearrangements and driver mutations. Notably, each studied tumour was found to contain a dominant subclonal lineage, making up over half of the cancer cells. The researchers proposed a model wherein sparse, long-lived cellular lineages accumulate mutations passively until a critical event triggers their rapid proliferation, culminating in a clinically detectable tumour mass. 

Other pioneering studies relied on \ac{CNA} inferred from single-cell sequencing that allowed to reconstruct phylogenies. They also showed that breast cancer is a patchwork of clones emerging from `punctuated' clonal expansions, rather than linear progression \parencite{Navin2011-qq,Wang2014-bp,Gao2016-qv}. Furthermore, these works indicated that clonal expansions often correlate with focal \ac{CNA} events. Although single-cell sequencing enhanced our understanding of divergence beyond major expansion events, the technique has limitations. These included low sensitivity and challenges in the accurate reconstruction of phylogenetic trees based on copy number profiles alone \pcref{box:phylogeny-reconstruction}. 

\infobox{box-phylogeny-reconstruction.pdf}{box:phylogeny-reconstruction}{Tumour Life History rectionstruction from \acs{WGS} data}{
    Remarkably, deep \ac{WGS} data provide sufficient information to partially reconstruct the life history of the tumour cell population, from zygote formation to the day of tumour sampling \parencite{Nik-Zainal2012-zz,Dentro2017-jb}. This is possible because mutation distribution contains information about the population structure that could be modelled computationally.

    \subsubsection*{Somatic mutation calling}
    The initial step entails differentiating genuine \acp{SNV} originating from cancer cells from normal germline cells and sequencing errors. The procedure can also extend to structural variants, which involves identifying corresponding sections of sequence reads. However, this task becomes more challenging when dealing with short-read sequencing.

    \subsubsection*{Cancer Cell Fraction}
    For bulk genomic sequence data, the \ac{VAF} for each mutation is calculated using read information:

    \begin{equation}
        VAF = \frac{r_{mutant}}{r_{total}}
    \end{equation}

    \ac{VAF} alone, however, fails to capture the tumour's subclonal architecture due to influences like tumour purity $\rho$ and local copy numbers $n_{loc, t}$ and $n_{loc, n}$. Instead, mutation copy number $n_i$ is used:

    \begin{equation}
        n_{i} = \frac{VAF_i}{\rho} [\rho n_{loc, t} + (1-\rho)n_{loc, n}]
    \end{equation}

    Algorithms such as \href{https://github.com/cancerit/cgpBattenberg}{Battenberg} provide values for ($n_{loc}$) and ($\rho$), by partitioning the genome into distinct copy-number profiles.

    In instances of multiple alleles or subclonal copy number changes, a complex model typically is necessary to accurately infer the number of chromosomes bearing the mutation $n_{chr}$. The \ac{CCF} is then calculated as:

    \begin{equation}
        CCF_i = n_i / n_{chr}
    \end{equation}

    Note, that a fully clonal mutation will have a \ac{CCF} of 1

    \subsubsection*{Subclonal architecture inference from \acp{SNV}}
    
    Clonal expansions within the tumour elevate the frequencies of lineage-specific mutations. Overlooking sporadic mutations that occur during this expansion allows us to model the data as a mixture of binomials. A Dirichlet process serves well for the task of inference of unknown number of clusters. The model is as follows:

    \begin{align}
        r_i &\sim Binomial(r_{total,i},\zeta_i\pi_i) \\
        \pi_i &\sim \mathcal{DP}(\alpha, P_0) 
    \end{align}

    Here $\zeta_i$  the proportion of reads expected for a fully clonal mutation, and $\pi_i$ is the true fraction of tumour cells carrying the mutation. $\pi_i$ is drawn from a stick-breaking process with the base distribution $P_0$ (locations of clusters) and concentration parameter $\alpha$ (the probability weights).

    \subsubsection*{Subclonal architecture inference from \acp{CNA}}

    \acp{CNA} can also inform subclonal population structures. These alterations are estimated by read depth and the disparity between maternal $n_A$  and paternal $n_B$ allelic reads. B-allele frequencies (BAF) indicate these changes:

    \begin{equation}
        BAF_i = \frac{n_{B,i} }{n_{A,i} + n_{B,i}} 
    \end{equation}
    
    Deviations from expected BAF, adjusted for tumour purity, point to subclonal copy number changes. Usually, at least two subclonal populations differing by a single copy number change must be identified.

    \subsubsection*{Phylogenetic tree reconstruction}

    The ``pigeonhole principle" commonly applies to discern evolutionary relationships among subclones. It says that the sum of \acp{CCF} for subclones cannot exceed that of their ancestor. Trees assemble accordingly. But pigeonhole alone can still lead to multiple tree solutions, e.g. in the case of a 30\% and 50\% subclone.
    
    Multiregional sequencing additionally provides better resolution of subclones with similar \acp{CCF}.

    \subsubsection*{Beyond single sample \ac{WGS}}

    Conducting the analysis in a \textbf{multi-regional} way enhances resolution. In a single sample, subclonal clusters may coincidentally exhibit similar \acp{CCF}. However, these clusters are likely to display different proportions in another sample. This variability enables the resolution of smaller clusters with lower presence. The natural extension of this strategy is \textbf{\ac{LCM}-\ac{WGS}} \pcref{box:technologies}.

    \textbf{Single-cell \ac{WGS}} offers significant improvements in inferring subclonal architecture, particularly in enhancing the resolution of the phylogenetic tree's leaves. This technique has a clear advantage over bulk \ac{WGS} in its ability to identify changes present in a single cell that has not yet undergone clonal expansion. However, while single-cell \ac{WGS} often provides enough coverage for reconstructing \ac{CNA} profiles, it faces challenges in accurately constructing phylogenetic trees based on \acp{SNV} as the data is sparse. Moreover, although tree inference based solely on \acp{CNA} is feasible, the results are typically even more ambiguous due to a higher volatility of \acp{CNA} compared to \acp{SNV}, making it difficult to draw definitive conclusions about evolutionary dynamics.

    \subsubsection*{Timing of evolutionary branches}
    Mutations are generally irreversible and increase monotonically over time, offering reliable event ordering within the same branch. However, they do not guarantee chronological timing due to variable mutation rates. To improve chronological estimates, one can limit the analysis to mutational processes ubiquitous in normal cells, like SBS1 or SBS5, and incorporate the patient's age. This method also enables timing of tumour genome rearrangements, as outlined in \textcite{Gerstung2020-sg}.
    }

\textcite{Yates2015-xk} conducted the first multi-regional \ac{WGS} study of primary breast cancers. Although the study's sensitivity was not exceptionally high, its use of a multi-region approach enhanced branch resolution over earlier studies, unveiling considerable genomic diversity across samples. Most cases exhibited long trunk mutations, similar to \textcite{Nik-Zainal2012-zz}, but the study also reported instances of early branching and extensive parallel evolution. Notably, mutations in the \gene{PTEN} and \gene{TP53} genes showed signs of convergent evolution, occurring independently across parallel branches. This study further confirmed that chromosomal events can happen at any evolutionary stage, both early and late, reshaping the genome and contributing to subclonal diversity.

Enhanced spatial resolution in sampling strategies, facilitated by techniques like tissue microdissection, enables more precise inquiries into breast cancer development \pcref{box:technologies}. In the study by \textcite{Casasent2018-gx}, \ac{LCM} isolated cells from both ductal and adjacent invasive regions of ductal carcinomas. Contrary to previous assumptions about breast cancer invasion, identical clones were found both inside and outside the ducts without significant changes. This suggests that the evolutionary changes necessary for invasion occur within the ducts and are initially contained by them. Another microdissection study paired with single-cell DNA sequencing by \textcite{Lips2022-kv} concentrated on invasive recurrence. The study linked this recurrence to untreated clones that had been present for an extended period within \ac{DCIS}.

\figuretextwidth{applications-evolution-tree.pdf}{applications-evolution-tree}
{Highly branched parallel evolution of breast cancer described by \textcite{Nishimura2023-mk}}
{Deep study of breast cancer case reveals complex, highly branched evolutionary path. Key driver events, specifically der(1;16) translocations, emerged independently in two branches when the patient was around 6 and 10 years old. Although only one branch led to an invasive phenotype, parallel driver events occurred across multiple branches. The complex relationship between genome and phenotype becomes evident from the mismatch between the number of driver events and the resulting phenotype. This suggests a potential role for the microenvironment in disease progression.}

The more comprehensive the microdissection strategy and study depth, the greater the number of parallel branches observed in the reconstructed phylogenetic trees. A detailed study by \parencite{Nishimura2023-mk} focused on the early evolution of \ac{DCIS}. The research revealed that many breast cancers exhibited a recurring chromosomal abnormality, der(1;16)\explain{}{\marginfig{side-plot-der1-16.pdf} Unbalanced translocation between Chr1 and Chr16 was frequently observed in study by \textcite{Nishimura2023-mk}.}, acquired early in cancer evolution, roughly around puberty. These der(1;16)-positive clones expanded across large breast regions, spawning multiple independent cancer branches as well as non-cancerous lesions \pcref{fig:applications-evolution-tree}. Conversely, most der(1;16)-negative clones remained restricted to single lobules post-puberty. The findings imply that a branching, multifocal model of cancer progression might be more prevalent than linear models. Additionally, the study revealed no correlation between the number of driver events and histology. This suggests that local microenvironments or other unmeasured intrinsic factors may play a role in cancer development. Further discussion on this discrepancy appears in \cref{sec:discussion-colcluding-remarks}.

Overall, the studies discussed above portray cancer evolution as a complex, multifaceted process characterised by considerable intratumour heterogeneity on a genomic level. Earlier studies focused on a most dominant clone reporting a mostly linear progression of cancer, likely due to technological limitations and undersampling. In contrast, more recent research uncovers a complex landscape with multiple parallel branches that might diverge early in cancer evolution. The mechanisms driving these branches remain a topic of debate, with possibilities including both positive selection pressure and neutral drift \parencite{Sottoriva2015-ci,Williams2016-qh}. The relationship between driver events and specific histology also remains ambiguous, even when focusing on individual evolutionary branches. Likely, this limitation arises because genomic studies focus solely on the intrinsic properties of cancer cells. While the extrinsic microenvironment probably plays but as yet poorly understood role in shaping cancer evolution.

\subsection{Microenvironment}

Single-cell proteomics and transcriptomics studies have substantially advanced our understanding of the tumour microenvironment in breast cancer, going well beyond bulk studies like PAM50 transcriptional classification \parencite{Wu2021-uq,Pal2021-rf,Wagner2019-zp}. These investigations confirm that the general structure of the breast cancer microenvironment closely aligns with the broader cancer landscape, as outlined in \cref{sec:mapping-the-cancer-ecosystem}. The microenvironment commonly comprises immune cells (both myeloid and lymphoid), stromal cells (including \acfp{CAF}, endothelial cells, \acfp{PVL} and adipocytes), and epithelial cells (luminal and myoepithelial). Breast cancer exhibits significant patient-to-patient variation in the relative proportions of these cell types. Moreover, spatially organised recurrent cellular communities correlate with patient survival \parencite{Jackson2020-em, Danenberg2022-zb}.

\subsubsection*{Immune cells}

In the context of cancer, immune cells are often simplistically categorised as either tumour-promoting, such as regulatory T cells (Tregs) and M2 macrophages, or tumour-inhibiting, such as cytotoxic CD8+ T cells, natural killer (NK) cells, CD4+ T-helper cells, and M1 macrophages \pcref{fig:intro-tme-composition}. Generally, the immune system acts as a tumour suppressor, and the presence of tumour infiltrating lymphocytes is linked to a better prognosis \pcref{fig:intro-tme-composition}. However, lymphocytes can enter a so-called `exhausted' dysfunctional state, characterised by the expression of co-inhibitory receptors such as \protein{CTLA-4}, \protein{PD-1} and \protein{LAG3} confirming that this is likely an oversimplification.

Indeed the landscape of immune cell roles in cancer is likely much more complex. For instance, while NK cells generally target tumour cells, some studies suggest they might also enhance tumour vascularisation and even exert immunosuppressive effects \parencite{Retecki2021-se}. Furthermore, T-cell activation states in breast cancer have been reported to represent a continuous and complex spectrum  rather than the discreet categories reported in normal breast  \parencite{Azizi2018-vc,Savas2018-vb}. T-cell diversity largely stems from variations in T-cell receptor (TCR) and antigen-presenting cell heterogeneity \parencite{Azizi2018-vc}. Such variability may result from differing tumour antigens across clonal populations. Such variability may result from differing tumour antigens across clonal populations. Finally, while myeloid cells are generally thought to display quite distinct activation states it has recently been reported that a single myeloid cell can simultaneously express both M1 and M2 programs \parencite{Azizi2018-vc}. 

Spatial analysis of immune cell distribution in breast cancer indicates non-uniform patterns linked to prognosis \parencite{Danenberg2022-zb, Wang2023-bo}. For instance, dysfunctional T cells often cluster near Tregs, which are believed to control their proliferation and activation. Large dysfunctional T-cell aggregates may signify tumour cells resistant to ongoing immune attacks, thereby chronically stimulating cytotoxic T cells. Conversely, structures containing antigen-presenting cells, macrophages, and T cells associate negatively with survival. In contrast, cell communities featuring granulocytes, and particularly \acf{TLS} with B cells, correlate with better survival outcomes.

\subsubsection*{Stroma}

In the stroma, fibroblasts are the most extensively studied cells. Researchers typically categorise these fibroblasts into two broad types: SMA- fibroblasts and SMA+ myofibroblasts \parencite{Costa2018-ir}. While efforts exist to categorise more types, these efforts face challenges due to the cell-type fluidity of fibroblasts \parencite{Cords2023-og,Wu2021-uq}. These cells serve essential structural functions and engage in immune cell recruitment and extracellular matrix remodelling.

Of specific interest is the \protein{SMA+} \protein{FAP+} subset of fibroblasts, which acts as a crucial immunosuppressive agent. This subset attracts T lymphocytes and facilitates their differentiation into immunosuppressive Tregs, thereby aiding in immune evasion in cancer. In contrast, \protein{SMA+} \protein{FAP-} fibroblasts lack this immunosuppressive function \parencite{Costa2018-ir}. Spatial distribution analysis indicates that myofibroblasts frequently reside at the tumour-stromal interface in breast cancer, which suggests their role in lymphocytic exclusion \parencite{Danenberg2022-zb}. Although this seems to promote tumour growth, other fibroblasts located near tertiary lymphoid structures (TLS) correlate with a better prognosis \parencite{Danenberg2022-zb, Cords2023-og} \pcref{fig:intro-cancer-neighbourhoods}. 

In a different spatial study, \textcite{Risom2022-uw} demonstrated that `desmoplastic' stroma, characterised by a high frequency of fibroblasts and intense collagen deposition around ducts, correlates with a favourable prognosis and reduced likelihood of progression to invasive cancer. Notably, this type of stroma coexists with a thin and discontinuous myoepithelial layer. This type of stroma may indicate a robust immune response, facilitated by a permeable myoepithelium that exposes the cancer cells to immune surveillance.

Endothelial cells and \acp{PVL} are other significant components of the tumour microenvironment. These cells constitute the vasculature, critical for tumour growth and metastasis. In the context of breast cancer, a dense vascular stroma correlates with a poor prognosis \parencite{Danenberg2022-zb}.

\figuretextwidth{applications-epithelial-plasticity.pdf}{applications-epithelial-plasticity}
    {Immune niches and breast cancer cell plasticity}
    {Experimental design from \textcite{Sinha2021-mf}. A lentivirus vector delivers \protein{acERBB2} into mammary gland cells, inducing the formation of \acf{DCIS}. The \ac{DCIS} cells stochastically adopt one of two phenotypes: 'indolent' or 'aggressive'. The 'indolent' phenotype features a luminal-like state and the presence of T cells, whereas the 'aggressive' phenotype is characterised by a basal-like state and high neutrophil infiltration. Intriguingly, reimplantation of cells from both phenotypes reveals functional indistinguishability, as they produce similar ratios of 'aggressive' and 'indolent' phenotypes in a new host. This experiment suggests that cells with equivalent oncogenic potential can exist in diverse phenotypic states. Additionally, neutrophil inhibition via anti-\protein{IL17} treatment reduces the likelihood of the 'aggressive' phenotype emerging, suggesting their role in promoting the 'aggressive' phenotype.}


\subsubsection*{Cancer Epithelial plasticity}

As previously discussed, the breast ductal system's epithelium comprises luminal and basal cell types, which exhibit significant plasticity. In cancer, this epithelial plasticity not only persists but also amplifies, leading to a diverse range of transcriptional states \parencite{Wagner2019-zp, Wu2021-uq, Pal2021-rf}. The study by \textcite{Wu2021-uq} dissected the intrinsic expression profiles of breast cancer cells into various transcriptional modules, each showing distinct patterns. Notably, the \acf{EMT} and proliferative modules appeared mutually exclusive. This observation corroborates the incompatibility between a mesenchymal-like state and proliferation in breast cancer \parencite{Tsai2012-hb}. However, despite the coexistence of multiple tumour cell phenotypes within all tumour ecosystems, one phenotype often dominates, potentially reflecting the expansion of the fittest tumour subclones \parencite{Wagner2019-zp}.

The extent to which breast cancer plasticity influences invasion has been explored by \textcite{Sinha2021-mf} through a mouse model study \pcref{fig:applications-epithelial-plasticity}. In this research, cancer cells sharing the same genetic background — specifically, activated \gene{ERBB2} — gave rise to two distinct types of \ac{DCIS} lesions within the same animal: `indolent' and `aggressive'. Importantly, cells from both lesion types demonstrated equal ability to initiate invasive tumours. The key differences between these two lesion types lay in their intrinsic phenotypes and surrounding immune environments. The `aggressive' lesions contained more basal-like cells and macrophages, while the `indolent' lesions were enriched for luminal cells and T cells, hinting that local environmental factors influence the tumour's invasive potential.

\subsection{Open questions on breast cancer progression}
\label{sec:application-open-questions}
Despite considerable advancements in breast cancer research, our understanding of disease progression and underlying mechanisms remains incomplete. We have identified key driver mutations and broadly understand the mutation processes that lead to their accumulation. We are familiar with the basic composition of the breast cancer microenvironment and the cellular communities typically associated with unfavourable clinical outcomes. Yet, a comprehensive understanding of how genomes and environment interact and influence each other eludes us. Our current models of cancer progression are simplistic, primarily because few studies have rigorously examined the phylogenetic structure of patients' breast cancer and even fewer have simultaneously characterised both genetic and \ac{TME} components. As a result, several critical questions about cancer progression remain unclear:

\textbf{What are the cells of origin, and to what extent do they dictate the future trajectory of cancer evolution?} Several studies suggest expressional similarity of a cancer subtype to the exact cell lineage. For example HR+ Luminal cancers cells are similar to luminal cell types, while TNBC cells have some characteristics of basal cells \parencite{Skibinski2015-rh, Taurin2020-mq} \pcref{sec:applications-normal-breast-development}. Tracing the cancer and normal cell lineages together with their intrinsic expression profiles may help to elucidate the cells of origin and their role in cancer progression.

\textbf{Does the environment play a role in the initial onset, or are the initiating factors solely genetic?} The study on phenotypically normal tissue, such as the one conducted by \textcite{Nishimura2023-mk}, reveals that numerous branches may contain driver events yet not appear malignant. A systematic profiling of the environment in which these branches exist could elucidate the environment's role in the initial onset of cancer.

\textbf{Why do some clones successfully colonise ducts while others do not?} The ductal system serves as the environment where \ac{CIS} evolves before progressing to invasive stages. Research on \ac{CIS} has reported clonal heterogeneity \parencite{Nishimura2023-mk, Yates2015-xk, Casasent2018-gx} and associated recurrence with minor clones that persist within the ductal system \parencite{Lips2022-kv}. The reasons for this diversity and the role of the environment in this process remain unclear.

\textbf{What types of environments trigger invasion, and what genetic backgrounds sustain clones in invasive lesions?} Evidence suggests that clones with the same genetic background could exist in both \ac{CIS} and invasive states simultaneously \parencite{Casasent2018-gx} and that the microenvironment contributes to the onset of invasion \parencite{Sinha2021-mf} \pcref{fig:applications-epithelial-plasticity}. However, the extend the genetic backgrounds of these clones influence the process remains unclear. The question of whether any \ac{CIS} clone can transition to an invasive state, given the right conditions, or if specific genetic backgrounds are required, warrants further investigation.

\textbf{In the context of metastasis, what roles do the environment and genetics within the primary tumour play, and what is the role of the environment at distant sites?} As discussed in \cref{sec:intro-metastasis}, multiple multi-region genomic sequencing studies have explored the phylogenetic links between primary tumours and their corresponding metastases in breast cancer patients \parencite{De_Mattos-Arruda2019-sb,Yates2015-xk,Yates2017-xc,Brown2017-de}. Evidence suggests that breast metastases often arise from clonal, rare seeding events \parencite{Brown2017-de, De_Mattos-Arruda2019-sb}, although \textcite{De_Mattos-Arruda2019-sb} suggest that polyclonal seeding may be more prevalent. This could be explained by several seeding events, or by an environment favourable for the formation of \acp{DTC} clusters, as suggested by mouse models \parencite{Cheung2016-nb}. It is also known that \acp{DTC} can seed (pre-)metastatic niches in bone, liver and probably other organs, and here they may remain dormant for years only to be triggered to awaken and cause overt metastasis many years or even decades later. The cause for this may be cellular intrinsic and/ or a change in the environment \parencite{Szczerba2019-mt,Diamantopoulou2022-sb}. The challenges associated with detecting and characterising dormant cancer cells in their premetastatic remains a major challenge for cancer research.

\explain{}{\marginfig{side-plot-metastasis-breast.pdf} While metastasis to the auxiliary \ac{LN} is common, it doesn't seem to directly relate to the formation of distant metastases. Whether ongoing evolution in the \ac{LN} influences the primary tumour site is currently unknown.}
\textbf{Do cancer - immune interactions in the lymph node shape the disease trajectory?}
Some studies propose that \ac{LN} metastases indicate cancer cells' ability to survive and grow in other organs, rather than directly aiding their dissemination \parencite{Fisher1977-ua, Ullah2018-xe}. This is surprising because \ac{LN} metastases exist in an environment abundant with immune cells. While these metastases may not directly advance to distant organs, they could influence breast cancer progression by modulating the immune response in the primary tumour.

While the spatial genomics technologies outlined in \cref{sec:chapter-basiss-model} may not offer immediate, definitive answers to these questions, they do provide a framework for further investigations. Specifically, the \ac{BaSISS} approach allows for the concurrent profiling of phylogeny and microenvironment across extensive tissue sections. In this chapter, I will show how data generated using this approach could help to investigate the relationship between genetic and microenvironmental evolution in breast cancer progression. Specifically, the analysis traces the evolutionary history of breast cancer from the earliest stages of tumour development to \ac{LN} metastasis.

\section{Results}
\label{sec:applications-results}

\subsection{Recap of the \acs{BaSISS} workflow}

\figuretextwidth{applications-bassis-recap.pdf}{applications-bassis-recap}{The \acs{BaSISS} workflow to generate cancer clone maps}{Following de novo mutation detection and subclone discovery in \ac{WGS} data, the \ac{BaSISS} workflow is performed as follows: (1) bespoke mutation-specific padlock probes are designed. (2) \ac{BaSISS} transcripts are detected. To achieve this, \ac{BaSISS} padlock probes hybridise to complementary DNA (cDNA) in situ. By virtue of a highly specific DNA ligase, only completely target-complementary padlock probes are ligated and form closed circles. Ligated probes are amplified through rolling circle amplification and their reader barcodes are detected in tissue space through sequencing by ligation with fluorophore-labelled interrogation probes and cyclical microscopy. (3) Mathematical modelling of \ac{BaSISS} signals and the genotype of clones is then performed to derive clone maps. (4) Subsequent phenotype and microenvironment characterization of clones is then possible, by integrating clone fields with spatial datasets acquired from serial tissue sections. The \ac{BaSISS} model and cell typing are described in \cref{sec:chapter-basiss-model,sec:chapter-basiss-multimodal}}

The \ac{BaSISS} workflow is centred around fresh frozen tissue blocks that undergo serial cryosectioning to generate tissue for bulk \ac{WGS} and z-stacked sections for in-tissue spatial clone mapping and spatial phenotyping \pcref{fig:applications-bassis-recap}. Following subclone detection from bulk \ac{WGS} data, there are three core \ac{BaSISS} steps. First, to facilitate detection of multiple clones of interest, \ac{BaSISS} padlock probes with sequence-specific oligonucleotide target recognition arms are designed towards both mutant and wild-type alleles of clone-defining somatic variants. A unique 4–5 nucleotide reader barcode on each probe enables multiplexing \parencite{Ke2013-ux}. \ac{BaSISS} targets can take the form of any expressed somatic mutation, including point mutations and rearrangement breakpoints, and can be supplemented with copy number alterations \pcref{tab:basiss-padlock-probes-P1,tab:basiss-padlock-probes-P2}. Second, \ac{BaSISS} and transcript detection are performed as previously described for gene expression \ac{ISS} using cyclical microscopy \parencite{Ke2013-ux,Svedlund2019-xb} \pcref{fig:applications-bassis-recap,sec:protocol-suppl-basiss-iss}.

Third, continuous spatial subclone maps are generated using a statistical algorithm that exploits \ac{BaSISS} signals as well as local cell counts (derived from the DAPI channel during the fluorescence microscopy of \ac{BaSISS}) using two-dimensional Gaussian processes \pcref{sec:chapter-basiss-model,fig:conceptual-bassis-model}. The variational Bayesian model also accounts for unspecific or wrongly decoded signals and variable probe efficiency and is augmented by variant allele fractions in the bulk genomic sequencing data. In an optional, fourth characterisation step, \ac{BaSISS} clone maps can be aligned and integrated with additional layers of spatial phenotype data. In this study, we performed spatially resolved single-cell transcriptomics using targeted \ac{ISS} (using a previously published 91 gene oncology, a novel 62 gene immune panel and drawing on published single-cell RNA sequencing data) \parencite{Svedlund2019-xb,Wu2021-uq} and \acf{IHC} staining \pcref{sec:chapter-basiss-multimodal,sec:protocol-suppl-ihc,sec:protocol-suppl-expression-panel,sec:protocol-suppl-immune-panel,appfig:appendix-multimodal-onco-panel,appfig:appendix-multimodal-immune-panel}. Additional sections were obtained to perform validation of our workflow using \ac{LCM} and low-input \ac{WGS} as previously described \parencite{Ellis2021-du} \pcref{sec:bassis-validation,sec:protocol-suppl-lcm-wgs,fig:basiss-lcm-validation}.

\subsection{Two cases of multifocal breast cancer}
\label{sec:applications-cohort-introduction}
The cohort includes eight tissue blocks from two patients (P1 and P2) who underwent a surgical mastectomy for a multifocal breast cancer. These patients were selected to permit a comparison between genetic and histological progression models in early breast cancer development \parencite{Cowell2013-du} \pcref{fig:applications-cohort,fig:applications-breast-structure}. P1 had two separate oestrogen receptor (ER)-positive, human epidermal growth factor receptor 2 (HER2)-negative \acfp{PBC} within a 5-cm bed of \ac{DCIS}; we used tissue blocks from both \acp{PBC} (samples P1-ER1 and P1-ER2) and three regions from \ac{DCIS} (samples P1-D1, P1-D2 and P1-D3). P2 had two separate PBCs of the `triple-negative' subtype (lacking the ER, progesterone receptor and HER2). We sampled both PBCs (samples P2-TN1 and P2-TN2) and an axillary lymph node that contained metastatic cancer deposits (sample P2-LN1) \pcref{fig:applications-cohort}.

\figuretextwidth{applications-cohort.pdf}{applications-cohort}{Breast cancer cohort description}{The two cases of multifocal primary breast cancer (PBC) used to develop the \acs{BaSISS} approach. Coloured tiles report the histological features within each sample and the experiments performed. The number of clones identified by WGS and targeted by \acs{BaSISS} are reported as white numerals. Notations are: NST - No special type, ER - oestrogen receptor positive, TN - triple negative, D - ductal carcinoma \textit{in situ}, LN - lymph node.}

\subsection{Charting histogenomic relationships}

Histology-driven sampling of well-defined stages of cancer progression can uncover mechanisms and markers of disease progression \parencite{Risom2022-uw,Casasent2018-gx,Cowell2013-du,Nirmal2022-sq}. Up to two-thirds of \acp{PBC} contain both invasive cancer and intermixed \ac{DCIS}, a non-obligate precursor lesion. How these distinct `stages' of cancer development might relate to genetic diversification within the same tissue is generally unknown \parencite{Kole2019-hl} \pcref{fig:applications-breast-structure}. To demonstrate that \ac{BaSISS} can chart these relationships across entire tissue sections, we examined three PBC samples with intermixed invasive and \ac{DCIS} histology: P1-ER1, P1-ER2 and P2-TN1 (\cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2} and \cref{appfig:appendix-applications-PBC-P1,appfig:appendix-applications-PBC-P2}a-c).

\figurefloat{applications-maps-PBC-P1.pdf}{applications-maps-PBC-P1}{Genetic clones mapped in histological context from two \acs{PBC} samples in P1}{\textit{\textbf{top}}, \ac{BaSISS} maps of two \acp{PBC} from P1 with intermixed \ac{DCIS} and invasive cancer. The most prevalent genetic clone is projected as a coloured field (corresponds to \textit{\textbf{middle}} section) on DAPI images (reported if the \ac{CCF} > 25\% and the inferred local cell density > 300 cells per mm\textsuperscript{2}). Pie charts report the \ac{WGS}-estimated clone compositions. Inset images are regions of P1-ER2 (H\&E-stained serial tissue sections) that represent three histological progression states. \textit{\textbf{middle}}, The phylogenetic tree was inferred from P1 multi-region \ac{WGS}. Branches and nodes are coloured to reflect the clone maps. Heatmaps report clone composition in 34 and 44 histologically annotated epithelial-containing microregions of P1-ER1 and P1-ER2, respectively. Microregions include individual ducts or randomly selected regions of invasive cancer. \textit{\textbf{bottom}}, \ac{IHC} per P1-ER1 (left) and P1-ER2 (right) for the proliferative marker \protein{Ki-67} in six clone territories (indicated by contour colour); the percentage of nuclei staining positive (brown) is reported. Notations are: HP - hyperplasia; N - normal ducts. Scale bars, 250 $\mu$m for microregions and 2.5 cm for clone maps.}

\figurefloat{applications-maps-PBC-P2.pdf}{applications-maps-PBC-P2}{Genetic clones mapped in histological context from the \acs{PBC} sample in P2}{\textit{\textbf{top}}, As in top panel \cref{fig:applications-maps-PBC-P1}, but a clone map of P2-TN1. Mini-images report \ac{ISS}-derived cell types (right) and H\&E tissue section snapshots of the two cancer growth patterns (GP1 and GP2) reported in P2-TN1 (left). \textit{\textbf{bottom}}, Phylogenetic tree for P2 and heatmap of 36 P2-TN1 microregions, as in middle panel \cref{fig:applications-maps-PBC-P1}. Branches relating to clones not detected in this sample (that is, only found in P2-LN1) are shaded grey. The bottom heatmap is the estimate by the histopathologist and reports the contribution of different growth patterns to the microregion, defined by distinct nuclear and architectural features. Scale bar, 2.5 mm.}

\ac{BaSISS} detected 2–4 subclones per \ac{PBC} in accordance with bulk \ac{WGS} data. Clone maps (top panels \cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2}) and the quantitative clonal composition of 73 individually annotated microregions (middle panels \cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2}, \cref{appfig:appendix-applications-PBC-P1}a,b and \cref{appfig:appendix-applications-PBC-P2}a,b) revealed that individual subclones form spatial patterns that were, by varying degrees, related to the histological progression states. Normal tissue elements, including immune aggregates and histologically normal ducts, appear unstained consistent with a wild-type status for the targeted clones (green and yellow contours, respectively; \cref{fig:applications-maps-PBC-P1}). In P1-ER2, an area of hyperplasia was predicted and confirmed by \ac{LCM}–\ac{WGS} to be genetically unrelated to the cancer (blue contour; bottom panel \cref{fig:basiss-lcm-validation} and top panel \cref{fig:applications-maps-PBC-P1}).

In each \ac{PBC}, the genetic and histological progression models were broadly consistent, in which the invasive disease was mainly composed of cells from the most recently diverged subclone: P1-red, P1-purple and P2-purple in samples P1-ER1, P1-ER2 and P2-TN1, respectively (middle panel \cref{fig:applications-maps-PBC-P1}, bottom panel \cref{fig:applications-maps-PBC-P2}). By contrast, earlier diverging clones co-localised entirely or in part to the histological pre-invasive lesion: \ac{DCIS}. For example, in P1-ER2, \ac{BaSISS} predicted that green branch mutations were completely absent from the invasive compartment, a conclusion that is supported by three separate microdissections (\ac{LCM}–\ac{WGS}) from distant regions of invasive cancer in P1-ER2 (bottom panel \cref{fig:basiss-lcm-validation} and \cref{appfig:appendix-applications-PBC-P1}c).

However, in each \ac{PBC}, there was a subclone that spanned both \ac{DCIS} and invasive histology, revealing that disconnects between histological and genetic progression states can exist. This was the case for clone P1-red in P1-ER1 and clone P1-purple in P1-ER2. These \ac{DCIS}-invasive spanning clones could be distinguished from each other by hundreds of private mutations, including different inactivating driver mutations in \gene{PTEN}, indicating parallel evolution along these divergent lineages that resulted in two distinct instances of cancer invasion (total mutation numbers label the phylogenetic tree branches; middle panel \cref{fig:applications-maps-PBC-P1}). The spatial predictions of the \ac{BaSISS} model of intraductal acquisition of \gene{PTEN} mutations and \gene{PTEN} protein loss was confirmed by \ac{LCM}–\ac{WGS} and \ac{IHC}, respectively (bottom panel \cref{fig:basiss-lcm-validation} and \cref{appfig:appendix-applications-PBC-P1}d). In sample P2-TN1, the only predicted driver point mutation was a deleterious mutation in the tumour suppressor gene \gene{TP53}, and this was detected in both \ac{DCIS} and invasive compartments and was also present in all cancer regions of the second \ac{PBC}, P2-TN2, consistent with an early onset in the development of this cancer (phylogenetic tree; \cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2}). These data therefore suggest that many, if not all, of the genetic events necessary to initiate the invasive transition in these three cancers were acquired within the ducts, and subsequently both intraductal expansion and stromal invasion ensued.

\subsection{Phenotypic changes accompany progression}

Next, by integrating additional layers of spatial data, we sought to establish how phenotypic changes relate to genetic-state and histological-state transitions. In P1-ER1 and P1-ER2, consistent with a more proliferative phenotype, \gene{PTEN}-mutant clone regions exhibited denser \protein{Ki-67} \ac{IHC} nuclear staining, than \gene{PTEN} wild-type ancestral clone regions (\ac{FDR} = 0.004 P1-red versus P1-orange; and \ac{FDR} = 0.03 P1-purple versus P1-green) (bottom panel \cref{fig:applications-maps-PBC-P1} and \cref{appfig:appendix-applications-PBC-P1}e). However, for a given genetic clone, the \protein{Ki-67} score was similar irrespective of whether it occupied a \ac{DCIS} or invasive state, indicating that upregulation of \protein{Ki-67} is temporally related to acquisition of a \gene{PTEN} mutation and precedes invasion.

By contrast, cellular resolution spatial transcriptomics analysis of P1-ER2 revealed that epithelial cell expression of several genes -- \gene{CLDN4} (encoding \protein{claudin 4}), \gene{ACTB} (encoding $\beta$-actin), \gene{KRT5} (encoding keratin 5) and \gene{CTSL2} (encoding lysosomal cysteine protease \protein{cathepsin V}) -- differed between \ac{DCIS} and invasive compartments occupied by the same, P1-purple, clone (\cref{appfig:appendix-applications-PBC-P1}f). These transcriptional changes might therefore be considered more closely linked to the histological transition rather than genetic changes traced by this approach. Expression of \gene{CLDN4} was consistently lower in the invasive compartment than to each \ac{DCIS} clone. However, for some genes such as \gene{ACTB,} expression patterns changed in opposing directions in the invasive cancer relative to the sampled \ac{DCIS} clone (expression is higher than P1-green \ac{DCIS} (\ac{FDR} = 0.02) and lower than P1-purple \ac{DCIS} (\ac{FDR} = 0.013)) or were highly specific to a genetically more distant \ac{DCIS} clone (\cref{appfig:appendix-applications-PBC-P1}f).

Attempts to isolate the changes associated with invasive transition might also be confounded by heterogeneity within the invasive compartment. In P2-TN1, we therefore sought to examine whether the two genetically distinct invasive subclones (P2-blue and P2-purple) were phenotypically distinct. The two cancer clones exhibited distinct morphological (nuclear and architectural) features ($P$ = 0.04, Fisher’s exact test) (H\&E image insets; \cref{fig:applications-maps-PBC-P2}) and occupied neighbourhoods with different stroma (\ac{FDR} = 0.02) and immune cells such as myeloid cell densities (\ac{FDR} = 0.08) (mini-image insets; top panel \cref{fig:applications-maps-PBC-P2} and \cref{appfig:appendix-applications-PBC-P2}a–c). Transcriptional programs were also distinct, with statistically significant differences in gene expression for 12 of 91 genes between clones (\cref{appfig:appendix-applications-PBC-P2}d). Together, these data indicate that the particular clones sampled can have a profound effect on attempts to identify the phenotypic changes implicated in driving or arising during histological progression.

\subsection{Growth patterns of pre-invasive clones}
\label{sec:applications-DICS-growth}
\figurefloat{applications-maps-DCIS.pdf}{applications-maps-DCIS}{Growth patterns and histological associations of \acs{DCIS} clones}{\textit{\textbf{top}} \ac{BaSISS} maps of pure \ac{DCIS} samples: P1-D1 and P1-D2. The most prevalent genetic clone is projected as a coloured field (which corresponds to \textit{\textbf{middle}}) on DAPI images (reported if the CCF > 25\% and the inferred local cell density > 300 cells per mm\textsuperscript{2}). Scale bar, 5 mm. The pie chart reports the \ac{WGS}-estimated clone composition. The white dashed contours delineate morphologically defined lobules. The beige contours mark 114 and 40 manually selected microregions in P1-D1 and P1-D2, respectively, the clonal composition of which is reported by the heatmaps in \textit{\textbf{middle}}. Microregions were manually selected and represent single or small groups of intimately related acini or ductules from the same lobule. \textit{\textbf{middle}}, The phylogenetic tree inferred from P1 multi-region \ac{WGS}. Branches and nodes are coloured to reflect the clone maps. Only branches detected in P1-D1 and P1-D2 are coloured. WT - wild type. \textit{\textbf{bottom-left}}, H\&E images report representative subclone histological features in regions selected from \textit{\textbf{top}}. Scale bars, 100 $\mu$m and 50 $\mu$m (vacuoles). \textit{\textbf{bottom-right}}, Stacked bar plot summarizes histological features of microregions dominated by P1-green (n = 66) or P1-orange (n = 72). Nuclear pleomorphism is a measurement of the amount of variability in size and shape of the nuclei and is a major determinant of the histological grade.}

To demonstrate that \ac{BaSISS} can be used to chart growth patterns in relation to complex tissue structures, we turned our attention to three \ac{DCIS} samples from P1 that spanned a tissue surface area of 224 mm\textsuperscript{2} (P1-D1, P1-D2 and P1-D3) (top panel \cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS}a). The adult female breast comprises multiple, branching ductal systems, termed lobes, that extend from the nipple surface to the acini of the lobules \parencite{Going2004-af, Schnitt2013-fw}. \ac{DCIS} arises from the duct epithelium and is considered a lobar disease as it typically involves the ducts and lobules of a single lobe \parencite{Pinder2010-vl}. Although \ac{DCIS} is known to be genetically heterogeneous \parencite{Casasent2018-gx}, how \ac{DCIS} clones are organised and grow through the wider duct system remains elusive \parencite{Thomson2001-sm}.

The clone maps generated for the three samples formed striking mosaics of mainly green and orange, and occasional blue and grey that localized to areas of histologically confirmed \ac{DCIS} (top panel \cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS}a). Immune clusters and occasional normal or hyperplastic ducts appeared white (unstained), consistent with a different genetic ancestry. In P1-D3, a 3-mm length of a large duct exhibited both a genetic and a histological transition from normal ductal epithelium to \ac{DCIS} along its length, confirming that, although neoplastic involvement was extensive in this lobe, it was incomplete (\cref{appfig:appendix-applications-DCIS}a). On dividing the glandular tissue into lobules (white dashed contours; \cref{fig:applications-maps-DCIS}), it was apparent that a handful of lobules contained a single clone, but often multiple clones co-occurred. Indeed, we were surprised to observe that the same clones repeatedly co-existed within lobules that spanned centimetres of tissue. These appearances seem at odds with the traditional model of clonal competition in which a fitter clone generates localised monoclonal sweeps.

\explain{}{\marginfig{side-plot-dcis-sweeps.pdf} Cartoon of a lobe of the breast with normal anatomy (left) and \ac{DCIS} (right), with lobules exhibiting monoclonal and polyclonal involvement.}

However, at finer, sublobular resolution, complete or near-complete clonal sweeps are the dominant pattern, as exemplified by assaying 146 representative microscopic regions that represent individual or small clusters of intimately related acini and ducts (beige contours; \cref{fig:applications-maps-DCIS} top). The existence of frequent clonal sweeps as inferred by \ac{BaSISS} (\cref{fig:applications-maps-DCIS} middle) was corroborated by \ac{LCM}–\ac{WGS} of additional microregions (\cref{appfig:appendix-applications-DCIS}b). In some instances, including P1-D1-88 (\cref{appfig:appendix-applications-DCIS}c) and P1-D2-0 (\cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS}d–f), clonal interfaces are directly observed within a continuous anatomical space. However, more commonly, rapid clone field transitions (see interactive maps at \href{https://www.cancerclonemaps.org/}{https://www.cancerclonemaps.org/}) coincided with the myoepithelial cell layer and/or basement membrane that define an acinus or ductule border. It thus transpires that the microanatomical structure of resident tissues can have, an as yet poorly understood, role in shaping observed subclonal architectures (top panel \cref{fig:applications-maps-DCIS}).

\subsection{\acs{DCIS} clone-specific phenotypes}

Integration of histological and spatial gene expression data from serial sections revealed that the \ac{DCIS} clones, P1-green and P1-orange, exhibit many phenotypic differences that are consistent across large tissue areas (\cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS,appfig:appendix-applications-DCIS-expression}). Histogenetic associations were very strong, with regions dominated by P1-green being more likely to have an intermediate rather than a low nuclear grade ($P$<0.0001; Fisher’s exact test after Bonferroni correction), exhibit more nuclear pleomorphism ($P$ < 0.0001), necrosis ($P$ < 0.0001), vacuoles ($P$ < 0.0001) and a non-solid architectural growth pattern ($P$ < 0.0001) (\cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS}).

Clone and cell type-resolved spatial gene expression analysis using targeted \ac{ISS} further corroborated phenotype–genotype correlations. A total of 28 of 91 interrogated genes were differentially expressed by the two main clones (\ac{FDR} < 0.1, fold change > 1.5 both ways; \cref{appfig:appendix-applications-DCIS-expression}). Consistent with a higher nuclear grade, P1-orange epithelial cells exhibited higher expression of the cell-cycle regulatory oncogenes \gene{CCND1} and \gene{CCNB1} and the oncogene \gene{ZNF703}, which have been linked to adverse clinical outcome \parencite{Solin2013-zy}. Overall, architectural and nuclear appearances and gene expression profiles were remarkably lineage-specific, and it was particularly notable that these different patterns could also be appreciated spatially, in regions with sublobular, microscopic clone intermixing, adding weight to the clone composition predictions by the model (\cref{appfig:appendix-applications-DCIS}d).

\subsection{Metastatic clones in a lymph node}
\label{sec:applications-LN}
Lymph node metastasis is associated with higher rates of cancer mortality \parencite{Jatoi1999-yr}. Whether it has an active role in facilitating cancer progression or simply reflects a more aggressive or distinct biology of certain clones is largely unknown. A substantial challenge is low cancer purity of diffusely infiltrated lymph nodes, which can make it difficult to separate cancer from immune cell-derived molecular signals. To demonstrate that \ac{BaSISS} can facilitate the simultaneous study of cancer and immune compartments in such challenging cases, we analysed \ac{BaSISS}, histological annotation and \ac{ISS} targeted gene expression datasets from sample P2-LN1 (\cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-LN,appfig:appendix-applications-LN-hypoxia}).

\figuretextwidth{applications-LN-genome.pdf}{applications-LN-genome}{genomic structures in P2-blue and P2-orange clones}{Plots of the genomic structures in P2-blue and P2-orange clones in the vicinity of the \gene{HER2} gene, derived from \ac{WGS} data of P2-TN2 and P2-LN1. Vertical lines represent genomic rearrangement breakpoints coloured by the phylogenetic tree branch where the event occurred. Dots represent local (binned) copy number. \gene{HER2} amplification, \gene{CACNB1} fusion and \gene{HER2} mutation are \ac{BaSISS} targets used to track this complex event. \acs{BFB}, breakage-fusion-bridge.}

\ac{BaSISS} in P2-LN1 targeted 13 trunk and branch alleles, including point mutations and an expressed novel internal fusion in the \gene{CACNB1} gene that was co-amplified with the clinically targetable breast cancer oncogene \gene{HER2} in a \acf{BFB} event (\cref{fig:applications-LN-genome} and Supplementary Data Table 1). The model detected two clones (P2-blue and P2-orange) that formed spatially segregated patterns in P2-LN1 (\cref{fig:applications-maps-LN}). Only P2-blue was detected in primary breast tumours (P2-TN1 and P2-TN2) (\cref{fig:applications-maps-PBC-P2} and \cref{appfig:appendix-applications-PBC-P2}b).

\figurefloat{applications-maps-LN.pdf}{applications-maps-LN}{Intrinsic and extrinsic features of metastatic subclones in a lymph node}{\textit{\textbf{top-left}}, \ac{BaSISS} map of P2-LN1, which relates to P2-TN1 \pcref{fig:applications-maps-PBC-P2} and P2-TN2 (cref{appfig:appendix-applications-PBC-P2}a,b). The most prevalent genetic clone colours are projected as coloured fields on the DAPI image (reported if the \ac{CCF} > 25\%; a threshold of 5\% is used in regions of diffusely infiltrating blue to allow visualization in very high normal contamination regions). Scale bar, 2.5 mm. Coloured contours define microregions with distinct metastatic cancer growth patterns (M-GP1 and M-GP2); `+' indicates the surrounding sinus epithelium. \textit{\textbf{top-right}}, Phylogenetic tree inferred from P2 multiregion \ac{WGS}.  Branch and node colours inform the clones mapped in \textit{\textbf{top-left}}. \gene{HER2} amplification (orange squares: pale - low, bright - high amplifications), \gene{CACNB1} fusion (red square) and \gene{HER2} mutation (red star) are \ac{BaSISS} targets used to track this complex event. The top heatmap reports the \ac{BaSISS} clone contribution to 39 histologically annotated microregions from a (regions with 5\% or more tumour cells are included). The bottom heatmap reports microregion histological features. Pan-CK, pan-cytokeratin. \textit{\textbf{middle-right}}, Representative areas of the two main growth patterns stained with H\&E. Scale bar, 100 $\mu$m. \textit{\textbf{bottom}}, From left to right: volcano plot of immune cell expression of the 62 genes in the \ac{ISS} immune panel, volcano plot of epithelial cell expression of the 91 genes in the \ac{ISS} immune panel. Significantly (\ac{FDR} > 0.1), differentially expressed (fold change of more than 1.5 both ways) genes are coloured. Violin plots depict clone-specific cell-type contribution posterior density of the generalized linear mixed model with region-specific random effect, and includes the 22 clone territories with a dominant clone fraction > 0.05 in P2-LN1. Significant comparisons were controlled for \ac{FDR} using the Benjamini–Hochberg procedure.}

Detailed histological annotation, blinded to the clone territories, was performed using a combination of H\&E, \protein{CD45} and pan-cytokeratin \ac{IHC} and identified multiple metastatic cancer growth patterns (coloured contours; \cref{fig:applications-maps-LN}). Intersecting the clone maps and histological annotations revealed strong associations between the two detected clones and the two main histological growth patterns ($P$ < 0.0001, Fisher’s exact test) (\cref{fig:applications-maps-LN} top-right). The P2-orange clone formed monotonous sheets of cancer cells, exhibited weak immunoreactivity for pan-cytokeratin and often occupied sinusoidal structures. By contrast, P2-blue cells stained more strongly for pan-cytokeratin and, when clustered, surround densely packed lymphocyte cores (\cref{fig:applications-maps-LN} right and \cref{appfig:appendix-applications-LN}a–d).

We sought to determine whether transcriptional differences support the spatial inference of clones. Consistent with the known HER2 amplification, P2-orange expressed higher levels of HER2 (bottom panel \cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-LN}c). A total of 17 of 91 genes were differentially expressed and many of these are implicated in critical biological cancer pathways and/or have recognized prognostic value, including \gene{CTSL2}, \gene{VEGFA} (encoding vascular endothelial growth factor receptor A) and CD24 \parencite{Sereesongsaeng2020-vp, Kwon2015-jk} (bottom panel \cref{fig:applications-maps-LN}). Spatially plotting these genes confirmed that clone-specific expression patterns are recapitulated within multiple, spatially distinct expansions across more than 1 cm\textsuperscript{2} of tissue (\cref{appfig:appendix-applications-LN}a–c).

Integration of spatial transcriptomics data also revealed that metastatic subclones occupied distinct immune microenvironments. Relative to P2-orange cells, P2-blue cells resided in neighbourhoods enriched for T cells and B cells (bottom panel \cref{fig:applications-maps-LN}). In fact, P2-blue cells frequently formed clusters around B cell-rich germinal-like centres, highlighting a potential clone-specific interaction with the adaptive immune system (middle-right panel \cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-LN}a,d). By contrast, P2-orange regions frequently resided inside the lymph node sinuses that were lined by endothelial cells expressing \gene{CD34} and \gene{PDGFRB} (middle-right panel \cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-LN-hypoxia}). Most of the immune cells in P2-orange regions were myeloid cells with expression profiles consistent with the presence of both M1 and M2 macrophages (\gene{CD163}, \gene{CD68}, \gene{HAVCR2} and \gene{FCGR3A}), and the most highly enriched gene, \gene{CXCL8}, is released by hypoxic macrophages \parencite{Li2015-ng} (bottom panel \cref{fig:applications-maps-LN}). Indeed, relative to P2-blue, it emerges that P2-orange experienced more hypoxic conditions manifesting as higher cancer cell expression of \gene{VEGFA} and necrotic regions (\cref{appfig:appendix-applications-LN-hypoxia}). Hypoxia signatures are associated with adverse clinical outcomes, probably because they reflect the emergence of environments that can select for hypoxia-tolerant clones and/or cancer proliferation rates outstrip neoangiogenesis \parencite{Cairns2004-vs}. Together, these data demonstrate how \ac{BaSISS} clone maps allow one to spatially relate such variation in microenvironments to individual clones.

\section{Discussion}
\label{sec:application-discussion}

Applying \ac{BaSISS} to a series of samples from the key stages of breast cancer progression -- carcinoma \textit{in situ}, invasive cancer and lymph node metastasis -- it is notable that virtually every sample exhibited a spatial organisation of clones, which warrants further investigation in larger cohorts. The fact that nearly all clones examined in this dataset displayed distinct clone-specific gene expression, stromal and immune microenvironments and microanatomical niches highlights the functional relevance of at least some subclonal diversification.

A key observation from these data is the variable relationship between genotype and phenotype. We examined three primary breast tumours with intermixed \ac{DCIS} and invasive disease, and in each case, a clone was identified that existed simultaneously in the two distinct histological states. In contrast, albeit in a single examined case, we observe remarkable consistency between genetic lineage and \ac{DCIS} grade spanning centimetres of tissue. While occasional examples of `upgrading' occur, which could be accounted for by unsampled genetic evolution, we find no evidence of transition to a lower grade. The lack of plasticity within the grade phenotype space is aligned with studies that report that \ac{DCIS} and invasive cancers are usually of the same grade \parencite{Gupta1997-om,Van_Luijt2016-vq}. Our findings could reflect stable differentiation states acquired by divergent cells, prior to \ac{DCIS} onset \parencite{Rakha2022-qm}.

Another key message, that echoes those of a recent spatial study in melanoma \parencite{Nirmal2022-sq}, is that measuring entire tissue sections rather than small fields of view is advantageous for understanding biology. Firstly, this approach reveals that patterns of cancer growth and organisation can vary across different scales. For example, in P1-D1, P1-D2 and P1-D3, at macroscopic scale we see \ac{DCIS} clones co-exist in multiple lobules, across an entire breast lobe, while at the microscopic level they are mainly segregated, distending individual acini or forming abutting populations. There are various possible explanations for the observed \ac{DCIS} growth patterns such as clonal co-operation or genetic drift \parencite{Janiszewska2019-zq, Turajlic2019-sr}. A number of recent mathematical modelling studies have emphasised the importance of tissue architecture in shaping evolution and in the reported case it is certainly plausible that altered rates of fixation and absorption arising from the physical bottlenecks in the different sized ducts and ductules could account for the observed genetic patterns \parencite{Lieberman2005-cy,Noble2022-eg,West2021-ar}. Secondly, within a lymph node the panoramic view allows one to appreciate that subclones repeatedly foster the same ecosystems – one that is immune depleted within the sinuses while the other clustered around B-cell aggregates consistent with a clone specific interactions with the adaptive immune response \parencite{Sharonov2020-vx}. This way of examining tissues naturally leads to a multitude of questions that each warrant follow up in its own extended cohort. The tools we have developed and approaches demonstrated in this study will provide others with a framework for achieving this.

The ability to chart clonal growth patterns and clone-specific genetic underpinnings of the tumour microenvironment is likely to be instrumental in elucidating how different evolutionary processes operate and manifest across different cancer types—or even in histologically normal tissues \parencite{Sottoriva2015-ci}. Understanding the forces of malignant progression, especially invasion and metastasis, and how interactions with the tumour microenvironment shape clinical outcomes \parencite{Risom2022-uw} appear of particular importance. Detailing the functional and microenvironmental characteristics of different clones is also relevant as some part of subclonal diversity in tumours may be due to selectively neutral drift, but the exact extent remains debated.

Particular advantages of the technology are that it is capable of interrogating very large tissue sections on the scale of squared centimetres, which enables studying entire cross-sections of smaller tumours. It is also comparably cheap, unlike solely relying on sequencing-based methods \parencite{Vickovic2019-tz}. The three main limitations of the approach are relatively low sensitivity, which currently precludes single-cell genotyping, a reliance on RNA with the resulting variation in gene expression levels of targeted transcripts, and the fact that clone-defining mutations need to be detected first by separate sequencing-based assays. Greater sensitivity and spatial resolution may be achieved by including more targets per clone and by favouring mutations with higher predicted expression levels, for example, in higher copy number states. A switch to hybridization-based sequencing and direct RNA-binding probes may also improve base-specific detection by several fold \parencite{Gyllborg2020-uq,Lee2022-ha}.

It is often stated that `nothing in biology makes sense except in the light of evolution' \parencite{Dobzhansky1973-va}, which is likely to be true for cancer biology. The ability to spatially locate and molecularly characterize different cancer subclones adds essential features to the spatial-omics toolkit. It provides a robust evolutionary framework that is necessary to interpret the biological relevance of many of the more plastic spatial characteristics of a cancer. Future widespread applications of spatial genomics approaches such as \ac{BaSISS} will uncover how cancers grow in different tissues and allow us to track, trace and characterize the ill-fated clones that are responsible for adverse clinical outcomes.

