\chapter{Applications}
\label{sec:chapter-basiss-applications}

\section{Breast Cancer}

\explain{}{\marginfig{side-plot-cancer-cases-2020.pdf}
Breast cancer was the most prevalent type of cancer, with approximately 2.2 million cases registered in 2020}

In 2020, breast cancer surpassed lung cancer to become the most diagnosed cancer type in the world \parencite{Sung2021-xv}. It accounted for approximately 11.7\% of all diagnosed cancer cases. This trend occurred irrespective of a country's development index, predominantly targeting the female population. Within this demographic, breast cancer constituted nearly a quarter of all diagnoses, and 15.5\% of all cancer-related deaths. An estimated 2.2 million cases were registered in 2020, and this number is expected to grow, partly due to obesity, the ageing population and the fact that females constitute the majority of the older population. Even steady improvements in treatment and early detection are unlikely to reverse this trend, as the deceleration in the decline of female breast cancer death rates appears to be aligned with a consistent increase in incident rates \parencite{Cronin2022-mc}.

\figuretextwidth{applications-breast-structure.pdf}{applications-breast-structure}
    {Mammary gland structure and breast cancer progression}
    {Breast cancer originates from the bilayer epithelial cells lining the branched system of the mammary gland, often specifically within the milk-producing \acf{TDLU}. Initially, these cancer cells divide and fill the lumen of the ducts and lobules, a phase termed carcinoma in situ. Subsequently, these cells may penetrate the basement membrane, invading adjacent tissues in a stage called invasive carcinoma. Eventually, the cancer cells gain the potential to enter the bloodstream and metastasise to distant body sites.}

Breast cancer typically originates within the ductal system of the breast, more specifically from a structure known as the \ac{TDLU}.\footnote{Surprisingly, many scientific papers and even the Dana-Farber Cancer Institute's \href{https://www.dana-farber.org/health-library/articles/what-is-lobular-breast-cancer-}{website} identify the \ac{TDLU} as the  \textbf{sole site of origin for all breast cancers}. This assertion appears unlikely, given that any dividing cell has the potential to become cancerous. The study by \textcite{Tabar2014-ea} suggest that ~ 25\% of cases actually originate in the duct and show no link to \ac{TDLU}.} The \ac{TDLU}, comprised of a terminal duct and multiple acini or lobules, functions as the site for milk production and secretion during lactation. When precancerous cells begin to actively divide (a process known as hyperplasia), they can spread throughout the ductal system. This fills the lumen of the ducts and lobules, a stage referred to as carcinoma in situ. Eventually, these cells may break through the basement membrane, invading the surrounding tissue in what is known as invasive carcinoma. At this stage, the cancer cells have the potential to enter the bloodstream and metastasise to other parts of the body \pcref{fig:applications-breast-structure}.

\subsubsection{Histological and Molecular Classification}
Breast cancer exhibits significant heterogeneity in both histological and molecular characteristics, which directly impact patient survival and treatment options. Histologically, breast cancer classification primarily hinges on the appearance of invasive cancer cells, categorised as either `ductal' or  `lobular'. It's crucial to note that these terms describe the visual resemblance to ductal or lobular tissue and do not indicate the tumour's origin  \parencite{Tabar2022-iq}. Additional factors, such as grade — indicating how closely cancer cells resemble normal cells — and various visual histological features, further refine this classification \parencite{Elston1991-md}.

On the molecular front, initial classification strategies targeted the presence of hormone receptors such as oestrogen (\protein{ER}) and progesterone (\protein{PR} or \protein{PgR}). Given their role in cell growth and differentiation during key life stages like pregnancy and puberty, these receptors became crucial markers for guiding hormonal therapy in breast cancer treatment and continue to guide classification today \parencite{Early_breast_cancer_trialists_collaborative_group1992-dy}.\explain{}{\marginfig{side-plot-ihc2pam-sankey.pdf} Although expression and protein-based systems contain similar clusters, the alignment between them is not precise. Data from \textcite{Kim2019-mz}}Groundbreaking research using transcriptional microarrays, notably by \textcite{Perou2000-hc} and \textcite{Sorlie2001-yy}, deepend the classification of ductal and lobular carcinomas. These studies identified five intrinsic categories based on gene expression patterns: basal-like, normal-breast-like, \protein{ER}\textsuperscript{+} luminal types A and B, and \gene{HER}\textsuperscript{+} groups. This gene-expression-based classification, commonly known as the PAM50 system, has gained wide clinical acceptance and has demonstrated correlations with patient survival \parencite{Wallden2015-bp}. An integrative clustering (IntClust) system emerged, build upon ten recurrent profiles of \acp{CNA} and exrpession \parencite{Curtis2012-hu}. In a clinical setting, a cost-effective surrogate approach often substitutes IntClust and PAM50 systems. This altenrative utilises an \ac{IHC} panel that measures \protein{ER}, \protein{PR}, \protein{Ki-67}, and \protein{HER2} to classify breast cancer into four primary subtypes: Luminal A, Luminal B, HER2-enriched, and \ac{TNBC} \parencite{Goldhirsch2013-xy}. 

\subsection{Ductal epithelium normal heterogeneity}
Clinical classification systems aid in predicting patient outcomes but do not fully capture cancer's heterogeneity. Although significant efforts have been made to understand the molecular evolution of breast cancer, the mechanisms governing its initiation and progression remain unclear.

This heterogeneity partly originates from the cells that give rise to breast cancer. The ductal epithelium, a key structure, comprises two distinct cell types: luminal (\protein{KRT8, KRT18}) and basal (\protein{KRT5, KRT14}). Luminal cells form the inner layer and are integral for milk production and duct formation. Basal cells, making up the outer layer, facilitate the contractile function necessary for milk release.

During developmental stages such as puberty, and physiological processes like pregnancy and lactation, hormonal regulation induces significant remodelling in the epithelium \parencite{Macias2012-su}. This suggests the presence of enduring progenitor cell lines within the tissue or a general proliferative capability. Evidence points to a developmental hierarchy among mammary gland cells \parencite{Skibinski2015-rh}. This hierarchical pattern, commonly observed in various tissues, tends to mitigate the risk of cancer development as discussed in \cref{sec:intro-tissue-organisation}.

The prevailing hypothesis postulates that a basal-like bipotent progenitor stem cell (\protein{KRT14}) within the mammary tissue can differentiate into both luminal and basal cells \parencite{Stingl2001-cb,Prater2014-qd,Rios2014-jj}. This bipotent progenitor may also give rise to unipotent progenitors, which can differentiate into either luminal or basal cells exclusively \parencite{Van_Keymeulen2011-um,Rios2014-jj,Tao2014-ol}.\explain{}{\marginfig{side-plot-breast-hierarchy.pdf} Continuous lineage hiearchy from \textcite{Nguyen2018-jl}} Nevertheless, these findings predominantly come from mouse models and in vitro studies. The existence of such a strict hierarchy and the role of the bipotent progenitor in human mammary glands remain subjects of ongoing debate \parencite{Skibinski2015-rh}. For example \textcite{Prater2014-qd} show that as many as 65\% of basal cell have prolifirative capability and can effectively form ductual-lobular structures \emph{in vivo}. Single-cell transcriptional trajectory analysis reveals a significant proportion of \protein{KRT14+} basal cells that serve as a transcriptional pivot point between luminal and specialised myoepithelial cells. This analysis also indicates the existence of two distinct transcriptional lineages within luminal cells: the `hormone-responsive' and the `secretory' types \parencite{Nguyen2018-jl}. The equipotency of basal cells in the mammary gland is striking, particularly because it elevates the risk of cancer formation \pcref{sec:intro-tissue-organisation}. One plausible explanation for this characteristic could be the requirement for rapid remodelling of the mammary gland system with each successive pregnancy. As mammals evolved, the need for tissue renewal may have taken precedence over cancer protection. We may now be facing the consequences of this evolutionary trade-off, as breast cancer has become the most prevalent type of cancer globally, even though it primarily affects only half of the human population.

Irrespective of the precise mechanisms underlying cancer genesis, transcriptional parallels are evident between luminal cells and luminal breast cancer, as well as between basal cells and basal-like intrinsic cancer types \parencite{Bhat-Nakshatri2021-jy}. Although some hypotheses suggest that the cell of origin dictates the phenotype \parencite{Skibinski2015-rh, Taurin2020-mq}, the imbalance of mutation frequencies among clinical subtypes \parencite{Cancer_Genome_Atlas_Network2012-gx,Russnes2017-eo} and known tumour transcriptional plasticity \parencite{Fan2020-vi, Su2015-ve,Yamamoto2014-th,Hein2016-lv} favour the view that the nature of the oncogenic factor significantly shapes the tumour's phenotype.

\subsection{Genomic landscape} 

The first definitive association between recurrent genetic alterations and breast cancer came with the discovery of the tumour suppressor genes \gene{BRCA1} (located on chromosome 17q21) and \gene{BRCA2}\explain{}{\marginfig{side-plot-brca-road.pdf} Cycling path in Cambridge\footnotemark with the entire sequence of \gene{BRCA2}} (on 13q12-13) \parencite{Wooster1994-xa, Hall1990-mg}. These genes play a crucial role in DNA repair, especially in epithelial cells, and are notably expressed in the mammary gland during development and pregnancy. Advances in genomic tools have enabled large-scale, systematic genomics analyses of thousands of tumours. These studies have revealed intricate patterns of mutations across different subtypes of breast cancer.

\footnotetext{My processing speed is approximately 60 bp/s, as I can slide along 10,257 base pairs of \gene{BRCA2} sequence in 2 minutes and 47 seconds. For comparison, \emph{Taq} polymerase operates at a rate of 150 bp/s, while the high-fidelity \emph{Pfu} polymerase works at a considerably slower pace of 15 bp/s. I deeply thank Lukas Weilguny for consistently providing slipstream support along the way.}

\figuretextwidth{applications-driver-pathways.pdf}{applications-driver-pathways}{Main driver genes and signalling pathways in breast cancer}{Comprehensive early genomics studies have identified key driver genes associated with breast cancer. These genes primarily target four main pathways: Receptor Tyrosine Kinases, PI(3)K/Akt, p53 signalling, and cell cycle checkpoint regulators. Mutations in genes contolling these pathways results in: cell survival through inhibition of apoptosis, proliferation, dedifferentiation and faults in DNA reparation. On the figure red indicates gain-of-function mutations or amplifications; blue represents loss-of-function mutations or deletions.}

Several early comprehensive genomics studies identified thousands likely driver mutations, spanning hundreds of cancer-related genes and several non-coding regions \parencite{Shah2009-xz, Cancer_Genome_Atlas_Network2012-gx,Shah2012-xz, Nik-Zainal2016-ek,Banerji2012-as,Ciriello2015-ey,Pereira2016-ov, Curtis2012-hu}. The most commonly mutated genes as found in \textcite{Cancer_Genome_Atlas_Network2012-gx} are \gene{TP53} (43\%), \gene{PIK3CA} (32\%), \gene{MYC} (20\%), \gene{CCND1} (16\%), \gene{PTEN} (16\%), \gene{ERBB2} (14\%), Chr8:(\gene{ZNF703}/\gene{FGFR1}) (12\%), \gene{GATA3} (10\%), \gene{RB1} (9\%), \gene{MAP3K1} (8\%). These genes predominantly participate in cell cycle regulation, reparation and growth factor signalling \pcref{fig:applications-driver-pathways}. However, most mutations in cancer-associated genes are infrequent, leading to significant genetic diversity among patients.

\explain{}{\marginfig{side-plot-apobec.pdf} \protein{APOBEC} is a cytidine deaminase. Under normal conditions it serves as a cellular defence against viral infections by mutating viral ssRNA and ssDNA. In cancer, dysregulated \protein{APOBEC} mutates ssDNA in the nucleus, causing a high frequency of C>T mutations}

Breast cancer mutagenesis encompasses a wide range of mechanisms. Chromosomal rearrangements are common and occur during extended periods of chromosomal instability \parencite{Curtis2012-hu, Gerstung2020-sg,Nik-Zainal2012-zz}. These chromosomal rearrengements are connected to the signatures strongly associated with \gene{BRCA1/2} and other DNA double-strand break repair pathways. Additionally, the \protein{APOBEC} signature is also prevalent \parencite{Nik-Zainal2016-ek, Banerji2012-as, Nik-Zainal2012-vo}. This signature causes a high frequency of C>T mutations, sometimes exhibiting focal hypermutation patterns, known as kataegis, in approximately half of the breast tumours. Given the diversity of driver genes and the extended period of mutagenesis, breast cancer appears to be driven by a cumulative effect of multiple accumulated drivers.

Notably, the frequency of driver mutations varies among breast tumour types. The \textbf{Luminal class} commonly features somatic mutations that activate the PI3K-AKT signalling pathway (\gene{PIK3CA}, \gene{PTEN}) and inactivate the GATA3 and JUN kinase (\gene{MAP3K1}) pathways. Tumour suppressors \gene{TP53} and \gene{RB1} remain largely intact in Luminal A cancers but are often inactivated in the more aggressive Luminal B subtype \parencite{Cancer_Genome_Atlas_Network2012-gx}.

In the \textbf{HER2-enriched class}, there is frequent high copy number amplification of \gene{ERBB2}, which accounts for elevated levels of \protein{HER2} protein expression. This amplification often co-occurs with amplification of the nearby gene, \gene{GRB7}. A systematic analysis of this group revealed subclusters that resemble luminal and basal types in both gene expression and mutation distribution. This suggests that rather than constituting an entirely independent intrinsic type, this group represents a composite of the broader breast cancer spectrum, further complicated by \gene{ERBB2} amplification \parencite{Ferrari2016-qj}.

The \textbf{\ac{TNBC} class} aligns closely with the basal-like intrinsic expression subtype, with approximately 75\% of \ac{TNBC}s being basal-like. This class exhibits a high frequency of TP53 mutations, especially among basal-like types, as well as a high level of \ac{CNA}s \parencite{Shah2012-xz, Cancer_Genome_Atlas_Network2012-gx}. This coincides with frequent inactivation of \gene{BRCA1} and \gene{BRCA2} genes, more so than in other subtypes. However, the mutation landscape of \ac{TNBC} is highly diverse overall. Deep sequencing of \ac{TNBC} reveales variable clonal frequencies at the time of diagnosis \parencite{Shah2012-xz}. This analysis identified numerous subclonal mutations targeting cytoskeletal genes, underscoring the extended and variable evolutionary history of this subtype.
\explain{}{\marginfig{side-plot-ductal-lobular.pdf} Ductal and lobular breast cancer are two histological subtypes differentiated primarily by the presence of a single mutation in the \gene{CDH1} gene}
Classification into \textbf{ductal} and \textbf{lobular} phentoypes also demonstrates a strong genomic basis. The hallmark feature of the lobular phenotype, the loss of \protein{E-cadherin} protein, correlates with mutations in the \gene{CDH1} gene in nearly all lobular cases \parencite{Ciriello2015-ey}. This pattern persists in both pure and mixed types, the latter being a clear blend of the two distinct genetic lineages.

Overall, bulk genomic studies indicate that the breast cancer genome exhibits significant diversity. This diversity is shaped by the evolutionary history of the tumour. While some patterns and dominant evolutionary trajectories are visible, they are often not clear-cut. The presence of mixed phenotypes within a single tumour and observed clonal and subclonal diversity at the time of diagnosis further complicate the landscape \parencite{Ciriello2015-ey,Pereira2016-ov,Shah2012-xz}. For a comprehensive understanding of the biology and therapeutic responses of patients, a more nuanced grasp of the tumour's evolutionary history at the subclonal level is essential.

\subsection{Clonal evolution}

Progress in whole-genome sequencing (WGS), enhanced multiregional tissue sampling techniques, and the advent of advanced computational tools collectively enable more intricate studies of breast cancer evolution \pcref{box:technologies}. First studies relied on \ac{CNA} inferred from single-cell sequencing that allowed to reconstruct phelogenies. They revealed that breast cancer is in fact a patchwork of clones, a result of a series of `punctuated' clonal expantions, rather than a linear gradually developing disease \parencite{Navin2011-qq,Wang2014-bp,Gao2016-qv}. These studeis also pointed out that clonal expansions are linked to focal \ac{CNA} events, while \acp{SNV} seem to accumulate in a gradual manner. However, these studies were limited by the low sensitivity of single-cell sequencing, and ambiguity of tree reconsturction from copy number profiles, without the use of point mutaitons.

\textcite{Yates2015-xk} conducted the first multiregional targeted sequencing of breast cancer. While the study was not highly sensitive, it revealed variable genomic diversity across samples. Most cases exhibited long trunk mutations, but the study also reported instances of early branching and extensive parallel evolution. Notably, mutations in the \gene{PTEN} and \gene{TP53} genes showed signs of convergent evolution, occurign independently across parallel branches. This study further identified that complex chromosomal events can happen at any evolutionary stage, both early and late, potentially reshaping the genome and contributing to phenotypic diversity at clonal and subclonal levels.

\figuretextwidth{applications-evolution-tree.pdf}{applications-evolution-tree}
{Highly branched parallel evolution of breast cancer described by \textcite{Nishimura2023-mk}}
{Deep study of breast cancer case reveals complex, highly branched evolutionary path. Key driver events, specifically der(1;16) translocations, emerged independently in two branches when the patient was around 6 and 10 years old. Although only one branch led to an invasive phenotype, parallel driver events occurred across multiple branches. The complex relationship between genome and phenotype becomes evident from the mismatch between the number of driver events and the resulting phenotype. This suggests a potential role for the microenvironment in disease progression. Notably, a single \gene{CDH1} gene mutation led to the formation of a lobular phenotype within a primarily ductal carcinoma}

Enhanced spatial resolution in sampling strategies, facilitated by techniques like tissue microdissections, enables more precise inquiries into breast cancer development \pcref{box:technologies}. In the study by \textcite{Casasent2018-gx}, \ac{LCM} isolated cells from both ductal and adjacent invasive regions of ductal carcinomas. Contrary to previous assumptions about breast cancer invasion, identical clones were found both inside and outside the ducts without significant changes. This suggests that the evolutionary changes necessary for invasion occur within the ducts and are initially contained by them. Another microdissection study paired with single-cell DNA sequencing by \textcite{Lips2022-kv}, which also employed single-cell DNA sequencing, concentrated on invasive recurrence. The study linked this recurrence to untreated clones that had been present for an extended period within \ac{DCIS}.

The more comprehensive the microdissection strategy and study depth, the greater the number of parallel branches observed in the reconstructed phylogenetic trees. A detailed study by \parencite{Nishimura2023-mk} focused on the early evolution of \ac{DCIS}. The research revealed that many breast cancers exhibited a recurring chromosomal abnormality, der(1;16)\explain{}{\marginfig{side-plot-der1-16.pdf} Unbalanced translocation between Chr1 and Chr16 was frequently observed in study by \textcite{Nishimura2023-mk}}, acquired early in cancer evolution, roughly around puberty. These der(1;16)-positive clones expanded across large breast regions, spawning multiple independent cancer branches as well as non-cancerous lesions \pcref{fig:applications-evolution-tree}. Conversely, most der(1;16)-negative clones remained restricted to single lobules post-puberty. The findings imply that a branching, multifocal model of cancer progression might be more prevalent than previously thought linear models. Additionally, the study found no correlation between the number of driver events and histology, indicating some influence of local microenvironments and potentially epigenetic drivers in cancer development.

\subsection{Microenvironment}

Single-cell proteomics and transcriptomics studies have substantially advanced our understanding of the tumour microenvironment in breast cancer, going well beyond traditional bulk PAM50 transcriptional classification \parencite{Wu2021-uq,Pal2021-rf,Wagner2019-zp}. These investigations confirm that the general structure of the breast cancer microenvironment closely aligns with the broader cancer landscape, as outlined in \cref{sec:mapping-the-cancer-ecosystem}. The microenvironment commonly comprises immune cells (both myeloid and lymphoid), stromal cells (including \acfp{CAF}, endothelial cells, \acfp{PVL} and adipocytes), and epithelial cells (luminal and myoepithelial). However, breast cancer exhibits significant heterogeneity in several aspects: relative proportions of cell types, the presence of niche cell subtypes, and the cellular communities associated with patients survival \parencite{Jackson2020-em, Danenberg2022-zb}.

\subsubsection*{Immune cells}

In the context of cancer, immune cells are often simplistically categorised as either tumour-promoting, such as regulatory T cells (Tregs) and M2 macrophages, or tumour-inhibiting, such as cytotoxic CD8+ T cells, natural killer (NK) cells, CD4+ T-helper cells, and M1 macrophages \pcref{fig:intro-tme-composition}. Generally, the immune system acts as a tumour suppressor, and the presence of immune cells in the tumour microenvironment is linked to a better prognosis \pcref{fig:intro-tme-composition}. However, lymphocytes can enter a so-called 'exhausted' dysfunctional state, characterised by the expression of co-inhibitory receptors such as \protein{CTLA-4}, \protein{PD-1} and \protein{LAG3}.

Despite these general classifications, the landscape of immune cell roles in cancer remains complex. Variability exists across studies in the identification of cell subtypes and their roles, sometimes leading to contradictions. For instance, while NK cells generally target tumour cells, some studies suggest they might also enhance tumour vascularisation and even exert immunosuppressive effects \parencite{Retecki2021-se}. This diversity in findings indicates that the immune landscape in cancer is more intricate than often portrayed.

In breast cancer research, studies show a continuous and complex spectrum of T-cell activation states \parencite{Azizi2018-vc}. This heterogeneity sharply differs from normal tissue and could account for varying clinical outcomes in patients by the abundace of certain cell states \parencite{Azizi2018-vc,Savas2018-vb}. T-cell diversity largely stems from variations in T-cell receptor (TCR) and antigen-presenting cell heterogeneity \parencite{Azizi2018-vc}. Such variability may result from differing tumour antigens across clonal populations.

Compared to T cells, myeloid cells display more distinct activation states. However, a single cell can simultaneously express both M1 and M2 programs \parencite{Azizi2018-vc}. A correlation exists between immunosuppressive T cells, PD-L1+ macrophages, and high-grade tumours \parencite{Wagner2019-zp}.

Spatial analysis of immune cell distribution in breast cancer indicates non-uniform patterns linked to prognosis \parencite{Danenberg2022-zb}. For instance, dysfunctional T cells often cluster near regulatory T cells (Tregs), which are believed to control their proliferation and activation. Large dysfunctional T-cell aggregates may signify tumour cells resistant to ongoing immune attacks, thereby chronically stimulating cytotoxic T cells. Conversely, structures containing antigen-presenting cells, macrophages, and T cells associate negatively with survival. In contrast, cell communities featuring granulocytes, and particularly \acf{TLS} with B cells, correlate with better survival outcomes.

\subsubsection*{Stroma}

In the stroma, fibroblasts are the most extensively studied cells. Researchers typically categorise these fibroblasts into two broad types: SMA- fibroblasts and SMA+ myofibroblasts \parencite{Costa2018-ir}. While efforts exist to categorise more types, these efforts face challenges due to the cell-type fluidity of fibroblasts \parencite{Cords2023-og,Wu2021-uq}. These cells serve essential structural functions and engage in immune cell recruitment and extracellular matrix remodelling.

Of specific interest is the \protein{SMA+} \protein{FAP+} subset of fibroblasts, which acts as a crucial immunosuppressive agent. This subset attracts T lymphocytes and facilitates their differentiation into immunosuppressive Tregs, thereby aiding in immune evasion in cancer. In contrast, \protein{SMA+} \protein{FAP-} fibroblasts lack this immunosuppressive function \parencite{Costa2018-ir}. Spatial distribution analysis indicates that myofibroblasts frequently reside at the tumour-stromal interface in breast cancer, which suggests their role in lymphocytic exclusion \parencite{Danenberg2022-zb}. Although this seems to promote tumour growth, other fibroblasts located near tertiary lymphoid structures (TLS) correlate with a better prognosis \parencite{Danenberg2022-zb, Cords2023-og} \pcref{fig:intro-cancer-neighbourhoods}. 

In a different spatial study, \textcite{Risom2022-uw} demonstrated that `desmoplastic' stroma, characterised by a high frequency of fibroblasts and intense collagen deposition around ducts, correlates with a favourable prognosis and reduced likelihood of progression to invasive cancer. Notably, this type of stroma coexists with a thin and discontinuous myoepithelial layer. This type of stroma may indicate a robust immune response, facilitated by a permeable myoepithelium that exposes the cancer cells to immune surveillance.

Endothelial cells and \acp{PVL} are other significant components of the tumour microenvironment. These cells constitute the vasculature, critical for tumour growth and metastasis. In the context of breast cancer, a dense vascular stroma correlates with a poor prognosis \parencite{Danenberg2022-zb}.

\figuretextwidth{applications-epithelial-plasticity.pdf}{applications-epithelial-plasticity}
    {Immune niches and breast cancer cell plasticity}
    {Expetimental design from \textcite{Sinha2021-mf}. A lentivirus vector delivers \protein{acERBB2} into mammary gland cells, inducing the formation of \acf{DCIS}. The \ac{DCIS} cells stochastically adopt one of two phenotypes: 'indolent' or 'aggressive'. The 'indolent' phenotype features a luminal-like state and the presence of T cells, whereas the 'aggressive' phenotype is characterised by a basal-like state and high neutrophil infiltration. Intriguingly, reimplantation of cells from both phenotypes reveals functional indistinguishability, as they produce similar ratios of 'aggressive' and 'indolent' phenotypes in a new host. This experiment suggests that cells with equivalent oncogenic potential can exist in diverse phenotypic states. Additionally, neutrophil inhibition via anti-\protein{IL17} treatment reduces the likelihood of the 'aggressive' phenotype emerging, suggesting their role in promoting the 'aggressive' phenotype.}


\subsubsection*{Cancer Epithelial plasticity}

As previously discussed, the breast ductal system's epithelium comprises luminal and basal cell types, which exhibit significant plasticity. In cancer, this epithelial plasticity not only persists but also amplifies, leading to a diverse range of transcriptional states \parencite{Wagner2019-zp, Wu2021-uq, Pal2021-rf}. The study by \textcite{Wu2021-uq} dissected the intrinsic expression profiles of breast cancer cells into various transcriptional modules, each showing distinct patterns. Notably, the \acf{EMT} and proliferative modules appeared mutually exclusive. This observation corroborates the incompatibility between a mesenchymal-like state and proliferation in breast cancer \parencite{Tsai2012-hb}. However, despite the coexistence of multiple tumour cell phenotypes within all tumour ecosystems, one phenotype often dominates, potentially reflecting the expansion of the fittest tumour subclones \parencite{Wagner2019-zp}.

The extent to which breast cancer plasticity influences invasion has been explored by \textcite{Sinha2021-mf} through a mouse model study \pcref{fig:applications-epithelial-plasticity}. In this research, cancer cells sharing the same genetic background — specifically, activated \gene{ERBB2} — gave rise to two distinct types of \ac{DCIS} lesions within the same animal: `indolent' and `aggressive'. Importantly, cells from both lesion types demonstrated equal ability to initiate invasive tumours. The key differences between these two lesion types lay in their intrinsic phenotypes and surrounding immune environments. The `aggressive' lesions contained more basal-like cells and macrophages, while the `indolent' lesions were enriched for luminal cells and T cells, hinting that local environmental factors influence the tumour's invasive potential.

\subsection{Metastasis}

As discussed in introductory \cref{sec:intro-metastasis}, metastasis involves a sequence of events. Initially, cancer cells detach from the primary tumour becoming \acf{DTC}. Subsequently, these cells colonise and proliferate in distant tissues. \ac{DTC} often undergo \ac{EMT} which facilitates dissimination through the loss of cell-cell contacts. Once they reach the metastatic site, they typically revert to their original epithelial morphology. This sequence of events is associated with a poor prognosis. In the context of breast cancer, metastatic disease is generally considered incurable and often results in death within a median survival period of less than three years \parencite{Harbeck2019-jx}. Furthemore, breast cancer display clear organotropism; the most frequent sites for metastasis, apart from axillary lymph nodes, are bones, lungs, and liver \parencite{Nguyen2022-jr}.

Metastasis in breast cancer occurs late in the tumour's evolutionary history, as indicated by phylogenetic analyses. Cells from the primary tumour that seed metastases or relapses acquire additional mutations not present in the primary tumour. These mutations frequently disrupt SWI/SNF and JAK-STAT signalling pathways and generally target an expanded repertoire of cancer genes \parencite{Yates2017-xc}. Intriguingly, certain oncogenic mutations, such as those in the \gene{CDH1} and \gene{CBFB} genes, appear to reduce the risk of metastasis \parencite{Nguyen2022-jr}. 
\explain{}{\marginfig{side-plot-metastasis-breast.pdf} While metastasis to the auxilary \ac{LN} is common, it doesn't seem to directly relate to the formation of distant metastases. Whether ongoing evolution in the \ac{LN} influences the primary tumour site is currently unknown.}
Differences exist between the genetic makeup of synchronous \acf{LN} metastases and distant metastases. The former are genetically very similar to the primary tumour, while the latter contain substantially more mutations - on average, 63\% more than the primary tumour. This increase in mutation rate over time suggests accelerated tumour evolution during progression to distant metastasis \parencite{Yates2017-xc}. In contrast, \ac{LN}s seem to have a passive role in the initiation of distant metastases. Although they hold significant prognostic value, they are likely to be an evolutionary dead-end. Resection of the \ac{LN} does not show a correlation with patient survival \parencite{Fisher1977-ua}. \ac{LN} metastases indicate cancer cells' ability to survive and grow in other organs, rather than directly aiding their dissemination \parencite{Ullah2018-xe}. This is surprising because \ac{LN} metastases exist in an environment abundant with immune cells. While these metastases may not directly advance to distant organs, they could modulate the immune response in the primary tumour, indirecrly influencing breast cancer progression.

Little is known about the precise mechanisms underlying the formation of \acp{DTC} and the role of \ac{TME} in this process. Single-cell profiling of primary and \ac{LN} metastatic sights showed general patterns of immune downregulation in the \ac{LN} locations, aligned with the broad concept of the pre-metastatic niche  \parencite{Liu2022-mt}. These niches generally exhibit inflammation and immunosuppression, facilitated by factors such as \protein{CCL2} and \protein{VEGF}, and involve the recruitment of myeloid cells, thus increasing the likelihood of metastatic growth in target organs \parencite{Peinado2017-hz}. Mouse model studies emphasise the potential role of neutrophils in accompanying \acp{DTC} within the bloodstream  \parencite{Szczerba2019-mt}. These studies also suggest that the \ac{TME} at the primary site may facilitate polyclonal seeding to distant locations \parencite{Cheung2016-nb}.

\subsection{Open questions on breast cancer progression}

Despite considerable advancements in breast cancer research, our understanding of disease progression and underlying mechanisms remains incomplete. We have identified key driver mutations and broadly understand the mutation processes that lead to their accumulation. We are familiar with the basic composition of the breast cancer microenvironment and the cellular communities typically associated with unfaivourable clinical outcomes. Yet, a comprehensive understanding of how genomes and environment interact and influence each other eludes us. Our current models of cancer progression are simplistic, primarily because few studies have rigorously examined the phylogenetic structure of patients' breast cancer and even fewer have simultaneously characterised both genetic and \ac{TME} components. As a result, several critical questions about cancer progression remain unclear:

\begin{itemize}
    \item What are the cells of origin, and to what extent do they dictate the future trajectory of cancer evolution?
    \item Does the environment play a role in the initial onset, or are the initiating factors solely genetic?
    \item Why do some clones successfully colonise ducts while others do not?
    \item What types of environments trigger invasion, and what genetic backgrounds sustain clones in invasive lesions?
    \item For metastasis, what role does the environment in the primary tumour play, and what is the role of the environment at distant sites?
    \item Does the evolution occurring in the lymph node impact the primary tumour sites?
\end{itemize}

While the spatial genomics technologies outlined in \cref{sec:chapter-basiss-model} may not offer immediate, definitive answers to these questions, they do provide a framework for further investigations. Specifically, \ac{BaSISS} approach allows for the concurrent profiling of phylogeny and microenvironment across extensive tissue sections. In this chapter, I apply this approach to a cohort of breast cancer patients to investigate the relationship between genetic and microenvironmental evolution in breast cancer progression tracing the evolutionary history of the disease from the earliest stages of tumour development to metastasis.

\section{Results}
\subsection{Two cases of multifocal breast cancer}

The cohort includes eight tissue blocks from two patients (P1 and P2) who underwent a surgical mastectomy for a multifocal breast cancer. These patients were selected to permit a comparison between genetic and histological progression models in early breast cancer development \parencite{Cowell2013-du} \pcref{fig:applications-cohort,fig:applications-breast-structure}. P1 had two separate oestrogen receptor (ER)-positive, human epidermal growth factor receptor 2 (HER2)-negative \acfp{PBC} within a 5-cm bed of \ac{DCIS}; we used tissue blocks from both \acp{PBC} (samples P1-ER1 and P1-ER2) and three regions from \ac{DCIS} (samples P1-D1, P1-D2 and P1-D3). P2 had two separate PBCs of the `triple-negative' subtype (lacking the ER, progesterone receptor and HER2). We sampled both PBCs (samples P2-TN1 and P2-TN2) and an axillary lymph node that contained metastatic cancer deposits (sample P2-LN1) \pcref{fig:applications-cohort}.

\figuretextwidth{applications-cohort.pdf}{applications-cohort}{Breast cancer cohort description}{The two cases of multifocal primary breast cancer (PBC) used to develop the \ac{BaSISS} approach. Coloured tiles report the histological features within each sample and the experiments performed. The number of clones identified by WGS and targeted by BaSISS are reported as white numerals. Notations are: NST - No special type, ER - oestrogen receptor positive, TN - triple negative, D - ductal carcinoma \textit{in situ}, LN - lymph node.}

\subsection{Charting histogenomic relationships}

Histology-driven sampling of well-defined stages of cancer progression can uncover mechanisms and markers of disease progression \parencite{Risom2022-uw,Casasent2018-gx,Cowell2013-du,Nirmal2022-sq}. Up to two-thirds of \acp{PBC} contain both invasive cancer and intermixed \ac{DCIS}, a non-obligate precursor lesion. How these distinct ‘stages’ of cancer development might relate to genetic diversification within the same tissue is generally unknown \parencite{Kole2019-hl} \pcref{fig:applications-breast-structure}. To demonstrate that \ac{BaSISS} can chart these relationships across entire tissue sections, we examined three PBC samples with intermixed invasive and \ac{DCIS} histology: P1-ER1, P1-ER2 and P2-TN1 (\cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2} and \cref{appfig:appendix-applications-PBC-P1,appfig:appendix-applications-PBC-P2}a-c).

\figurefloat{applications-maps-PBC-P1.pdf}{applications-maps-PBC-P1}{Genetic clones mapped in histological context from two \ac{PBC} samples in P1}{\textit{\textbf{top}}, \ac{BaSISS} maps of two \acp{PBC} from P1 with intermixed \ac{DCIS} and invasive cancer. The most prevalent genetic clone is projected as a coloured field (corresponds to \textit{\textbf{middle}} section) on DAPI images (reported if the \ac{CCF} > 25\% and the inferred local cell density > 300 cells per mm\textsuperscript{2}). Pie charts report the \ac{WGS}-estimated clone compositions. Inset images are regions of P1-ER2 (H\&E-stained serial tissue sections) that represent three histological progression states. \textit{\textbf{middle}}, The phylogenetic tree was inferred from P1 multiregion \ac{WGS}. Branches and nodes are coloured to reflect the clone maps. Heatmaps report clone composition in 34 and 44 histologically annotated epithelial-containing microregions of P1-ER1 and P1-ER2, respectively. Microregions include individual ducts or randomly selected regions of invasive cancer. \textit{\textbf{bottom}}, \ac{IHC} per P1-ER1 (left) and P1-ER2 (right) for the proliferative marker \protein{Ki-67} in six clone territories (indicated by contour colour); the percentage of nuclei staining positive (brown) is reported. Notations are: HP - hyperplasia; N - normal ducts. Scale bars, 250 $\mu$m for microregions and 2.5 cm for clone maps.}

\figurefloat{applications-maps-PBC-P2.pdf}{applications-maps-PBC-P2}{Genetic clones mapped in histological context from the \ac{PBC} sample in P2}{\textit{\textbf{top}}, As in top panel \cref{fig:applications-maps-PBC-P1}, but a clone map of P2-TN1. Mini-images report \ac{ISS}-derived cell types (right) and H\&E tissue section snapshots of the two cancer growth patterns (GP1 and GP2) reported in P2-TN1 (left). \textit{\textbf{bottom}}, Phylogenetic tree for P2 and heatmap of 36 P2-TN1 microregions, as in middle panel \cref{fig:applications-maps-PBC-P1}. Branches relating to clones not detected in this sample (that is, only found in P2-LN1) are shaded grey. The bottom heatmap is the estimate by the histopathologist and reports the contribution of different growth patterns to the microregion, defined by distinct nuclear and architectural features. Scale bar, 2.5 mm.}

\ac{BaSISS} detected 2–4 subclones per \ac{PBC} in accordance with bulk \ac{WGS} data. Clone maps (top panels \cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2}) and the quantitative clonal composition of 73 individually annotated microregions (middle panels \cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2}, \cref{appfig:appendix-applications-PBC-P1}a,b and \cref{appfig:appendix-applications-PBC-P2}a,b) revealed that individual subclones form spatial patterns that were, by varying degrees, related to the histological progression states. Normal tissue elements, including immune aggregates and histologically normal ducts, appear unstained consistent with a wild-type status for the targeted clones (green and yellow contours, respectively; \cref{fig:applications-maps-PBC-P1}). In P1-ER2, an area of hyperplasia was predicted and confirmed by \ac{LCM}–\ac{WGS} to be genetically unrelated to the cancer (blue contour; bottom panel \cref{fig:basiss-lcm-validation} and top panel \cref{fig:applications-maps-PBC-P1}).

In each \ac{PBC}, the genetic and histological progression models were broadly consistent, in which the invasive disease was mainly composed of cells from the most recently diverged subclone: P1-red, P1-purple and P2-purple in samples P1-ER1, P1-ER2 and P2-TN1, respectively (middle panel \cref{fig:applications-maps-PBC-P1}, bottom panel \cref{fig:applications-maps-PBC-P2}). By contrast, earlier diverging clones colocalized entirely or in part to the histological pre-invasive lesion: \ac{DCIS}. For example, in P1-ER2, \ac{BaSISS} predicted that green branch mutations were completely absent from the invasive compartment, a conclusion that is supported by three separate microdissections (\ac{LCM}–\ac{WGS}) from distant regions of invasive cancer in P1-ER2 (bottom panel \cref{fig:basiss-lcm-validation} and \cref{appfig:appendix-applications-PBC-P1}c).

However, in each \ac{PBC}, there was a subclone that spanned both \ac{DCIS} and invasive histology, revealing that disconnects between histological and genetic progression states can exist. This was the case for clone P1-red in P1-ER1 and clone P1-purple in P1-ER2. These \ac{DCIS}-invasive spanning clones could be distinguished from each other by hundreds of private mutations, including different inactivating driver mutations in \gene{PTEN}, indicating parallel evolution along these divergent lineages that resulted in two distinct instances of cancer invasion (total mutation numbers label the phylogenetic tree branches; middle panel \cref{fig:applications-maps-PBC-P1}). The spatial predictions of the \ac{BaSISS} model of intraductal acquisition of \gene{PTEN} mutations and \gene{PTEN} protein loss was confirmed by \ac{LCM}–\ac{WGS} and \ac{IHC}, respectively (bottom panel \cref{fig:basiss-lcm-validation} and \cref{appfig:appendix-applications-PBC-P1}d). In sample P2-TN1, the only predicted driver point mutation was a deleterious mutation in the tumour suppressor gene \gene{TP53}, and this was detected in both \ac{DCIS} and invasive compartments and was also present in all cancer regions of the second \ac{PBC}, P2-TN2, consistent with an early onset in the development of this cancer (phylogenetic tree; \cref{fig:applications-maps-PBC-P1,fig:applications-maps-PBC-P2}). These data therefore suggest that many, if not all, of the genetic events necessary to initiate the invasive transition in these three cancers were acquired within the ducts, and subsequently both intraductal expansion and stromal invasion ensued.

\subsection{Phenotypic changes accompany progression}

Next, by integrating additional layers of spatial data, we sought to establish how phenotypic changes relate to genetic-state and histological-state transitions. In P1-ER1 and P1-ER2, consistent with a more proliferative phenotype, \gene{PTEN}-mutant clone regions exhibited denser \protein{Ki-67} \ac{IHC} nuclear staining, than \gene{PTEN} wild-type ancestral clone regions (\ac{FDR} = 0.004 P1-red versus P1-orange; and \ac{FDR} = 0.03 P1-purple versus P1-green) (bottom panel \cref{fig:applications-maps-PBC-P1} and \cref{appfig:appendix-applications-PBC-P1}e). However, for a given genetic clone, the \protein{Ki-67} score was similar irrespective of whether it occupied a \ac{DCIS} or invasive state, indicating that upregulation of \protein{Ki-67} is temporally related to acquisition of a \gene{PTEN} mutation and precedes invasion.

By contrast, cellular resolution spatial transcriptomics analysis of P1-ER2 revealed that epithelial cell expression of several genes—\gene{CLDN4} (encoding claudin 4), \gene{ACTB} (encoding $\beta$-actin), \gene{KRT5} (encoding keratin 5) and \gene{CTSL2} (encoding lysosomal cysteine protease cathepsin V)—differed between \ac{DCIS} and invasive compartments occupied by the same, P1-purple, clone (\cref{appfig:appendix-applications-PBC-P1}f). These transcriptional changes might therefore be considered more closely linked to the histological transition rather than genetic changes traced by this approach. Expression of \gene{CLDN4} was consistently lower in the invasive compartment than to each \ac{DCIS} clone. However, for some genes such as \gene{ACTB,} expression patterns changed in opposing directions in the invasive cancer relative to the sampled \ac{DCIS} clone (expression is higher than P1-green \ac{DCIS} (\ac{FDR} = 0.02) and lower than P1-purple \ac{DCIS} (\ac{FDR} = 0.013)) or were highly specific to a genetically more distant \ac{DCIS} clone (\cref{appfig:appendix-applications-PBC-P1}f).

Attempts to isolate the changes associated with invasive transition might also be confounded by heterogeneity within the invasive compartment. In P2-TN1, we therefore sought to examine whether the two genetically distinct invasive subclones (P2-blue and P2-purple) were phenotypically distinct. The two cancer clones exhibited distinct morphological (nuclear and architectural) features ($P$ = 0.04, Fisher’s exact test) (H\&E image insets; \cref{fig:applications-maps-PBC-P2}) and occupied neighbourhoods with different stroma (\ac{FDR} = 0.02) and immune cells such as myeloid cell densities (\ac{FDR} = 0.08) (mini-image insets; top panel \cref{fig:applications-maps-PBC-P2} and \cref{appfig:appendix-applications-PBC-P2}a–c). Transcriptional programs were also distinct, with statistically significant differences in gene expression for 12 of 91 genes between clones (\cref{appfig:appendix-applications-PBC-P2}d). Together, these data indicate that the particular clones sampled can have a profound effect on attempts to identify the phenotypic changes implicated in driving or arising during histological progression.

\subsection{Growth patterns of pre-invasive clones}

\figurefloat{applications-maps-DCIS.pdf}{applications-maps-DCIS}{Growth patterns and histological associations of \ac{DCIS} clones}{\textit{\textbf{top}} \ac{BaSISS} maps of pure \ac{DCIS} samples: P1-D1 and P1-D2. The most prevalent genetic clone is projected as a coloured field (which corresponds to \textit{\textbf{middle}}) on DAPI images (reported if the CCF > 25\% and the inferred local cell density > 300 cells per mm\textsuperscript{2}). Scale bar, 5 mm. The pie chart reports the \ac{WGS}-estimated clone composition. The white dashed contours delineate morphologically defined lobules. The beige contours mark 114 and 40 manually selected microregions in P1-D1 and P1-D2, respectively, the clonal composition of which is reported by the heatmaps in \textit{\textbf{middle}}. Microregions were manually selected and represent single or small groups of intimately related acini or ductules from the same lobule. \textit{\textbf{middle}}, The phylogenetic tree inferred from P1 multiregion \ac{WGS}. Branches and nodes are coloured to reflect the clone maps. Only branches detected in P1-D1 and P1-D2 are coloured. WT - wild type. \textit{\textbf{bottom-left}}, H\&E images report representative subclone histological features in regions selected from \textit{\textbf{top}}. Scale bars, 100 $\mu$m and 50 $\mu$m (vacuoles). \textit{\textbf{bottom-right}}, Stacked bar plot summarizes histological features of microregions dominated by P1-green (n = 66) or P1-orange (n = 72). Nuclear pleomorphism is a measurement of the amount of variability in size and shape of the nuclei and is a major determinant of the histological grade.}

To demonstrate that BaSISS can be used to chart growth patterns in relation to complex tissue structures, we turned our attention to three DCIS samples from P1 that spanned a tissue surface area of 224 mm\textsuperscript{2} (P1-D1, P1-D2 and P1-D3) (top panel \cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS}a). The adult female breast comprises multiple, branching ductal systems, termed lobes, that extend from the nipple surface to the acini of the lobules \parencite{Going2004-af, Schnitt2013-fw}. DCIS arises from the duct epithelium and is considered a lobar disease as it typically involves the ducts and lobules of a single lobe \parencite{Pinder2010-vl}. Although \ac{DCIS} is known to be genetically heterogeneous \parencite{Casasent2018-gx}, how DCIS clones are organized and grow through the wider duct system remains elusive \parencite{Thomson2001-sm}.

The clone maps generated for the three samples formed striking mosaics of mainly green and orange, and occasional blue and grey that localized to areas of histologically confirmed \ac{DCIS} (top panel \cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS}a). Immune clusters and occasional normal or hyperplastic ducts appeared white (unstained), consistent with a different genetic ancestry. In P1-D3, a 3-mm length of a large duct exhibited both a genetic and a histological transition from normal ductal epithelium to DCIS along its length, confirming that, although neoplastic involvement was extensive in this lobe, it was incomplete (\cref{appfig:appendix-applications-DCIS}a). On dividing the glandular tissue into lobules (white dashed contours; \cref{fig:applications-maps-DCIS}), it was apparent that a handful of lobules contained a single clone, but often multiple clones co-occurred. Indeed, we were surprised to observe that the same clones repeatedly co-existed within lobules that spanned centimetres of tissue. These appearances seem at odds with the traditional model of clonal competition in which a fitter clone generates localised monoclonal sweeps.

\explain{}{\marginfig{side-plot-dcis-sweeps.pdf} Cartoon of a lobe of the breast with normal anatomy (left) and \ac{DCIS} (right), with lobules exhibiting monoclonal and polyclonal involvement}

However, at finer, sublobular resolution, complete or near-complete clonal sweeps are the dominant pattern, as exemplified by assaying 146 representative microscopic regions that represent individual or small clusters of intimately related acini and ducts (beige contours; \cref{fig:applications-maps-DCIS} top). The existence of frequent clonal sweeps as inferred by BaSISS (\cref{fig:applications-maps-DCIS} middle) was corroborated by \ac{LCM}–\ac{WGS} of additional microregions (\cref{appfig:appendix-applications-DCIS}b). In some instances, including P1-D1-88 (\cref{appfig:appendix-applications-DCIS}c) and P1-D2-0 (\cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS}d–f), clonal interfaces are directly observed within a continuous anatomical space. However, more commonly, rapid clone field transitions (see interactive maps at \href{https://www.cancerclonemaps.org/}{https://www.cancerclonemaps.org/}) coincided with the myoepithelial cell layer and/or basement membrane that define an acinus or ductule border. It thus transpires that the microanatomical structure of resident tissues can have, an as yet poorly understood, role in shaping observed subclonal architectures (top panel \cref{fig:applications-maps-DCIS}).

\subsection{DCIS clone-specific phenotypes}

Integration of histological and spatial gene expression data from serial sections revealed that the DCIS clones, P1-green and P1-orange, exhibit many phenotypic differences that are consistent across large tissue areas (\cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS,appfig:appendix-applications-DCIS-expression}). Histogenetic associations were very strong, with regions dominated by P1-green being more likely to have an intermediate rather than a low nuclear grade ($P$<0.0001; Fisher’s exact test after Bonferroni correction), exhibit more nuclear pleomorphism ($P$ < 0.0001), necrosis ($P$ < 0.0001), vacuoles ($P$ < 0.0001) and a non-solid architectural growth pattern ($P$ < 0.0001) (\cref{fig:applications-maps-DCIS} and \cref{appfig:appendix-applications-DCIS}).

Clone and cell type-resolved spatial gene expression analysis using targeted \ac{ISS} further corroborated phenotype–genotype correlations. A total of 28 of 91 interrogated genes were differentially expressed by the two main clones (\ac{FDR} < 0.1, fold change > 1.5 both ways; \cref{appfig:appendix-applications-DCIS-expression}). Consistent with a higher nuclear grade, P1-orange epithelial cells exhibited higher expression of the cell-cycle regulatory oncogenes \gene{CCND1} and \gene{CCNB1} and the oncogene \gene{ZNF703}, which have been linked to adverse clinical outcome \pcref{Solin2013-zy}. Overall, architectural and nuclear appearances and gene expression profiles were remarkably lineage-specific, and it was particularly notable that these different patterns could also be appreciated spatially, in regions with sublobular, microscopic clone intermixing, adding weight to the clone composition predictions by the model (\cref{appfig:appendix-applications-DCIS}d).

\subsection{Metastatic clones in a lymph node}

Lymph node metastasis is associated with higher rates of cancer mortality41. Whether it has an active role in facilitating cancer progression or simply reflects a more aggressive or distinct biology of certain clones is largely unknown. A substantial challenge is low cancer purity of diffusely infiltrated lymph nodes, which can make it difficult to separate cancer from immune cell-derived molecular signals. To demonstrate that \ac{BaSISS} can facilitate the simultaneous study of cancer and immune compartments in such challenging cases, we analysed \ac{BaSISS}, histological annotation and \ac{ISS} targeted gene expression datasets from sample P2-LN1 (\cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-LN,appfig:appendix-applications-LN-hypoxia}).

\figuretextwidth{applications-LN-genome.pdf}{applications-LN-genome}{genomic structures in P2-blue and P2-orange clones}{Plots of the genomic structures in P2-blue and P2-orange clones in the vicinity of the \gene{HER2} gene, derived from \ac{WGS} data of P2-TN2 and P2-LN1. Vertical lines represent genomic rearrangement breakpoints coloured by the phylogenetic tree branch where the event occurred. Dots represent local (binned) copy number. \gene{HER2} amplification, \gene{CACNB1} fusion and \gene{HER2} mutation are \ac{BaSISS} targets used to track this complex event. \acs{BFB}, breakage-fusion-bridge.}

BaSISS in P2-LN1 targeted 13 trunk and branch alleles, including point mutations and an expressed novel internal fusion in the \gene{CACNB1} gene that was co-amplified with the clinically targetable breast cancer oncogene \gene{HER2} in a \acf{BFB} event (\cref{fig:applications-LN-genome} and Supplementary Data Table 1). The model detected two clones (P2-blue and P2-orange) that formed spatially segregated patterns in P2-LN1 (cref{fig:applications-maps-LN}). Only P2-blue was detected in primary breast tumours (P2-TN1 and P2-TN2) (\cref{fig:applications-maps-PBC-P2} and \cref{appfig:appendix-applications-PBC-P2}b).

\figurefloat{applications-maps-LN.pdf}{applications-maps-LN}{Intrinsic and extrinsic features of metastatic subclones in a lymph node}{\textit{\textbf{top-left}}, \ac{BaSISS} map of P2-LN1, which relates to P2-TN1 \pcref{fig:applications-maps-PBC-P2} and P2-TN2 (cref{appfig:appendix-applications-PBC-P2}a,b). The most prevalent genetic clone colours are projected as coloured fields on the DAPI image (reported if the \ac{CCF} > 25\%; a threshold of 5\% is used in regions of diffusely infiltrating blue to allow visualization in very high normal contamination regions). Scale bar, 2.5 mm. Coloured contours define microregions with distinct metastatic cancer growth patterns (M-GP1 and M-GP2); `+' indicates the surrounding sinus epithelium. \textit{\textbf{top-right}}, Phylogenetic tree inferred from P2 multiregion \ac{WGS}.  Branch and node colours inform the clones mapped in \textit{\textbf{top-left}}. \gene{HER2} amplification (orange squares: pale - low, bright - high amplifications), \gene{CACNB1} fusion (red square) and \gene{HER2} mutation (red star) are \ac{BaSISS} targets used to track this complex event. The top heatmap reports the \ac{BaSISS} clone contribution to 39 histologically annotated microregions from a (regions with 5\% or more tumour cells are included). The bottom heatmap reports microregion histological features. Pan-CK, pan-cytokeratin. \textit{\textbf{middle-right}}, Representative areas of the two main growth patterns stained with H\&E. Scale bar, 100 $\mu$m. \textit{\textbf{bottom}}, From left to right: volcano plot of immune cell expression of the 62 genes in the \ac{ISS} immune panel, volcano plot of epithelial cell expression of the 91 genes in the \ac{ISS} immune panel. Significantly (\ac{FDR} > 0.1), differentially expressed (fold change of more than 1.5 both ways) genes are coloured. Violin plots depict clone-specific cell-type contribution posterior density of the generalized linear mixed model with region-specific random effect, and includes the 22 clone territories with a dominant clone fraction > 0.05 in P2-LN1. Significant comparisons were controlled for \ac{FDR} using the Benjamini–Hochberg procedure.}

Detailed histological annotation, blinded to the clone territories, was performed using a combination of H\&E, \protein{CD45} and pan-cytokeratin \ac{IHC} and identified multiple metastatic cancer growth patterns (coloured contours; \cref{fig:applications-maps-LN} and Supplementary Table 2). Intersecting the clone maps and histological annotations revealed strong associations between the two detected clones and the two main histological growth patterns ($P$ < 0.0001, Fisher’s exact test) (\cref{fig:applications-maps-LN} top-right). The P2-orange clone formed monotonous sheets of cancer cells, exhibited weak immunoreactivity for pan-cytokeratin and often occupied sinusoidal structures. By contrast, P2-blue cells stained more strongly for pan-cytokeratin and, when clustered, surround densely packed lymphocyte cores (\cref{fig:applications-maps-LN} right and \cref{appfig:appendix-applications-LN}a–d).

We sought to determine whether transcriptional differences support the spatial inference of clones. Consistent with the known HER2 amplification, P2-orange expressed higher levels of HER2 (bottom pannel \cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-LN}c). A total of 17 of 91 genes were differentially expressed and many of these are implicated in critical biological cancer pathways and/or have recognized prognostic value, including CTSL2, VEGFA (encoding vascular endothelial growth factor receptor A) and CD24 \parencite{Sereesongsaeng2020-vp, Kwon2015-jk} (bottom pannel \cref{fig:applications-maps-LN}). Spatially plotting these genes confirmed that clone-specific expression patterns are recapitulated within multiple, spatially distinct expansions across more than 1 cm\textsuperscript{2} of tissue (\cref{appfig:appendix-applications-LN}a–c).

Integration of spatial transcriptomics data also revealed that metastatic subclones occupied distinct immune microenvironments. Relative to P2-orange cells, P2-blue cells resided in neighbourhoods enriched for T cells and B cells (bottom pannel \cref{fig:applications-maps-LN}). In fact, P2-blue cells frequently formed clusters around B cell-rich germinal-like centres, highlighting a potential clone-specific interaction with the adaptive immune system (middle-right pannel \cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-LN}a,d). By contrast, P2-orange regions frequently resided inside the lymph node sinuses that were lined by endothelial cells expressing \gene{CD34} and \gene{PDGFRB} (middle-right pannel \cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-LN-hypoxia}). Most of the immune cells in P2-orange regions were myeloid cells with expression profiles consistent with the presence of both M1 and M2 macrophages (\gene{CD163}, \gene{CD68}, \gene{HAVCR2} and \gene{FCGR3A}), and the most highly enriched gene, \gene{CXCL8}, is released by hypoxic macrophages \parencite{Li2015-ng} (bottom pannel \cref{fig:applications-maps-LN}). Indeed, relative to P2-blue, it emerges that P2-orange experienced more hypoxic conditions manifesting as higher cancer cell expression of \gene{VEGFA} and necrotic regions (\cref{appfig:appendix-applications-LN-hypoxia}). Hypoxia signatures are associated with adverse clinical outcomes, probably because they reflect the emergence of environments that can select for hypoxia-tolerant clones and/or cancer proliferation rates outstrip neoangiogenesis \parencite{Cairns2004-vs}. Together, these data demonstrate how \ac{BaSISS} clone maps allow one to spatially relate such variation in microenvironments to individual clones.

\section{Discussion}

Here we present \ac{BaSISS}, a pipeline that combines a highly multiplexed fluorescence microscopy-based protocol and algorithms to map and phenotypically characterize the unique set of subclones of cancer. These maps served as the basis for further spatially and single-cell-resolved molecular and histological characterization of each clone. Applying \ac{BaSISS} to a series of samples from the key stages of breast cancer progression—carcinoma in situ, invasive cancer and lymph node metastasis—it is notable that virtually every sample exhibited a spatial organization of clones, which warrants further investigation in larger cohorts. The fact that nearly all clones examined in this dataset displayed distinct clone-specific gene expression, stromal and immune microenvironments and microanatomical niches highlights the functional relevance of at least some subclonal diversification.

The ability to chart clonal growth patterns and clone-specific genetic underpinnings of the tumour microenvironment is likely to be instrumental in elucidating how different evolutionary processes operate and manifest across different cancer types—or even in histologically normal tissues \parencite{Sottoriva2015-ci}. Understanding the forces of malignant progression, especially invasion and metastasis, and how interactions with the tumour microenvironment shape clinical outcomes \parencite{Risom2022-uw} appear of particular importance. Detailing the functional and microenvironmental characteristics of different clones is also relevant as some part of subclonal diversity in tumours may be due to selectively neutral drift, but the exact extent remains debated.

Particular advantages of the technology are that it is capable of interrogating very large tissue sections on the scale of squared centimetres, which enables studying entire cross-sections of smaller tumours. It is also comparably cheap, unlike solely relying on sequencing-based methods \parencite{Vickovic2019-tz}. The three main limitations of the approach are relatively low sensitivity, which currently precludes single-cell genotyping, a reliance on RNA with the resulting variation in gene expression levels of targeted transcripts, and the fact that clone-defining mutations need to be detected first by separate sequencing-based assays. Greater sensitivity and spatial resolution may be achieved by including more targets per clone and by favouring mutations with higher predicted expression levels, for example, in higher copy number states. A switch to hybridization-based sequencing and direct RNA-binding probes may also improve base-specific detection by several fold \parencite{Gyllborg2020-uq,Lee2022-ha}. Further discussion of the implications of our observations and limitations of the method is provided in a Supplementary Note.

It is often stated that `nothing in biology makes sense except in the light of evolution' \parencite{Dobzhansky1973-va}, which is likely to be true for cancer biology. The ability to spatially locate and molecularly characterize different cancer subclones adds essential features to the spatial-omics toolkit. It provides a robust evolutionary framework that is necessary to interpret the biological relevance of many of the more plastic spatial characteristics of a cancer. Future widespread applications of spatial genomics approaches such as \ac{BaSISS} will uncover how cancers grow in different tissues and allow us to track, trace and characterize the ill-fated clones that are responsible for adverse clinical outcomes.

