\chapter{Introduction}
\label{sec:chapter-introduction}

\section*{Contributions}

This chapter draws upon the work detailed in \fullauthcite{Seferbekova2023-wg}, for which I am the co-first author contributing equaly with \ac{zaira}.

I conducted literature research in collaboration with \ac{zaira}, and we both wrote the manuscript under the supervision of \ac{lucy} and \ac{moritz}. Based on a discussion among all authors, \ac{zaira} created the initial figures, which a professional illustrator later redrew.

The content of this chapter has been adapted from the original review manuscript, with only minor stylistic and structural alterations.

\section{Componentes of cancer evolution}

\figuretextwidth{intro-spatial-biology-general.pdf}{intro-spatial-biology-general}
    {Spatial biology of cancer evolution \parencite{Seferbekova2023-wg}}
    {\emph{Top}, Somatic evolution and cancer development. Human cells accumulate mutations throughout their lifetime. A subset of these, so-called driver mutations, are associated with neoplastic transformation. Ongoing mutation and selection cause subclonal divergence within the tumour clone. \emph{Bottom}, Spatial technologies add new layers of information. A tumour is a mosaic of subclones with distinct micro-anatomy and tumour microenvironment. Spatial analyses offer new avenues for charting and understanding the biology and evolution of the cancer ecosystem at scales ranging from a single cell to whole tissue.}


The drivers of evolution are \explain{\emph{mutation}}{Changes in the DNA sequence that are inherited across cell generations. Sources of mutation are erroneous replication, biochemical alterations of DNA and failed DNA repair. To date, all cells within the human body are found to accumulate mutations over their lifetime} and \explain{\emph{selection}}{In evolution, natural selection denotes the process of survival and reproduction of the fittest organism within a given environment} \parencite{Cairns1975-oz,Nowell1976-sm}. Mutation of somatic cells is an inevitable and persistent consequence of life \parencite{Martincorena2015-br, Moore2021-yr, Li2021-th}. In most normal tissues, mutations accumulate at a steady rate of around 15–50 per cell per year of life, with only the germline known to exhibit lower rates \parencite{Moore2021-yr}. Tumour cells often exhibit an elevated mutation burden due to the variety of mutational processes they have been exposed to during life and, in some cases, due to acquired hypermutation \parencite{Alexandrov2020-uo}. The continuous accumulation of mutations inevitably leads to diversification at the level of single cells both in tumours and normal tissues.

The second force of evolution is selection, which describes how a fitter lineage outgrows its relatives. Selection operates at the level of a wide range of heritable phenotypes that derive from genomics and epigenomics. \explain{Clones}{A clone is a population of cells that derive from a common ancestor. Cancers are found to be clones deriving from a single cell that expanded during the lifetime of a host. Clonal alterations are mutations that are present in all cells of a cancer because their occurrence preceded the expansion of the tumour} with a selective advantage in the environment to which they are exposed will expand, while those with a disadvantage will tend to disappear \pcref{fig:intro-spatial-biology-general}. Advantageous variants are therefore enriched in genomes of aged normal tissues and cancers \parencite{Greenman2006-cx,Martincorena2017-uw}. More than 500 so-called cancer driver genes have been reported to date \parencite{Lawrence2013-mi,Gonzalez-Perez2013-vo,Sondka2018-xf}, which are believed to cause different cancer hallmark traits and enabling characteristics that involve cell-intrinsic mechanisms as well as interactions with the \explain{\ac{TME}}{A combination of non-tumour cells, such as stromal and immune cells, vessels, metabolites, signalling molecules, and other extracellular components among which tumour cells exist} \parencite{Hanahan2000-ze, Hanahan2022-eb}.

While mutation continuously generates subclonal diversity at the level of single cells, it is well documented that tumours are mosaics of subclones each comprising hundreds of thousands of cells that have arisen from a shared ancestor \parencite{Shah2009-xz,Nik-Zainal2012-zz,McGranahan2015-gb,Andor2016-hi, Dentro2021-yb} \pcref{fig:intro-spatial-biology-general}; these patterns have been further confirmed by single-cell studies \parencite{Navin2011-qq,Wang2014-bp,Casasent2018-gx,McPherson2016-zu,Laks2019-rp,Salehi2021-ad}. The mechanisms of these expansions remain debated. It has been shown that cancer \explain{subclones}{Further clones emerging within a tumour from one founder clone. Subclonal mutations are limited to a fraction of cancer cells and occur during tumour expansion} exhibit signs of \explain{positive selection}{The spread of advantageous alleles within a population} and driver gene mutations that can also be found clonally \parencite{Dentro2021-yb}. However, it is also conceivable that subclones branching at early stages of tumour development reach considerable size without a selective advantage as they continue to expand with the same tumour, a phenomenon termed neutral evolution \parencite{Williams2016-qh}.

Subclonal mosaicism is often found to be spatially variegated as demonstrated in diverse studies based on tumour macrodissection \parencite{Navin2010-uw,Gerlinger2012-qm,De_Bruin2014-ow,Gerlinger2014-hr, Yates2015-xk,Morrissy2017-vy,Jamal-Hanjani2017-uv,Watkins2020-gc} and laser-capture microdissection \parencite{Casasent2018-gx, Heide2022-ev,Woodcock2020-nt,Grossmann2021-sl,Zhao2022-xd,Heide2019-rr,Su2018-rr,Bao2018-kj} \pcref{fig:intro-spatial-biology-general}. These clonal variegation patterns were shown to be accompanied by differential gene expression in multiple cancer types, including renal \parencite{Gerlinger2012-qm}, colorectal  \parencite{Househam2022-cp}, lung \parencite{Biswas2019-kb} and breast cancer \parencite{Lomakin2022-ks}, using spatially resolved genomic and transcriptomic analyses, indicating the presence of subclone-specific gene expression and associations between certain subclones and characteristic \ac{TME}s.

The exact mechanisms creating these phenomena are not fully understood, largely owing to a dearth of methods that can molecularly characterize whole tumour sections with spatial resolution. A variety of non-exclusive explanations have been proposed. A rapid expansion could lead to a fragmentation of closely related clones \parencite{Sottoriva2015-ci} even though it has also been argued that cell dispersal is a crucial element shaping tumour structure \parencite{Waclaw2015-yw,Gallaher2019-xx}. However, it is also possible that tissue micro-anatomy, such as the ductal system of the breast, provides a template for clonal segregation (discussed further in the \cref{sec:intro-tissue-organisation}). Lastly, it is conceivable that locally distinct microenvironments or niches favour the selection of particular clones.

Various new spatially resolved genomic, transcriptomic and proteomic technologies offer novel insights and answers to these unresolved questions \pcref{fig:intro-spatial-biology-general}. The technological aspects enabling the nascent field of spatial cancer biology are summarised in \cref{box:technologies} and have been extensively reviewed elsewhere \parencite{Lewis2021-ic}. Individual technologies differ in the multiplexity of their readouts and their spatial resolution as well as in their sensitivity and available field of view. Combining genomic, transcriptomic and proteomic layers enables a rich characterisation of \explain{tumour ecosystems}{The collective set of heterogeneous cells in the vicinity of a tumour comprising cancer cells and the TME}.

This introduction focuses on the spatial aspects of cancer evolution, summarises how spatial transcriptomics and proteomics reveal a complex landscape of the tumour ecosystem, discusses how interactions with the \ac{TME} are integral to cancer evolution, and proposes potential clinical applications and future directions. Particular focus is given to the role of tissue micro-anatomy in controlling the rate of evolution, as well as cellular interactions with the \ac{TME} within which cancer cells evolve.

\section{Tissue organisation controls evolution}
\label{sec:intro-tissue-organisation}

The rate at which malignant clones emerge and spread through the tissue is controlled by its organisation. Levels of organisation include the micro-anatomical \explain{tissue architecture}{The micro-anatomical spatial organization of the tissue. Typical examples are layered epithelial tissues, glands and crypts. Tissue architecture can be combined with other means of tissue organization such as differentiation hierarchies} as well as differentiation hierarchies with few stem cells feeding successively larger pools of differentiated cells. The breakdown of these protective principles is a key feature of cancer development. However, the resident tissue structure, for example, the ductal system in the breast or epithelial layers of the oesophagus, can also influence the rate of progression at pre-invasive stages. The transformation ends with overwhelming metastatic disease.

\subsection*{Normal tissue organisation suppresses evolution}

\figuretextwidth{intro-normal-tissue-evolution.pdf}{intro-normal-tissue-evolution}
    {Normal tissue structure suppresses somatic evolution \parencite{Seferbekova2023-wg}}
    {Top: in addition to physical constraints, the differentiation hierarchy in the colonic crypt limits the number of cells with fixation potential, which reduces the chances of malignant transformation. Bottom: highly structured tissues, such as the colon, slow evolution by constraining the spread of clones. Physically unstructured tissues, such as the haematopoietic system, exhibit the fastest rate of evolution and clones, which replace the entire organ}

It has long been hypothesized that somatic tissues are structured to suppress the rate of somatic evolution \parencite{Cairns1975-oz}. These ideas are based on fundamental theoretical considerations that the rate of evolution depends on population structure. Generally, the rate of evolution is determined by the rate at which new mutations are generated (a product of the number of cells and the mutation rate per cell), the probability that new mutants sweep through the population \parencite{Lieberman2005-cy} and the time it takes to do so \parencite{Frean2013-rq,Tkadlec2019-cp}. In addition to suppressing the mutation rate, differentiation hierarchies can reduce the chance of emergence and slow the spread of mutant lineages. The haematopoietic system constitutes a basic example \parencite{Lopes2007-je}: its 10\textsuperscript{13} cells derive from around 100,000 haematopoietic stem cells \parencite{Lee-Six2018-pn,Sender2016-og}, which themselves divide slowly and produce a range of faster-dividing differentiated cells \parencite{Humphries2008-jj}. This hierarchy reduces the rate of evolution, as only mutations in the comparably small number of stem cells and potentially the first progenitors, or mutations that lead to a differentiation blockage, will remain in the population. The colon extends these organisational principles by its compartmentalization into micro-anatomical crypts; in each crypt, a similar differentiation hierarchy operates in which a very small number of stem cells replenishes the colonic epithelium \parencite{Humphries2008-jj}. As differentiating cells are fated to die within ~5 days, only mutations arising in the stem cells persist \parencite{Cairns1975-oz,Nowak2003-pq,Sender2021-ow}. Furthermore, micro-anatomy largely constrains the fixation of stem-cell mutations to individual crypts \pcref{fig:intro-normal-tissue-evolution}.

Systematic studies of normal tissues confirm that, in tissues with a more complex architecture, clones typically remain small in size and develop independently in comparison to less structured tissues where clonal expansions tend to be larger \parencite{Li2021-th}. In a recent investigation of 517 normal colonic crypts, only 1\% harboured cancer driver gene mutations, and those that were mutated were confined to single crypts \parencite{Lee-Six2019-fy}. Similar observations have been made for glandular tissues such as the endometrium \parencite{Moore2020-qb}: although most of the glands harboured driver mutations, they were limited to a single mutation or a few neighbouring mutations. Conversely, it is well established that the haematopoietic system harbours mutant lineages whose frequencies can reach up to 100\% \parencite{Genovese2014-wm,Jaiswal2014-tz}. Other epithelial tissues, including the skin, oesophagus or bladder, have an intermediate degree of tissue structure and are patchworks of microscopic clones \parencite{Moore2021-yr,Martincorena2015-yu,Martincorena2018-rl,Lawson2020-as} \pcref{fig:intro-normal-tissue-evolution}. In such tissues, the fixation probability is high but the time to fixation is greater than in the haematopoietic system because the dynamics are confined to two dimensions, which reduces the interfaces at which a fitter variant may replace less-fit competitors. Of note, not all mutations causing a selective advantage are associated with malignant transformation. For example, in the skin and oesophagus, \gene{NOTCH1} mutant clones expand and suppress malignant evolution while maintaining normal tissue function \parencite{Martincorena2015-yu,Martincorena2018-rl, Colom2021-mk,Fowler2021-bl, Abby2021-ik}.

\subsection*{Breakdown of evolutionarily confining tissue architectures in cancer}

The fundamental observation that dedifferentiation and the loss of normal tissue architectures are correlated with cancer aggressiveness was made a century ago and still forms part of modern histopathological grading systems that direct cancer treatment \parencite{Louis2007-ia,Elston1991-md,Epstein2016-un,Greenough1925-wg}. In these systems, higher grades correspond to more aggressive tumours, typically composed of less differentiated cells.

\figuretextwidth{intro-carcinogenesis.pdf}{intro-carcinogenesis}
    {Spatial steps of cancer progression \parencite{Seferbekova2023-wg}}
    {Normal tissue structure breaks down during tumour progression. This process is exemplified here by a carcinoma: breach of the basement membrane is followed by tumour invasion into the neighbouring stroma, lymphangiogenesis and angiogenesis. Through blood and lymphatic vessels, cancer cells with metastatic potential can transfer to lymph nodes and distant organs, giving rise to metastases at these new sites. Metastases containing cancer cells of different lineages within the same lesion are called polyclonal, whereas metastases containing cancer cells of the same lineage are called monoclonal. Metastatic subclones may later return to the primary site and recolonize it. ECM, extracellular matrix; HSC, haematopoietic stem cell}

The breakdown of such evolutionarily confining tissue structures is a hallmark of cancer. It is especially noticeable among carcinomas, which derive from epithelial tissues \pcref{fig:intro-carcinogenesis}. At the earliest stages of progression, so-called \explain{carcinomas in situ}{Neoplastic expansions of epithelial cells that are confined to the normal tissue structure in which they arose, without invasion of the adjacent stroma} are still restrained to the resident tissue structures, such as the duct of the breast. Similar to the considerations of normal tissues, the resident tissue structures may delay clonal sweeps. This tissue-based restraint was observed in a case of breast cancer, where spatial genomic analysis revealed two distinct subclones occupying a checkerboard pattern of micro-anatomical niches within the ductal system \parencite{Lomakin2022-ks}.

During \explain{invasion}{The process of penetration and spread of cancer cells into the neighbouring normal tissue. In the case of carcinomas, invasion involves a breach of the basement membrane}, tissue architectures are lost, probably via different mechanisms in each tissue type, but a frequent property of carcinomas is the \ac{EMT}. \ac{EMT} enables epithelial cells to lose their polarity and escape from the confinement of planar epithelial tissue organisation (as reviewed in \textcite{Polyak2009-fi}). In colorectal adenocarcinoma, \ac{EMT} can be induced via the \gene{WNT} signalling pathway by loss-of-function mutations in \gene{APC} or somatic mutations of \gene{CTNNB1} inhibiting its degradation \parencite{Morin1997-ez}. Both alterations are believed to be a key step in the development of adenomas, the neoplastic precursor to invasive adenocarcinoma. A further step of invasion is the breakdown and remodelling of the \ac{ECM}, thought to also be facilitated by the microenvironment (see the \cref{sec:mapping-the-cancer-ecosystem}).

The consequence of this breakdown is likely the acceleration of malignant evolution as the fixation probability and the rate of fixation are higher in unstructured populations with a greater degree of mixing. Several theoretical investigations have demonstrated that spatial constraints can control the mode of tumour evolution \parencite{West2021-ar,Noble2022-eg}. Another consequence is that different selective pressures apply, which may be one cause of the characteristic sequences of cancer progression such as the classic \gene{APC}-\gene{KRAS}-\gene{TP53} model for colorectal carcinoma \parencite{Fearon1990-el,Gerstung2020-sg} and the four stages of \gene{TP53} inactivation in the progression of pancreatic ductal adenocarcinoma from a pre-malignant to malignant state \parencite{Baslan2022-rb}. In this light, a pan-cancer analysis revealed that the portfolio of late and subclonal driver gene mutations tends to be more diverse than the set of early alterations \parencite{Gerstung2020-sg}.

\subsection*{Metastatic dissemination}

The ability of cancer cells to leave a primary tumour site, seed in distant organs and generate metastatic deposits presents landmark stages that are strongly predictive of poor clinical outcomes \parencite{Lambert2017-ll}. This inherently spatial process involves, upon tissue invasion, intravasation of single or small aggregates of cancer cells, circulation through the blood system, arrest in the capillaries of the target organ, extravasation, and proliferation \parencite{Fidler2003-lh} \pcref{fig:intro-carcinogenesis}. These steps thus involve a range of micro-anatomical bottlenecks and possible differential selective pressures in different stages and target organs.

Several multi-region genomic sequencing studies have investigated the phylogenetic relationships of primary tumours and metastases from the same patient \parencite{Gerlinger2012-qm,De_Bruin2014-ow,Yates2015-xk,Jamal-Hanjani2017-uv,Jones2008-sd,Mamlouk2017-wt,Zhou2021-oj,Yates2017-xc,Noorani2020-dp,Karlsson2018-kv}. These data show that most metastases have diverged from the primary tumour clone typically shortly before or after its \explain{most recent common ancestor}{In cancer evolution, the most recent founder cell from which all other tumour cells have directly descended}, which is often dated 1–5 years before diagnosis \parencite{Yates2017-xc,Noorani2020-dp,Gundem2015-kg,Hu2020-ui}. The origin of \explain{metastasis}{A process of cancer spreading from the primary disease site to lymph nodes or other organs, resulting in the formation of metastatic deposits known as metastases} appeared to be mostly clonal \parencite{Brastianos2015-ox,Brown2017-de,De_Mattos-Arruda2019-sb} even though, in advanced disease, intermixed subclones have been found in multiple metastases, suggesting spread from one metastasis to another \parencite{Noorani2020-dp,Gundem2015-kg,Aceto2014-lu}. Pan-cancer analyses of metastatic cancers have revealed that metastases exhibit fewer intra-metastatic subclones than primary tumours \parencite{Priestley2019-wi,Martinez-Jimenez2022-tl,Nguyen2022-jr}, which may in part be a consequence of sampling fewer cells (often core needle biopsies). From the perspective of driver gene mutations, it has been argued that treatment-naive metastasis has broadly the same driver gene mutations as the matched primary \parencite{Reiter2019-bc}. Characteristics of metastatic cancer include high levels of genomic instability, \gene{TP53} mutations and whole-genome duplications, shared by both the primary tumour and metastases \parencite{Priestley2019-wi,Nguyen2022-jr}. However, \explain{adjuvant therapies}{Cancer treatments given after primary therapy (particularly following surgical resection of a tumour) to reduce the risk of relapse} can select for specific mutations, for example, prostate and breast cancers treated with endocrine deprivation therapies are enriched for resistance mutations in \gene{AR} and \gene{ESR1}, respectively \parencite{Martinez-Jimenez2022-tl,Nguyen2022-jr}.

Although it emerges that tissue architecture has a profound influence on somatic evolution, it is important to realise that cellular interactions with normal cells are also likely to affect the fate of cancer clones. The composition of the TME and its role in cancer evolution will be discussed in the next two sections.

\section{Mapping the cancer ecosystem}
\label{sec:mapping-the-cancer-ecosystem}
The cellular composition of tumour masses is highly variable, with the neoplastic lineage contributing as few as 10\% of cells in some cases \parencite{Aran2015-dw}. The remaining cells derive from the TME, which is usually dominated by immune and stromal cells in addition to vessels, adipocytes and pericytes. Together, these cells and other physical structures, such as \ac{ECM} and chemical gradients, form an environment termed the cancer ecosystem \parencite{Somarelli2021-va}. Over the past decade, \ac{scRNAseq} analyses have catalogued the cellular composition of the cancer ecosystem \parencite{Darmanis2017-yb,Venteicher2017-mo,Pombo_Antunes2021-md,Puram2017-mn,Tirosh2016-rl,Izar2020-tf,Lambrechts2018-hx,Chung2017-ic}. More recently, spatial transcriptomic and proteomic technologies have started to uncover how these diverse cell types organise into communication interfaces and niches \parencite{Schapiro2017-gy,Arnol2019-fv} (\cref{box:analysis,box:technologies}).

\subsection*{Cellular composition of the \ac{TME}}
The \ac{TME} has been studied extensively and, therefore, this section only presents a high-level summary of major cell types. Inflammation is a hallmark of cancer \parencite{Hanahan2011-cd} and reflects the interplay of antitumorigenic and pro-tumorigenic immune responses \pcref{fig:intro-tme-composition}. The antitumour response is primarily generated by cytotoxic \protein{CD8}+ T lymphocytes that recognise antigens presented on the tumour cell surface \parencite{Raskov2021-gb,Philip2022-ly}. The success of this response is dependent on priming and prior education by professional antigen-presenting cells (dendritic cells) and is facilitated by \protein{CD4}+ T helper cells \parencite{Borst2018-tp,Waldman2020-kf}. B cells have diverse functions, including antigen presentation, antibody production and cytokine-mediated cytolytic effects \parencite{Sharonov2020-vx}. In many cancers, B cells form dense aggregates with T cells and dendritic cells to form \acp{TLS} \parencite{Schumacher2022-mr} that are generally associated with favourable cancer outcomes and responses to \explain{immune-checkpoint blockade}{A type of anticancer immunotherapy that blocks checkpoint proteins on T cells or their targets on cancer cells and promotes an immune response and killing of the tumour} \parencite{Sautes-Fridman2019-nl}. Natural killer cells might have a particularly important role in anticancer immunity due to their ability to recognise stressed cells irrespective of \explain{neoantigen}{A tumour-specific antigen that is the result of a somatic coding mutation in the corresponding part of a gene. Antigens are protein fragments presented on the cell surface by the human leukocyte antigen (HLA) complex and recognised by different cells of the immune system} presentation \parencite{Wolf2023-hu}.

\figuretextwidth{intro-tme-composition.pdf}{intro-tme-composition}
    {Cellular composition of the \acf{TME} \parencite{Seferbekova2023-wg}}
    {The TME is composed of different cell types, involved in tumour-suppressing (on the left) and tumour-promoting (on the right) interactions with cancer cells. While TME composition can be captured by single-cell omic approaches (such as single-cell RNA sequencing), its spatial context is lost as a consequence of tissue dissociation. CAFs, cancer-associated fibroblasts; CTL, cytotoxic T lymphocyte; ECM, extracellular matrix; EMT, epithelial-to-mesenchymal transition; NK, natural killer; T\textsubscript{reg}, regulatory T cell}

By contrast, \protein{CD4}+ regulatory T (T\textsubscript{reg}) cells can suppress the immune response through cytokine release (for example, \protein{IL-10}) or by modulation of antigen presentation functions. In certain situations, some immune cells, such as macrophages, myeloid-derived suppressor cells and granulocytes, can actively promote tumour progression through two-way interactions with the tumour, other immune cells and the stroma \parencite{Goswami2023-qv,Murdoch2008-cu,DeNardo2019-tk}. The stroma also plays an active role in shaping the functional immune response to cancer. A variety of studies have demonstrated that cancer-associated fibroblasts and myofibroblasts typically change shape, become more proliferative, undergo diverse transcriptional and metabolic changes, engage in angiogenesis, and interact with leukocytes and tumour cells while the resulting \ac{ECM} changes in chemical composition and mechanical properties \parencite{Valkenburg2018-kn,Kalluri2016-xz,Lendahl2022-sx,Sahai2020-ok,Davidson2020-rc}.

\subsection*{Cellular interactions and neighbourhoods}

Although \ac{scRNAseq} studies have generated comprehensive catalogues of detailed TME cell types and states, these data only allow circumstantial insights into how cells interact. Spatial transcriptomic and proteomic technologies are beginning to show that the cells constituting the cancer ecosystem self-organise into micro-anatomical neighbourhoods, which could provide insights into the frequency and nature of different cellular interactions \pcref{fig:intro-cancer-neighbourhoods} \pcref{box:analysis}. The majority of spatial omic cancer studies to date have used spatial transcriptomics as commercialised by \ac{Visium} \pcref{box:technologies}. These studies reveal spatially structured TMEs in breast \parencite{Stahl2016-nq,Wu2021-uq,Andersson2021-pu}, prostate \parencite{Berglund2018-gh}, pancreatic \parencite{Moncada2020-ck}, colorectal \parencite{Qi2022-by}, skin \parencite{Ji2020-gn}, brain \parencite{Ravi2022-ut}, liver \parencite{Wu2021-wb}, bladder \parencite{Gouin2021-zx} and other cancer types \parencite{Erickson2022-zh,Barkley2022-gx}. The ability to use whole-transcriptome readouts also helped establish the presence and broad co-localization of rare cell types \pcref{fig:intro-cancer-neighbourhoods}. However, the current super-cellular resolution (55 $\mu$m, ~3–10 cells) requires informatic deconvolution of the measured signals in each spot.

\figuretextwidth{intro-cancer-neighbourhoods.pdf}{intro-cancer-neighbourhoods}
    {Cellular interactions and neighbourhoods \parencite{Seferbekova2023-wg}}
    {Cellular interactions: spatial co-localization and concomitant expression of receptors and ligands are indications of cellular interactions. Cellular neighbourhoods: spatially resolved proteomic studies detect clinically relevant neighbourhoods of molecularly related cell types \parencite{Danenberg2022-zb}. Cellular interfaces: subcellular resolution imaging reveals receptor–ligand localization at cellular interfaces \parencite{Nirmal2022-sq}}

Highly multiplexed antibody detection approaches have been used to generate single-cell resolution readouts of the spatial distribution of the major cell types in breast cancer, colon cancer and melanoma, amongst others \parencite{Danenberg2022-zb,Jackson2020-em,Nirmal2022-sq,Schurch2020-lp}. These approaches permit the detection of microscopic neighbourhoods with distinct composition of tumour, immune, stromal and vascular cells. In two studies that analysed hundreds of breast cancers by applying imaging mass cytometry \pcref{box:technologies} to tissue microarrays, several recurrent neighbourhoods were identified \parencite{Danenberg2022-zb,Jackson2020-em} \pcref{fig:intro-cancer-neighbourhoods}. The biological feasibility of the identified neighbourhoods was provided by their relationship with well-defined clinical subtypes and survival outcomes. For example, consistent with a failing antitumorigenic response, ‘suppressed expansion structures’ contained many dysfunctional T and T\textsubscript{reg} cells and were enriched in breast cancers with poor clinical outcomes. By contrast, a ‘vascular stroma’ signature was enriched in a very favourable outcome subgroup of hormone receptor-positive breast cancers \pcref{fig:intro-cancer-neighbourhoods} . The spatial patterns formed by individual cell types were also examined. In breast cancers, whereas T cells were diffusely scattered, B cells tended to form aggregates, were in multiple neighbourhood types, and were associated with disparate clinical subtypes and clinical outcomes, suggesting that B cell function might be dependent on the precise TME context \parencite{Wieland2021-ep} \pcref{fig:intro-cancer-neighbourhoods}. However, some tumour phenotypes, such as cell proliferation, can be shaped by further long-range interactions \parencite{Gaglia2022-zn}.

The proximity of cells does not necessarily equate to an interaction. To address this limitation, a combination of \ac{CyCIF} \pcref{box:technologies} and high-resolution three-dimensional optical deconvolution was used to study the physical features that might be associated with cellular interactions \parencite{Nirmal2022-sq}. In melanoma tissues, probable interactions were identified amongst approximately 20\% of immune cells at the tumour–stroma interface \pcref{fig:intro-cancer-neighbourhoods}. These interactions were often complex, involving three or more cells (for example, a melanoma cell contacting two \protein{CD8}+ cytotoxic T lymphocytes and one \protein{CD4}+ T\textsubscript{reg} cell) with associated cell surface molecular polarisation and receptor–ligand interaction. In some cases, cellular processes from macrophages extended more than 10 $\mu$m to make contact with other cells. Standard fine sections would miss many of these interactions, whereas basic proximity measurements would tend to overestimate the nature of interactions. As discussed in \cref{box:analysis}, although these analyses remain technically challenging and have low throughput, they demonstrate the unique potential afforded by spatial omics for the mapping of functional cellular interactions in tissue space.

\infobox{box-technologies.pdf}{box:technologies}{Enabling technologies}{
    A range of spatially resolved approaches for mapping DNA, RNA and proteins has been developed \parencite{Lewis2021-ic,Rao2021-lc}. Higher spatial resolution typically implies smaller molecular quantities. Other important parameters are the field of view, which ranges from millimetres squared to centimetres squared, and method sensitivity.
    \subsubsection*{Macrodissection and microdissection}
    The foundations were laid by macrodissection followed by multi-omic profiling but new technologies offer a microscopic and single-cell resolution. In multi-region sequencing \parencite{Gerlinger2012-qm,Yates2015-xk}, the collection of several biopsy samples from multiple geographically separated tissue regions is coupled with later bulk sequencing of DNA or RNA. \acf{LCM} allows the profiling of even smaller regions of interest using laser ablation \parencite{Emmert-Buck1996-nk}. \ac{LCM} is very versatile as it can be combined with high throughput sequencing, including whole-genome sequencing, and single-cell RNA or DNA sequencing \parencite{Casasent2018-gx,Ellis2021-du,Nichterwitz2016-pf,Cui_Zhou2022-fx}.
    \subsubsection*{Spatial molecular barcodes}
    This set of methods utilizes probes with unique barcode sequences to capture molecules and then map them to specific spatial locations. So far, these technologies have mainly focused on transcriptomic profiling with spatial transcriptomics \parencite{Stahl2016-nq}, 10x Visium and Slide-seq \parencite{Rodriques2019-co,Stickels2021-cm} being the most popular protocols among many others \parencite{Vickovic2019-tz,Liu2020-vw,Cho2021-ho}. Spatially resolved DNA sequence capture \parencite{Zhao2022-xd} has enabled spatially resolved genome-wide copy number calling along with several new methods for spatial epigenomic profiling \parencite{Deng2022-ad,Deng2022-gh}. Although these methods support unbiased discovery and result in data with high molecular content, the resolution is limited by the diameter of each spot and currently does not allow profiling at the single-cell level or point mutation detection. Further improvements in resolution are currently being developed \parencite{Vickovic2019-tz, Cho2021-ho,Chen2022-mp}.
    \subsubsection*{Image-based technologies}
    This group of methods involves in situ hybridization of fluorescently labelled probes (FISH) to nucleic acids of interest to identify their location with single-cell or subcellular resolution. A variety of multiplexed extensions exists \parencite{Chen2015-ab,Lyubimova2013-jd,Chen2016-if,Coskun2016-jr,Codeluppi2018-nd,Wang2018-sl, Lee2015-ge,Qian2020-mp,Chen2018-dg,Alon2021-ce,Gyllborg2020-uq}, including multiplexed error-robust FISH (MERFISH) \parencite{Su2020-fb,Lu2022-yq}, sequential FISH (seqFISH) \parencite{Eng2019-rx}, RNAscope \parencite{Wang2012-pv} and in situ sequencing (ISS) \parencite{Ke2013-ux} for spatial transcriptomic profiling. The latter two protocols have been further adapted to permit mutation detection using the Base Scope \parencite{Baker2017-dv} and BaSISS protocols \parencite{Lomakin2022-ks}. Immunohistochemistry (IHC) or immunofluorescence uses antibodies to localise proteins or other molecules. Highly multiplexed spatial proteomics with dozens of targets can be achieved using DNA barcodes \parencite{Goltsev2018-dd,Saka2019-lb}, multiplex immunofluorescence (CyCIF) \parencite{Lin2018-qf} and other approaches \parencite{Lin2015-ml,Gerdes2013-kn,Keren2019-cu}. Multi-omic approaches, such as GeoMx or CosMx \parencite{He2022-ee,Bergholtz2021-ql}, allow researchers to assess multiple layers of biological information within one cell simultaneously. Image-based technologies provide high-resolution tissue profiling but require a pre-defined panel of targets.
    \subsubsection*{Mass spectrometry}
    A further class of methods combines metal-conjugated antibodies and mass spectrometry. Spatial resolution is provided by spatial laser ablation (imaging mass cytometry (IMC)) \parencite{Giesen2014-yd} or ion beams (multiplexed ion beam imaging (MIBI)) \parencite{Angelo2014-gf}. These approaches detect proteins and smaller molecules, such as lipids, metabolites, and drugs, in single cells \parencite{Giesen2014-yd}, as reviewed in \textcite{Mund2022-kf}. Deep Visual Proteomics (DVP) combines artificial intelligence-driven LCM with mass spectrometry to map protein abundances onto tissue slides \parencite{Mund2022-fw}.}

\subsection*{Tumour histology and macrostructure}

The histological appearance of tumour cells and their micro-anatomical structures also reflect the underlying genomic alterations as revealed by two recent pan-cancer analyses using artificial intelligence-based digital histopathology \parencite{Kather2020-bt,Fu2020-cp}. In some cases, histological patterns have been found to be indicative even of distinct subclonal alterations \parencite{Lomakin2022-ks,Coudray2018-jm,Loeffler2022-xj}. It is well established that tumours exhibit characteristic histological structures indicative of their natural history \pcref{fig:intro-cancer-macrostructures}. Many cancers exhibit adjacent areas of carcinoma in situ, which are considered to be precursor lesions. This phenomenon is well established in melanoma \parencite{Nirmal2022-sq} and many adenocarcinomas: in breast cancer up to two-thirds of cases have histologically distinct areas of ductal carcinoma in situ \parencite{Kole2019-hl}, around 15\% of colorectal adenocarcinomas exhibit adjacent adenomas \parencite{Ponz_de_Leon1990-zy}, and ~45\% of oesophageal adenocarcinomas show detectable areas of its precursor lesion Barrett oesophagus \parencite{Sawas2018-mt}. Interestingly, the phylogenetic and histological relationships between precursor lesions and invasive cancer are not always aligned with patterns indicative of branching \parencite{Ross-Innes2015-sq,Stachler2015-ca} or multiclonal invasion \parencite{Casasent2018-gx}. Therefore, spatial genomic and transcriptomic analyses using histologically informed dissection allow for the exploration of mechanisms and phenotypes of disease progression in greater detail, revealing subclone and histology-specific changes \parencite{Lomakin2022-ks}.

\figuretextwidth{intro-cancer-macrostructures.pdf}{intro-cancer-macrostructures}
    {Tumour macrostructure \parencite{Seferbekova2023-wg}}
    {Different tumour regions can be identified in some tumours based on histological staining. Such histopathological findings can be supplemented and underpinned by detailed molecular maps}

At the scale of millimetres, cancers are often described in terms of a tumour core that is separated from the surrounding stroma by an invasive margin \pcref{fig:intro-cancer-macrostructures}. Transcriptional diversity has been reported across these landscapes \parencite{Berglund2018-gh}, with hypoxia and \ac{EMT}-like signatures localizing to the core and invasive margin, respectively \parencite{Puram2017-mn,Thomlinson1955-em}. The core–stroma distinction is fundamental to common classifications of immune infiltrates that correlate with cancer subtype, clinical outcome and therapy response \parencite{Hammerl2021-fs,Keren2018-or,Galon2014-oi}. At one end of the immune infiltrate continuum are inflamed tumours, where plentiful immune cells infiltrate the tumour core, at the other end are immune deserts that are devoid of infiltrates, and between these are immune-excluded tumours where immune cells are largely restricted to the neighbouring stroma \parencite{Hegde2020-sw}. However, recent spatial studies in pancreatic and breast cancer reveal that these immune characteristics can be highly localised and coexist in different parts of the same tumour \parencite{Danenberg2022-zb,Grunwald2021-zk,Tavernari2021-yb}. A challenge will therefore be to decipher how localised variation in immune response shapes the evolutionary outcome of the tumour.

\section{Evolution of the cancer ecosystem}

The cancer ecosystem detailed in the previous section changes the dynamics of cancer evolution as tumour cells face selection pressures that maximize growth-promoting and minimize growth-inhibiting interactions. Foremost are cell–cell and other local interactions with the immune system but also with fibroblasts, vasculature and even neurons, which can lead to the selection of specific genomic alterations \pcref{fig:intro-cancer-tme-coevolution}. The fact that the \ac{TME} is itself spatially heterogeneous suggests that tumour cells experience localised variations in selective pressure.

\figuretextwidth{intro-cancer-tme-coevolution.pdf}{intro-cancer-tme-coevolution}
    {Co-evolution of tumour cells and microenvironments \parencite{Seferbekova2023-wg}}
    {Some tumour subclones attract characteristic \acfp{TME}, which subsequently spatially overlap with the clonal territories. Alternatively, microenvironments may select for specific tumour characteristics such as immune evasion; here, immune evasion is shown as impaired antigen presentation due to loss of expression of human leukocyte antigen 1 (HLA1).}

\subsection*{Evolution of immune evasion}

Various parts of the immune system constrain tumour growth. The observed range of associations between specific genetic alterations and the immune system cells of the \ac{TME} \parencite{Rooney2015-yb,Thorsson2019-fk} illustrates the presence of selective pressures on cancer evolution. Neoantigens, which derive from coding mutations in cancer genomes, have been reported to elicit T cell reactivity in lung cancer \parencite{McGranahan2016-yd}. In tumours with greater cytolytic activity, loss of neoantigen presentation capabilities was more frequent, which is indicative of greater selective pressures \parencite{Rooney2015-yb,Shukla2015-kl}. Other signs of selection for immune evasion include mutations in \gene{CASP8}, which serve as a potential way to resist apoptosis induced by \protein{FASL} \parencite{Rooney2015-yb}. Furthermore, gains of immunosuppressors, such as \protein{CD274} (also known as \protein{PDL1}), are widespread among Hodgkin lymphomas and are detected in ~0.7\% of solid tumours \parencite{Goodman2018-mn}.

The signs of \explain{immune evasion}{The process by which tumour cells develop several mechanisms that help them to continue to grow and expand by escaping immune control. Immune evasion is thought to be typically preceded by two other phases: elimination, when the immune system recognises and eliminates tumour cells, and equilibrium, when the pressure from the immune system stalls tumour growth and expansion, and negative selection takes place} increase during later stages in cancer evolution. No evidence of \explain{negative selection}{The removal of deleterious variants from the population} was reported in diploid genomes of normal tissues and among clonal alterations in primary cancers, indicating that there is no depletion of coding mutations during normal evolution \parencite{Martincorena2017-uw,Van_den_Eynden2019-aq}. Yet, the signs of selection for immune evasion become stronger after clonal expansion at the subclonal level. For example, immune evasion via deletions of \ac{HLA} class I alleles, predominantly subclonally and in metastases, has been described in lung cancer \parencite{McGranahan2017-ob} and multiple other cancer types \parencite{Watkins2020-gc} \pcref{fig:intro-cancer-tme-coevolution}. This mode of evolution is plausible as it may require a certain tumour size for the immune system to recognise its neoantigens; at this stage, however, a mutation enabling escape from these new selective pressures arises in a single cell of the tumour and seeds the growth of a tumour subclone. While it is possible for it to replace all other tumour cells, the timing at which this occurs depends on the strength of immune control and its escape rate.

Furthermore, selective pressures of the immune system determine prognosis and immune evasion is found in relapses. It has been reported that strongly immunogenic neoantigens are prognostically favourable in pancreatic cancer \parencite{Balachandran2017-zq} but are preferentially lost at recurrence \parencite{Luksza2022-jj}. In melanoma, mutations of \gene{JAK1} or \gene{JAK2}, which inhibit cytolysis, and in \gene{B2M}, which reduce antigen presentation, are often found in samples that acquired resistance to immune-checkpoint inhibitors \parencite{Sade-Feldman2017-mj,Zaretsky2016-rs}.

\subsection*{Maps of cancer–\ac{TME} interactions}

Establishing associations between genomic alterations and the microenvironment can be challenging due to the large genomic heterogeneity between primary cancers. Conceptually, spatial genomics offers new ways of analysing such interactions at high spatial resolution and also within the same tumour. For intratumour diversity, the observations occur in an isogenic background and the set of genomic differences between subclones is small compared to the large intertumour diversity \pcref{fig:intro-cancer-tme-observations}. A proof of concept has been established in breast cancer, where various degrees of immune infiltration were found at the level of cancer subclones \parencite{Lomakin2022-ks}. Interestingly, such observations could also be linked to the stage of invasion, with some \ac{TME} regions defined by the subclonal lineage and others mostly by histological progression from carcinoma in situ to invasive cancer. Direct experimental evidence was provided by Perturb-map, a mouse in vivo spatial CRISPR screen, which revealed that specific gene knockouts lead to characteristic changes in the immune microenvironment. For example, it was shown that T cell infiltration is increased by \ko{Socs1} knockout in the tumour but decreased by \ko{Tgfbr2} knockout \parencite{Dhainaut2022-nj} \pcref{fig:intro-cancer-tme-observations}.

\figuretextwidth{intro-cancer-tme-observations.pdf}{intro-cancer-tme-observations}
    {Co-evolution of tumour cells and microenvironments \parencite{Seferbekova2023-wg}}
    {Examples of experimentally observed associations between clones and \ac{TME}s. The yellow \ko{Tgfbr2} knockout tumour clone exhibits a fibroblast-enriched stroma with no presence of T cells, whereas the green \ko{Socs1} knockout clone has a stroma infiltrated with T cells \parencite{Dhainaut2022-nj}.}

Further cell types beyond immune cells have been associated with cancer evolution. Fibroblasts are an extremely versatile group of cells that perform various roles that are exploited during tumour growth. For example, cancer-associated fibroblasts can be educated by cancer cells with gain-of-function mutations in \gene{TP53} to create a pro-tumorigenic microenvironment \parencite{Vennin2019-cf}. Furthermore, fibroblasts can amplify tumour-intrinsic oncogenic signalling in \gene{KRAS}\textsuperscript{G12D}-mutant pancreatic adenocarcinoma \parencite{Tape2016-hb}. Spatial genomics adds to these observations. In breast cancer, imaging mass cytometry \pcref{box:technologies} revealed enrichment of different fibroblast and myofibroblast populations in \gene{TP53}-mutant and genomically unstable cancers \parencite{Ali2020-zo}; however, the nature of those associations is unclear. The aforementioned Perturb-map screening revealed that \ko{Tgfbr2} knockout created a fibro-mucinous stroma characterized by TGF$\beta$-activated fibroblasts and T cell exclusion \parencite{Dhainaut2022-nj}.

Another cell-extrinsic and environmental factor influencing cancer evolution is hypoxia, the often localised depletion of oxygen in tumours; it is well established as a marker of poor outcomes \parencite{Harris2002-ty} and has been shown to select for certain mutations such as \gene{TP53} \parencite{Graeber1996-em}. A recent study combining \ac{scRNAseq} and immunofluorescence found that hypoxic microniches contain quiescent cancer cells resistant to T cell attack \parencite{Baldominos2022-xe}.

Lastly, spatial genomics may also shed light on how different cancer subclones interact with each other. Reports of clonal cooperation, observed in heterologous xenotransplants, date back to at least 1989 \parencite{Miller1989-hp}. Elements of clonal cooperation have since been described as involving direct and microenvironmentally mediated interactions \parencite{Alonso-Curbelo2021-dq,Cleary2014-hg,Zhou2017-nm,Tammela2017-nd,Williams2020-fj}. The ability to comprehensively map genomic evolution, subclone-specific gene expression and \ac{TME}s is thus likely to further elucidate the full extent and mechanisms of this phenomenon.

\subsection*{Disseminated tumour cells, metastatic niches and organotropism}

Metastases are seeded by \acp{DTC}. The mechanisms that determine whether any given \ac{DTC} survives, enters a dormant state or forms a metastasis are poorly defined, but it is evident that these cells can exploit a variety of existing cellular interactions and spatial microenvironments. For example, dissemination was shown to depend on environmental triggers tied to the circadian rhythm \parencite{Diamantopoulou2022-sb}. In breast cancer, \acp{DTC} physically associated with neutrophils were shown to exhibit a more aggressive phenotype \parencite{Szczerba2019-mt}. Additionally, aggregates or even fusions of \acp{DTC} and macrophages have been described, further illustrating that metastatic dissemination may also rely on a range of cellular interactions \parencite{Adams2014-xq}.

\figuretextwidth{intro-organotropism.pdf}{intro-organotropism}
    {Organotropism and the seed and soil hypothesis of metastasis \parencite{Seferbekova2023-wg}}
    {Cancer cells from the primary tumour migrate throughout the body via blood vessels, with some of them able to seed metastatic lesions in a distant organ. The target TME determines the fitness of clones at the site of metastasis favouring some tumour lineages over others.}

The existence of organotropism, whereby certain cancer types and specific molecular alterations exhibit distinct patterns of metastatic organ involvement, is well established and is often considered in terms of the classical seed and soil hypothesis suggested by Paget, which posits that certain tissues provide host environments susceptible to specific tumour types \parencite{Fidler2003-lh,Paget1889-ha} \pcref{fig:intro-organotropism}. Organotropism seems to be mostly determined by the cell of origin of the primary tumour, the ability of \acp{DTC} to interact with the metastatic host environment and the anatomical proximity of certain target organs such as liver metastases in colorectal cancer. In breast cancer, brain metastases were shown to be facilitated by the interactions with glutamatergic neurons \parencite{Zeng2019-vf}, and a similar pattern was also observed in primary brain cancers where neoplastic cells form networks with normal astrocytes via synapses \parencite{Venkataramani2022-kg}.

The notion that both intrinsic and microenvironmental features of cellular populations in the primary tumour influence metastatic rates is further supported by lineage-tracing studies in mouse models \parencite{Quinn2021-lu}. However, a genomic element also emerges. A recent pan-cancer analysis of 25,000 metastases of 10 cancer types revealed 57 genetic associations of organotropism within the same primary tumour type \parencite{Nguyen2022-jr}. Although the presence of many genomic alterations was associated with increased rates of metastasis in multiple organs, there were also examples of genomic alterations associated with reduced burden, for example, \gene{RBM10} mutations being less prevalent in brain metastases of lung cancers. However, it is worth noting that these genetic patterns appeared to be mostly specific to certain primary tumour sites rather than to target sites.

A series of studies reported that tumours prime target tissues to create pre-metastatic niches facilitating colonisation and metastatic outgrowth \parencite{Peinado2017-hz}. This occurs prior to or on the arrival of \acp{DTC} through soluble factors and extracellular vesicles released by the primary tumour that can alter the systemic immune response and induce localised regions of vascular and stromal remodelling in target organs. Furthermore, \acp{DTC} can enter dormant niches that facilitate long-term survival and even evasion of therapy. In prostate cancer, \acp{DTC} can compete with haematopoietic stem cells for the occupation of established endosteal niches \parencite{Shiozawa2011-cm}, whereas perivascular niches have been observed in bone, lung and brain \parencite{Kienast2010-ur,Ghajar2013-id}. In melanoma, age-dependent changes of fibroblasts have been reported to induce emergence from dormancy in lung deposits \parencite{Fane2022-sd}. Similarly, in a mouse model of melanoma, metastases in lymph nodes were found to promote immune evasion and facilitate further metastasis in distant organs \parencite{Reticker-Flynn2022-ho}. As all of these phenomena involve cellular and spatial interactions, spatial genomic, transcriptomic and proteomic analyses are likely to provide further insights into these processes.

\section{Translational opportunities}
As discussed in the preceding sections, spatial genomic methods generate rich insights into the cellular composition, organisation and interactions that shape the evolving cancer ecosystem. For any given cancer, the genome bears the scars of its unique, past evolutionary journey; however, clinical interventions are designed to shape the future disease course \pcref{fig:intro-translations}. A major hope for cancer care is therefore that evolutionary features might be leveraged to better predict the clinical trajectory. Because tumour evolution is a spatial process, spatially resolved features may provide additional means to predict outcomes. Furthermore, a better understanding of the cellular interactions within the ecosystem might identify potential therapeutic vulnerabilities that could be exploited to control the evolutionary process \parencite{Andersson2021-pu,Moncada2020-ck,Ji2020-gn,Van_Maldegem2021-ta,Moldoveanu2022-qu}. Initial studies hint towards the opportunities afforded by these approaches for novel biomarker discovery \parencite{Danenberg2022-zb,Keren2018-or, Moldoveanu2022-qu}, in understanding the clinical relevance of genetic subclonal structure \parencite{Zhao2022-xd,Lomakin2022-ks,Erickson2022-zh} and in identifying therapeutic vulnerabilities through a deepened understanding of cellular interactions in both human samples \parencite{Moncada2020-ck,Ji2020-gn,Keren2018-or} and preclinical animal models \parencite{Dhainaut2022-nj}.

\figuretextwidth{intro-translations.pdf}{intro-translations}
    {Translational opportunities \parencite{Seferbekova2023-wg}}
    {Subclone maps, \acf{TME} composition and histological context each provide clinically and prognostically relevant features. Integrating these levels of information may help identify ill-fated clonal lineages that may not be found using either component alone. For example, the metastatic potential of two invasive subclonal lineages may differ based not only on the portfolio of their driver mutations but also on the \ac{TME}s that they inhabit. \ac{TIL}s, tumour infiltrating lymphocytes; \ac{TLS}, tertiary lymphoid structure.}


\infobox{box-analysis.pdf}{box:analysis}{Spatial omic analysis}{
    A typical tumour cross-section covers several centimetres of tissue containing up to 1,000,000 cells. Spatial omic methods characterize the molecular features of these cells with a resolution as fine as ~0.2 $\mu$m, depending on the method. The wealth and detail of data bring a series of data analysis challenges \parencite{Palla2022-gk} (see the figure).

    \subsubsection*{Raw signal analysis}

    In the case of molecular barcode-based technologies \pcref{box:technologies}, primary steps of raw data processing include aligning and counting reads or molecular barcodes per genomic region of interest, with sequencing barcodes defining the spatial sector. Image-based methods, by contrast, utilize fluorescent signals, often measured across multiple fluorescence channels and cycles. These signals must be detected, quantified and decoded to their corresponding target molecule, which can be defined by a specific combination of colours across cycles. For whole-slide imaging, fields of view need to be stitched together into one tissue image \parencite{Lin2021-mz}. Background fluorescence, blur and signal crowding can negatively affect the ability to decode signals \parencite{Ke2013-ux,Gataric2021-yt,Dries2021-ps,Perkel2019-vp,Hickey2022-wj}. Technologies with a super-cellular resolution require cell decomposition algorithms, where known cell types usually obtained from the single-cell data serve as a reference \parencite{Lomakin2022-ks,Moncada2020-ck,Kleshchevnikov2022-ub,Nieto2021-pd}. Signals with a subcellular resolution are spatially aggregated either by clustering \parencite{Park2021-hi,Petukhov2022-pv,Soldatov2019-wj,Prabhakaran2022-qe} or based on cell segmentation derived from nuclear DAPI signals, membrane markers or haematoxylin and eosin (H\&E) staining \parencite{Qian2020-mp,Petukhov2022-pv,Schapiro2022-nc,Lohoff2022-du}.

    \subsubsection*{Data integration}

    Single-cell resolved data can be mapped \parencite{Zhang2022-fv} to deconvolve larger spots or to annotate known cell types. Single-cell whole transcriptomes or multi-omics can be further mapped into space, leveraging a smaller common set of reference genes that are interrogated spatially \parencite{Zhang2022-fv, Stuart2019-mi}. Another option is to integrate multiple spatially resolved modalities from consecutive tissue sections. Alignment of distorted sections and 3D reconstruction of a tissue block can be done using image registration algorithms \parencite{Kiemen2020-dc}. Multimodal characterisations of microscopically matched tissue areas, including spatial transcriptomics, proteomics and histopathology, are also possible \parencite{Lomakin2022-ks,Velten2022-gc}.

    \subsubsection*{Spatial statistics}

    Spatial differential expression models enable the description of novel cell states showing distinct spatial patterns that are difficult to recover from single-cell data \parencite{Svensson2018-eu,Lopez2022-ar}. A group of cells may appear enriched in a recurrent way forming distinct neighbourhoods \parencite{Danenberg2022-zb,Nirmal2022-sq,McCaffrey2022-ui,Dong2022-ty}, a pattern which could be picked up by various topic and graph-based models. Mechanistic models attempt to explain expression plasticity mediated by cell–cell interactions and signalling at the cell interface \parencite{Arnol2019-fv}.

    \subsubsection*{Spatial lineage tracing}

    Spatial RNA or DNA sequencing technologies provide wide but low genomic coverage, which limits lineage tracing to copy number alterations and a few point mutations \parencite{Zhao2022-xd,Nagasawa2021-tu,Elyanow2021-tq}. Current protocols also have a relatively low spatial resolution \pcref{box:technologies}. Microscopy methods target specific point mutations and require prior knowledge of the clonal structure. Yet, they achieve a higher spatial resolution to map a more fine-grained clonal structure across wide fields of view \parencite{Lomakin2022-ks}. The incremental nature of branching evolution also provides opportunities to associate the genetic changes on each branch with phenotypic changes.
}
    

\subsection*{Spatial biomarker discovery}

Spatial biomarkers predict clinical outcomes by leveraging information about cellular organisation or intercellular relationships. In its most simplistic form, this approach might measure the distribution of a certain cell type in relation to another. This is the basic concept of commonly used, histomorphological scoring of \acp{TLS} or \acp{TIL} within the tumour or stromal compartment \pcref{fig:intro-cancer-neighbourhoods}. Stromal \acp{TIL} are associated with a superior prognosis and response to chemotherapy or immune-checkpoint inhibitors in triple-negative breast cancer, non-small-cell lung cancer and melanomas \parencite{Azimi2012-aa,Lee2016-hg,Helmink2020-gz,Chen2020-ua,Salgado2015-ne}. However, in isolation, the reliability of \acp{TIL} as a biomarker is hampered by heterogeneity and irreproducibility \parencite{Salgado2015-ne}. Recent studies that use highly multiplexed, in situ, proteomic-based assays have started to dissect this complexity by characterizing \ac{TIL} subtypes, expression, cellular co-localization and neighbourhood patterns, identifying prognostically meaningful \ac{TIL}-related patterns \parencite{Moldoveanu2022-qu,Keren2018-or}. In general, head-to-head clinical trials are needed to confirm whether multiplex biomarkers carry additional prognostic and predictive value compared to single biomarkers such as \protein{PDL1} immunohistochemistry \parencite{Lu2019-lo}. Notably, the Immunoscore \parencite{Galon2006-ow} is a digital pathology-based assay that measures \protein{CD3}+ and \protein{CD8}+ lymphocyte density in the tumour core and invasive margin and has been proven to surpass clinical staging systems for disease-free survival prediction, demonstrating that higher-dimension datasets could be clinically transformative \parencite{Pages2018-ka}.

Beyond a role in refining histology-based biomarkers, spatial omic approaches can also facilitate de novo biomarker discovery. This was demonstrated by two breast cancer studies (discussed in the \cref{sec:mapping-the-cancer-ecosystem}) that extracted recurrent cellular communities from multiplex proteomic and morphological data using machine learning algorithms \parencite{Danenberg2022-zb,Jackson2020-em}. Another use of multiplexed tissue imaging was an ovarian cancer study, which found that interactions of exhausted \protein{CD8}+ T cells, \protein{PDL1}+ macrophages and\protein{PDL1}+ tumour cells were linked to treatment response \parencite{Farkkila2020-gk}. In these studies, the power to detect clinically meaningful features was derived from the fact that hundreds of samples could be analysed at high throughput by using tissue microarrays. However, the trade-off was that only a tiny part of the main tumour mass was sampled (core needle biopsies with a diameter of 0.6–0.8 mm), whereas the invasive margin and stroma were largely unsampled. An important next step for spatial molecular analyses is to derive catalogues of typical cancer ecosystems at a tissue-wide scale and at single-cell resolution. These reference sets could serve as a basis for designing more focused studies that might take advantage of lower-plex, higher-throughput or rational subsampling approaches. The scalability and reproducibility of omic data will be critical factors for successful implementation within large-scale clinical studies.

\subsection*{Clinically relevant subclones}
An unanswered question for personalized medicine is how to deal with subclonal driver mutations that can be targeted by antitumour therapy. By integrating additional layers of spatial data, it is possible to establish the spatial context and thus the clinical relevance of subclonal alterations \parencite{Lomakin2022-ks,Gonzalez-Silva2020-fn} \pcref{fig:intro-translations}. For example, systemic therapies administered after primary breast cancer surgery are directed towards covert metastatic disease, with the intention of eliminating these cells before metastatic deposits have a chance to form. As we cannot directly assay disseminated tumour cells, treatment paradigms are based upon the properties of the primary tumour. It is reasonable to assume that a subclonal mutation within the invasive cancer might also be carried by disseminated tumour cells and could drive metastasis formation. By contrast, if the mutation is entirely restricted to precursor lesions, and hence a clone that has not yet even exhibited invasive capacity, we can be relatively confident that it will not be a driver of metastasis.

Integrating spatial transcriptomic or proteomic data provide further insights into the \ac{TME} and the gene expression properties of subclones. This additional characterisation will be helpful for deriving predictions of the functional relevance of a particular subclonal driver mutation, such as by considering a range of intrinsic and extrinsic phenotypes, including vascular invasion or a high-risk \ac{TME} niche \pcref{fig:intro-translations}. Although further research is needed, it is possible that spatial analyses could be used in early detection settings or to complement histopathology-determined completeness of excision, with the latter application yielding opportunities for biomarker-driven surgical and adjuvant radiotherapy clinical trials.

\subsection*{Preclinical studies}

Although spatial genomic analysis enables reconstruction of the evolutionary past and prediction of the clinical future, spatial genomic experiments in animal models can add direct mechanistic evidence of the roles of particular cancer-associated mutations and/or the effects of spatial aspects of the cancer \ac{TME}. Such experiments are expected to provide clinically important insights into the extracellular consequences of perturbations in cell-intrinsic processes, with many opportunities for biomarker discovery \parencite{Van_Maldegem2021-ta,Janiszewska2019-zq}. A particularly exciting approach is the combination of spatial profiling with in vivo CRISPR screens \parencite{Ji2020-gn,Dhainaut2022-nj}. Using this approach, both cell-intrinsic phenotypic and local TME associations of tens of genetically distinct clones can be studied simultaneously in a single animal, leading to insights into the precise mechanisms that lead to diverse clinical fates of genetically divergent clones that nonetheless share many genetic and host characteristics \parencite{Dhainaut2022-nj}. These models could lead the way in developing pan-ecosystem therapeutic strategies that simultaneously target cell-intrinsic vulnerabilities and harness the properties of the \ac{TME} in an attempt to terminate cancer evolution.

\section{Conclusions and future perspectives}

While it is widely accepted that the population genetics of carcinogenesis can be well described in terms of somatic evolution, the spatiotemporal details of the process are not fully understood. Open questions remain over the extent to which cellular interactions with the \ac{TME} and micro-anatomical constraints influence cancer evolution. This situation begins to change with the availability of spatial genomic, transcriptomic and proteomic technologies that enable charting of the growth patterns of distinct subclones and characterisation of the composition and structure of their microenvironments. Connecting these different levels of intratumour heterogeneity is therefore key to understanding how tumours grow as it reveals how tumour subclones interact with their microenvironment and how this may lead to the selection of aggressive traits.

Although there is a broad range of technologies with different advantages and disadvantages, it emerges that cellular resolution is essential for understanding the micro-anatomy of the cancer ecosystem and localizing molecular signals to specific cells. Mapping how cells are spatially organised into subclones and neighbourhoods and how different cells interface is a key step for understanding the transcriptional diversity revealed by catalogues of \ac{scRNAseq} studies. Another consideration is the field of view. A typical primary tumour is more than 1 cm in diameter at diagnosis; hence, technologies enabling the mapping of entire tumour sections are desirable. Lastly, enabling spatially resolved genomics with the resolution of single nucleotides is important for mapping the subclonal landscape. Whereas spatial transcriptomic and proteomic approaches are reaching maturity and are available as commercial platforms, spatial genomics lags behind. A reason for this is the limited amount of DNA per cell, compared to the much greater number of copies of RNA and protein, which may be compounded by tissue sections only containing fractions of a nucleus. Even though it is possible to calculate copy number alterations using low-coverage DNA sequencing or even RNA sequencing, it remains challenging to estimate phylogenetic relationships and distances. Developing computational approaches such as integrating mathematical modelling \parencite{Gatenbee2022-hb}, mathematical oncology and artificial intelligence-based algorithms may help to overcome these challenges and generate further quantitative insights.

These challenges notwithstanding, the opportunities for spatial cancer genomics are formidable. Charting the tumour ecosystem with clonal resolution as well as cellular and molecular details of the \ac{TME} will have great value. Doing so across whole tumour sections and cancer types will create insightful atlases that not only provide snapshots of cancer composition but also insights into past tumour evolution. In addition to a more detailed understanding of tumour growth, these atlases have the potential to reveal the mechanism by which the microenvironment is co-opted by tumour cells and how the immune system suppresses tumour proliferation. These processes are already exploited by immune-checkpoint inhibitors, and enhanced molecular and spatial detail may yield further therapeutic targets or facilitate more effective use of existing therapies. Furthermore, the derivation of spatial biomarkers and the detection of diagnostic applications of spatial genomics are likely to inform treatment; this is especially the case now that spatial genomic, transcriptomic, proteomic and metabolomic methods are starting to be used in clinical trials. Such a multimodal approach is required to elucidate the inherent complexity of the \ac{TME} — the result will be a revolution of histopathology that is genomically and molecularly informed.

The availability of various single-cell and spatially resolved assays and the emerging insights that these approaches offer thus warrant new concerted efforts. Previous initiatives, such as the \textcite{Cancer_Genome_Atlas_Research_Network2013-ox}, the \textcite{International_Cancer_Genome_Consortium2010-ww, ICGCTCGA_Pan-Cancer_Analysis_of_Whole_Genomes_Consortium2020-vu}, have created valuable community resources detailing the molecular and genomic properties across cancers. It thus seems natural for emerging projects, such as The Human Tumour Atlas Network \parencite{Rozenblatt-Rosen2020-bl}, to follow in this spirit in order to create the next generation of molecularly resolved cancer atlases.

\section{Aims of this thesis}

As discussed in the previous section, spatial genomics is a rapidly evolving field with many opportunities for the study of the open quesitons in cancer evolution. 

In this thesis, I aim to develop and apply computational methods to analyse spatially resolved genomic data in order to gain insights into the evolutionary dynamics of cancer. 

In \cref{sec:chapter-basiss-model} I

In \cref{sec:chapter-basiss-multimodal} I

In \cref{sec:chapter-basiss-applications} I

In \cref{sec:chapter-discussion} I









