\chapter{Introduction}

\section*{Contributions}

\section{Componentes of cancer evolution}

The drivers of evolution are mutation and selection \parencite{Cairns1975-oz,Nowell1976-sm}. Mutation of somatic cells is an inevitable and persistent consequence of life \parencite{Martincorena2015-br, Moore2021-yr, Li2021-th}. In most normal tissues, mutations accumulate at a steady rate of around 15–50 per cell per year of life, with only the germline known to exhibit lower rates \parencite{Moore2021-yr}. Tumour cells often exhibit an elevated mutation burden due to the variety of mutational processes they have been exposed to during life and, in some cases, due to acquired hypermutation \parencite{Alexandrov2020-uo}. The continuous accumulation of mutations inevitably leads to diversification at the level of single cells both in tumours and normal tissues.

The second force of evolution is selection, which describes how a fitter lineage outgrows its relatives. Selection operates at the level of a wide range of heritable phenotypes that derive from genomics and epigenomics. Clones with a selective advantage in the environment to which they are exposed will expand, while those with a disadvantage will tend to disappear \cref{fig:intro-spatial-biology-general}. Advantageous variants are therefore enriched in genomes of aged normal tissues and cancers \parencite{Greenman2006-cx,Martincorena2017-uw}. More than 500 so-called cancer driver genes have been reported to date \parencite{Lawrence2013-mi,Gonzalez-Perez2013-vo,Sondka2018-xf}, which are believed to cause different cancer hallmark traits and enabling characteristics that involve cell-intrinsic mechanisms as well as interactions with the \ac{TME} \parencite{Hanahan2000-ze, Hanahan2022-eb}.

While mutation continuously generates subclonal diversity at the level of single cells, it is well documented that tumours are mosaics of subclones each comprising hundreds of thousands of cells that have arisen from a shared ancestor \parencite{Shah2009-xz,Nik-Zainal2012-zz,McGranahan2015-gb,Andor2016-hi, Dentro2021-yb} \cref{fig:intro-spatial-biology-general}; these patterns have been further confirmed by single-cell studies \parencite{Navin2011-qq,Wang2014-bp,Casasent2018-gx,McPherson2016-zu,Laks2019-rp,Salehi2021-ad}. The mechanisms of these expansions remain debated. It has been shown that cancer subclones exhibit signs of positive selection and driver gene mutations that can also be found clonally \parencite{Dentro2021-yb}. However, it is also conceivable that subclones branching at early stages of tumour development reach considerable size without a selective advantage as they continue to expand with the same tumour, a phenomenon termed neutral evolution \parencite{Williams2016-qh}.

Subclonal mosaicism is often found to be spatially variegated as demonstrated in diverse studies based on tumour macrodissection \parencite{Navin2010-uw,Gerlinger2012-qm,De_Bruin2014-ow,Gerlinger2014-hr, Yates2015-xk,Morrissy2017-vy,Jamal-Hanjani2017-uv,Watkins2020-gc} and laser-capture microdissection \parencite{Casasent2018-gx, Heide2022-ev,Woodcock2020-nt,Grossmann2021-sl,Zhao2022-xd,Heide2019-rr,Su2018-rr,Bao2018-kj} \cref{fig:intro-spatial-biology-general}. These clonal variegation patterns were shown to be accompanied by differential gene expression in multiple cancer types, including renal \parencite{Gerlinger2012-qm}, colorectal  \parencite{Househam2022-cp}, lung \parencite{Biswas2019-kb} and breast cancer \parencite{Lomakin2022-ks}, using spatially resolved genomic and transcriptomic analyses, indicating the presence of subclone-specific gene expression and associations between certain subclones and characteristic \ac{TME}s.

The exact mechanisms creating these phenomena are not fully understood, largely owing to a dearth of methods that can molecularly characterize whole tumour sections with spatial resolution. A variety of non-exclusive explanations have been proposed. A rapid expansion could lead to a fragmentation of closely related clones \parencite{Sottoriva2015-ci} even though it has also been argued that cell dispersal is a crucial element shaping tumour structure \parencite{Waclaw2015-yw,Gallaher2019-xx}. However, it is also possible that tissue micro-anatomy, such as the ductal system of the breast, provides a template for clonal segregation (discussed further in the section ‘Tissue organization controls evolution’). Lastly, it is conceivable that locally distinct microenvironments or niches favour the selection of particular clones.

Various new spatially resolved genomic, transcriptomic and proteomic technologies offer novel insights and answers to these unresolved questions \cref{fig:intro-spatial-biology-general}. The technological aspects enabling the nascent field of spatial cancer biology are summarized in Box 1 and have been extensively reviewed elsewhere \parencite{Lewis2021-ic}. Individual technologies differ in the multiplexity of their readouts and their spatial resolution as well as in their sensitivity and available field of view. Combining genomic, transcriptomic and proteomic layers enables a rich characterization of tumour ecosystems.

In this Review, we focus on the spatial aspects of cancer evolution, summarize how spatial transcriptomics and proteomics reveal a complex landscape of the tumour ecosystem, discuss how interactions with the \ac{TME} are integral to cancer evolution, and propose potential clinical applications and future directions. We focus particularly on the role of tissue micro-anatomy in controlling the rate of evolution as well as cellular interactions with the \ac{TME} within which cancer cells evolve.

\section{Tissue organisation controls evolution}

The rate at which malignant clones emerge and spread through the tissue is controlled by its organization. Levels of organization include the micro-anatomical tissue architecture as well as differentiation hierarchies with few stem cells feeding successively larger pools of differentiated cells. The breakdown of these protective principles is a key feature of cancer development. However, the resident tissue structure, for example, the ductal system in the breast or epithelial layers of the oesophagus, can also influence the rate of progression at pre-invasive stages. The transformation ends with overwhelming metastatic disease.

\subsection*{Normal tissue organization suppresses evolution}

It has long been hypothesized that somatic tissues are structured to suppress the rate of somatic evolution \parencite{Cairns1975-oz}. These ideas are based on fundamental theoretical considerations that the rate of evolution depends on population structure. Generally, the rate of evolution is determined by the rate at which new mutations are generated (a product of the number of cells and the mutation rate per cell), the probability that new mutants sweep through the population \parencite{Lieberman2005-cy} and the time it takes to do so \parencite{Frean2013-rq,Tkadlec2019-cp}. In addition to suppressing the mutation rate, differentiation hierarchies can reduce the chance of emergence and slow the spread of mutant lineages. The haematopoietic system constitutes a basic example \parencite{Lopes2007-je}: its 1013 cells derive from around 100,000 haematopoietic stem cells \parencite{Lee-Six2018-pn,Sender2016-og}, which themselves divide slowly and produce a range of faster-dividing differentiated cells \parencite{Humphries2008-jj}. This hierarchy reduces the rate of evolution, as only mutations in the comparably small number of stem cells and potentially the first progenitors, or mutations that lead to a differentiation blockage, will remain in the population. The colon extends these organizational principles by its compartmentalization into micro-anatomical crypts; in each crypt, a similar differentiation hierarchy operates in which a very small number of stem cells replenishes the colonic epithelium \parencite{Humphries2008-jj}. As differentiating cells are fated to die within ~5 days, only mutations arising in the stem cells persist \parencite{Cairns1975-oz,Nowak2003-pq,Sender2021-ow}. Furthermore, micro-anatomy largely constrains the fixation of stem-cell mutations to individual crypts \cref{fig:intro-normal-tissue-evolution}.

Systematic studies of normal tissues confirm that, in tissues with a more complex architecture, clones typically remain small in size and develop independently in comparison to less structured tissues where clonal expansions tend to be larger \parencite{Li2021-th}. In a recent investigation of 517 normal colonic crypts, only 1\% harboured cancer driver gene mutations, and those that were mutated were confined to single crypts \parencite{Lee-Six2019-fy}. Similar observations have been made for glandular tissues such as the endometrium \parencite{Moore2020-qb}: although most of the glands harboured driver mutations, they were limited to a single mutation or a few neighbouring mutations. Conversely, it is well established that the haematopoietic system harbours mutant lineages whose frequencies can reach up to 100\% \parencite{Genovese2014-wm,Jaiswal2014-tz}. Other epithelial tissues, including the skin, oesophagus or bladder, have an intermediate degree of tissue structure and are patchworks of microscopic clones \parencite{Moore2021-yr,Martincorena2015-yu,Martincorena2018-rl,Lawson2020-as} \cref{fig:intro-normal-tissue-evolution}. In such tissues, the fixation probability is high but the time to fixation is greater than in the haematopoietic system because the dynamics are confined to two dimensions, which reduces the interfaces at which a fitter variant may replace less-fit competitors. Of note, not all mutations causing a selective advantage are associated with malignant transformation. For example, in the skin and oesophagus, NOTCH1 mutant clones expand and suppress malignant evolution while maintaining normal tissue function \parencite{Martincorena2015-yu,Martincorena2018-rl, Colom2021-mk,Fowler2021-bl, Abby2021-ik}.

\subsubsection*{Breakdown of evolutionarily confining tissue architectures in cancer
}
The fundamental observation that dedifferentiation and the loss of normal tissue architectures are correlated with cancer aggressiveness was made a century ago and still forms part of modern histopathological grading systems that direct cancer treatment \parencite{Louis2007-ia,Elston1991-md,Epstein2016-un,Greenough1925-wg}. In these systems, higher grades correspond to more aggressive tumours, typically composed of less differentiated cells.

The breakdown of such evolutionarily confining tissue structures is a hallmark of cancer. It is especially noticeable among carcinomas, which derive from epithelial tissues \cref{fig:intro-carcinogenesis}. At the earliest stages of progression, so-called carcinomas in situ are still restrained to the resident tissue structures, such as the duct of the breast. Similar to the considerations of normal tissues, the resident tissue structures may delay clonal sweeps. This tissue-based restraint was observed in a case of breast cancer, where spatial genomic analysis revealed two distinct subclones occupying a checkerboard pattern of micro-anatomical niches within the ductal system \cref{Lomakin2022-ks}.

During invasion, tissue architectures are lost, probably via different mechanisms in each tissue type, but a frequent property of carcinomas is the \ac{EMT}. \ac{EMT} enables epithelial cells to lose their polarity and escape from the confinement of planar epithelial tissue organization (as reviewed in \textcite{Polyak2009-fi}). In colorectal adenocarcinoma, \ac{EMT} can be induced via the \gene{WNT} signalling pathway by loss-of-function mutations in \gene{APC} or somatic mutations of \gene{CTNNB1} inhibiting its degradation \parencite{Morin1997-ez}. Both alterations are believed to be a key step in the development of adenomas, the neoplastic precursor to invasive adenocarcinoma. A further step of invasion is the breakdown and remodelling of the \ac{ECM}, thought to also be facilitated by the microenvironment (see the section ‘Mapping the cancer ecosystem’).

The consequence of this breakdown is likely the acceleration of malignant evolution as the fixation probability and the rate of fixation are higher in unstructured populations with a greater degree of mixing. Several theoretical investigations have demonstrated that spatial constraints can control the mode of tumour evolution \parencite{West2021-ar,Noble2022-eg}. Another consequence is that different selective pressures apply, which may be one cause of the characteristic sequences of cancer progression such as the classic \gene{APC}-\gene{KRAS}-\gene{TP53} model for colorectal carcinoma \parencite{Fearon1990-el,Gerstung2020-sg} and the four stages of \gene{TP53} inactivation in the progression of pancreatic ductal adenocarcinoma from a pre-malignant to malignant state \parencite{Baslan2022-rb}. In this light, a pan-cancer analysis revealed that the portfolio of late and subclonal driver gene mutations tends to be more diverse than the set of early alterations \parencite{Gerstung2020-sg}.

