\chapter{Data Integration: Biological Insights from  Multiple Data Modalities}
\label{sec:chapter-basiss-multimodal}

\section*{Declaration}

This chapter is based on the supplementary methods and technical results from:
\\~\\
 \fullauthcite{Lomakin2022-ks}. 
\\~\\
The work I present here is primarily my own contribution. I developed and implemented the core mathematical and algorithmic toolkit to interpret \acs{BaSISS}, \acs{ISS}, \acs{IHC} and histopathologic data under the supervision of \ac{moritz} and with inputs from \ac{artem1} and \ac{vitalii}. 

I analysed and interpreted the data, and drafted the original article and figures, again under the supervision of \ac{moritz} and \ac{lucy}. You can find all the code on \href{https://github.com/gerstung-lab/BaSISS}{Github}. The code for hierarchical logistic regression I wrote together with \ac{vitalii}. It is available at \href{https://github.com/dissatisfaction-ai/scHierarchy}{scHierarchy} repo.

Another first author of this paper, \ac{jessica}, in collaboration with \ac{peter}, \ac{mats}, and \ac{lucy}, designed the initial study of \acs{BaSISS} and \acs{ISS}. \ac{jessica}, \ac{mats}, and \ac{carina} conducted \acs{BaSISS}, \acs{ISS} and \acs{IHC} experiments and provided the raw data. \ac{junsung}, \ac{vasyl}, \ac{tong}, and \ac{milana} preprocessed and decoded \acs{ISS} and \acs{BaSISS} data. I had access to the decoded \acs{ISS} and \acs{BaSISS} data. \ac{hege} provided histopathological description of the microregions.

The Introductory \cref{sec:modalities-multimodal-intro}, Data Characterisation \cref{sec:modalities-iss}, Hierarchical Logistic Regression \cref{sec:modalities-schierarchy}, Multimodal Visualisation \cref{sec:modalities-cancerclonemaps} and Summary \cref{sec:modalities-summary} are original. I borrowed the rest of the sections from the original manuscript and reworked them to fit the thesis format, rewording, expanding and illustrating them for clarity. All the main and margin figures are original except \cref{fig:multimodal-iss} and \cref{fig:multimodal-markers}, which I reproduced from the original paper with stylistic and compositional adjustements. 

\section{Multimodal Analsis}

\label{sec:modalities-multimodal-intro}

Cancer progression involves complex interplay between the evolution of genome, cell state plasticity, spatial architecture, and interaction with the microenvironment \pcref{sec:chapter-introduction,sec:applications-breast-intro}; the stufy of tumorigenesis therefore benefits from integrating multiple layers of genetic and non-genetic information at the single-cell level. Unimodal analyses, which focus solely on one aspect such as genetics or transcription, offer an incomplete view. To capture the various axes of variation, integrative multi-omics approaches become essential \parencite{Nam2021-xt,Baysoy2023-qr}.

Although ideal measurements would capture all modalities in the same cell simultaneously, current technologies fall short. Existing multi-omics technologies face limitations in throughput and the number of modalities they can measure concurrently \parencite{Baysoy2023-qr}. Consequently, researchers often acquire unmatched single-cell information modalities across several biological samples. They then infer a joint representation in latent space, using techniques conceptually akin to canonical correlation analysis \parencite{Stuart2019-mi} or non-linear matrix factorisation \parencite{Argelaguet2018-oz,Velten2022-gc}. However, this strategy suffers from inevitable information loss when aligning distinct feature spaces.
\explain{}{\marginfig{side-plot-multimodal-morency.pdf} Two common ways of multimodal data representation that reflect cross-modal interactions. From \textcite{Liang2022-gp}}

Current multi-omics methods typically measure two modalities, one of which is usually the transcriptome. Examples include CITE-seq (protein and transcription, \textcite{Stoeckius2017-rz}), scATAC-seq (DNA-accessibility and transcriptome, \textcite{Lareau2019-xr}), and G\&T-seq (genome and transcriptome, \textcite{Macaulay2015-gh}). While these pairs are natural parts of the correspoding assays, they often exhibit low multiplexity and biases; for instance, CITE-seq generally measures only around 300 surface proteins. Nonetheless, one can use common dimensions as pivots for data integration and imputation, thereby better preserving information than when integrating over distinct feature spaces \parencite{Hao2021-qn}.

The methods outlined above offer a multimodal depiction of cells. However, because these assays are dissociative, they compromise the spatial structure. In contrast, spatial omics methods uniquely maintain the tissue context of cellular measurements \pcref{box:technologies}. Yet, they encounter similar challenges to those faced by single-cell multi-omics methods. Specifically, these methods share issues of sparsity and often necessitate integration with single-cell reference data obtained from dissociative studies \pcref{box:analysis}. 

For spatial omics methods, integration of multiple spatially-resolved modalities is feasible across consecutive tissue sections \pcref{box:analysis}. Unlike in single-cell omics, where integration occurs for the same cell, spatial omics focuses on local neighbourhoods. This approach relies on the assumption that the cellular composition and properties remain relatively constant between adjacent slides. However, since most tissue are inhomogenious, this assumption holds true only if the tissue sections are closely situated, typically within only a few cell layers. Algorithms for image registration enable the alignment of moderatly distorted sections and the 3D reconstruction of a tissue block \parencite{Kiemen2020-dc} unifying information obtained in different experiments. 

Spatial omics tools offer valuable context for cellular functions, exceeding the phenotypic descriptions available through traditional genomics. These tools elucidate the spatial organisation of cells within tissues, and its impact on cellular behaviour \pcref{sec:mapping-the-cancer-ecosystem}. Analysis tools range from simple statistical tests that examine differences in cellular composition to more complex unsupervised clustering algorithms based on topic models \parencite{Nirmal2022-sq} and spatial graphs \parencite{Danenberg2022-zb}. For information-rich data, more explicit models of cell interaction are possible \parencite{Fischer2023-go}.

In summary, a nuanced understanding of cancer necessitates the integration of multiple data modalities, including spatial information. Although multi-purpose tools like Seurat, MOFA \parencite{Argelaguet2018-oz}, and MEFISTO \parencite{Velten2022-gc} offer valuable platforms for data exploration, they may lack the specificity required to answer targeted questions in cancer biology, particularly when data quality is suboptimal. In this chapter, I will outline the spatial data collected alongside \ac{BaSISS}, describe the integration methods employed to define and locate cell types, and present customised statistical tests designed to answer the specific question: \textbf{``What are the phenotypic and microenvironmental differences between cancer clones?"}

\section{Results}

\label{sec:modalities-multimodal-results}

Knowing the spatial distribution of clonal densities with the BaSISS model described in \cref{sec:chapter-basiss-model}, one may characterise the clones phenotypically based on additional spatially matched data. These include:

\begin{itemize}
    \item Histopathological phenotype
    \item \ac{IHC} staining
    \item \ac{ISS} expression signals assigned to nuclei
    \item Cell type annotations
\end{itemize}

\explain{}{\marginfig{side-plot-tissue-block.pdf} The tissue block undergoes sectioning to yield material for the designated experiment. Subsequently, the various data modalities are spatially aligned. The thickness of each slide measures 10 $\mu$m.}

The modalities of spatially resolved information exist across multiple consecutive tissue slides. These slides exhibit spatial deformations from one layer to another, complicating the registration process. To integrate information across all modalities, a trained pathologist, \acf{hege}, manually annotated regions of breast tissue. These annotations targeted areas with similar histopathological phenotypes, which should be in close physical proximity on the z-stack.

\subsection{Histopathological phenotype}
\label{sec:modalities-histopath}

In our study, histology served dual purposes: it helped define micro-anatomical and cellular features, and designate microregions for feature projection across consecutive slides and assays \pcref{sec:intro-histology}. \acl{hege} and \acf{lucy} annotated microregions on \acf{H\&E} stained fresh-frozen samples, uninfluenced by prior knowledge of the sample's genetic clonal structure.

For \acf{DCIS} cases, annotation focused on regions presumed to be myoepithelially enclosed, as indicated by \ac{H\&E} or other histological images (refer to \cref{fig:applications-breast-structure} for details). Due to the limitations imposed by fresh freezing, certain features like mitotic activity and chromatin structure were difficult to assess. Consequently, nuclear grade (see \cref{sec:applications-breast-intro}) in \ac{DCIS} relied mainly on nuclear size. \acs{hege} assessed the presence or absence of nuclear vacuoles using a 20\% cut-off criterion. Challenges in identifying \ac{TDLU} and ducts were resolved by considering the presence of branches in morphology \pcref{fig:applications-breast-structure}.

For invasive tumour sections lacking clear histological structures, \acs{hege} and \acs{lucy} manually demarcated similarly sized regions of invasive tissue. This approach enabled uniform sampling across the entire tissue surface area.

In specific samples, P2-TN1 and P2-LN1, \acs{hege} reported multiple independent growth patterns. For P2-LN1, areas were selected based on these growth patterns, with the aim of creating regions of similar size \pcref{fig:multimodal-iss}. In the case of P2-TN1, where different growth structures were less distinctly demarcated, \acs{hege} reported the proportion of cells belonging to each growth pattern within predefined regions.

\subsection{\acl{IHC} data}
\label{sec:modalities-ihc}

\acl{IHC} serves as a robust method for localising specific proteins within tissue sections. While the technique offers limited multiplexing capabilities, it remains a standard approach for obtaining spatial information on protein expression levels.

\acf{carina} incubated tissue sections with antibodies targeting \protein{PanCK}, \protein{CD45}, and \protein{HER2}. These slides underwent both \ac{BaSISS} and \ac{ISS} protocols. In subsequent slides, \acs{carina} targeted additional proteins, such as \protein{SM-MHC}, \protein{P63}, \protein{PR}, \protein{Ki-67}, and \protein{PTEN}. For further context on the role of these proteins in breast cancer, refer to \cref{sec:applications-breast-intro}.

\ac{lucy} utilised Qupath digital software for the quantitative evaluation of stained nuclei. The analysis focused on assessing the expression of \protein{Ki-67}, \protein{PTEN}, and \protein{PR} in selected regions \pcref{appfig:appendix-applications-PBC-P1}.

\subsection{\acl{ISS} data}
\label{sec:modalities-iss}

\figurefloat{multimodal-iss.pdf}{multimodal-iss}{\acl{ISS} data for the lymph node breast cancer sample}{\textit{\textbf{top}}, Display of \ac{BaSISS} fields alongside two selected \ac{ISS} panels, focusing on selected genes (\gene{HER2}, \gene{CD24} - onco panel and \gene{MS4A1}, \gene{CD68}, \gene{CD3D} - immune panel). White outlines mark microregions, identified by a histopathologist, based on histological growth patterns. \textit{\textbf{bottom}}, Scatterplots show the log$_2$-fold change in gene expression between samples, as measured by RNAseq and combined \ac{ISS} oncology and immune experiments. High correlation values (r = Pearson's correlation) confirm on-target probe performance. The analysis includes genes that meet the following criteria: a Transcripts Per Million (TPM) count greater than 25 in RNAseq and over 1000 detections per million cells in \ac{ISS}, minimising deviations due to low counts.}

The study design employs In Situ Sequencing (\ac{ISS}) technology to characterise the transcriptional phenotype of tissue samples. The \ac{ISS} dataset included two panels: a previously published 91-gene panel targeting oncology-related markers and a novel 62-gene immune panel designed for immune cell profiling \pcref{fig:multimodal-iss}. I assessed the immune panel's performance by correlating its expression levels with paired bulk RNA-seq data in the \acf{TNBC} case outlined in \cref{sec:applications-cohort-introduction}. The choice of this specific case stemmed from its elevated levels of immune cells, the panel's designed target. Once normalised for technological differences, data exhibited a significant correlation (r = 0.69–0.78, Pearson's) as shown in \pcref{fig:multimodal-iss}.

Transcript detection, particularly differential expression near clones, often provides functional insights if a gene has a clear association with a specific cell type. For instance, the presence of the \gene{MS4A1} gene strongly indicates the existence of B-cells. However, many genes lack a clear association with a cell type, complicating interpretation. The detection of \gene{ACTA2}, for example, could signify the presence of either smooth muscle cells, myofibroblasts, or basal epithelial cells. Additionally, functional interpretation is confounded by the cell signal's origin; epithelial-mesenchymal transition (EMT) markers in epithelial cells signify transition, whereas the same markers in stromal cells are standard. Therefore, accurate interpretation of \ac{ISS} data requires knowledge of the originating cell type.

While the \ac{ISS} panel aimed to identify specific cell types and was partially based on the OncotypeDX panel, the gene selection process did not consult single-cell atlases. Instead, the chosen genes were primarily informed by the existing literature, which often focuses on protein characteristics unmeasurable by \ac{ISS}. Moreover, the data was generated using an older version of the \ac{ISS} technology based on sequencing by ligation, resulting in a low signal count per cell (fewer than 3 signals per cell when assigned) [link to the protocol].

Given the data's sparsity, the use of sophisticated methods designed to resolve niche cell types, such as cell2location \parencite{Kleshchevnikov2022-ub} and SSAM \parencite{Park2021-hi}, based on expression signatures turned out to give suboptimal results. Consequently, I opted for an alternative approach: employing \ac{scRNAseq} data to first identify cell markers that unequivocally define cell types, and subsequently using these markers for tissue profiling.

\subsection{Hierarchical logistic regression for cell marker discovery}
\label{sec:modalities-schierarchy}

\explain{}{\marginfig{side-plot-atlas.pdf} Cell-type organisation inherently exhibits a hierarchical structure, often reflected in single-cell atlases annotations. Data from \textcite{Wu2021-uq}}

The objective is to identify easily usable cell type markers for classifying individual cells within a tissue. To make decoding simple, these markers should be unique to a specific cell type or, alternatively, a unique combination of markers should denote a specific cell type. Identifying unique markers is challenging because most genes are involved in cellular programmes that multiple cell types share, particularly when the classification is granular. Current methods for discovering cell type markers rely on statistical tests \parencite{Wolf2018-kx}, heuristic approaches for dimensionality reduction \parencite{Dumitrascu2021-dg, Dai2022-ve, Missarova2021-vg}, or machine learning classification models \parencite{Nelson2022-vg} coupled with feature importance analysis. These methods generally fail to explicitly incorporate the concept of combinatorial encoding of cell types.

One natural approach to improving cell type identification exploits the hierarchical organisation of cell types. In this model, we assume that higher level group cell markers, such as those for all epithelial or immune cells, are obligatorily expressed across their respective subtypes. Nonetheless, we permit subtypes to display non-unique markers that might be shared among multiple groups as soon as this group belongs to a different overarching type. The key requirement is that this higher level cell type remains identifiable.

Consider, for example, the immune-cycling and epithelial-cycling cell states. Both would express markers indicative of cell cycling. Traditional marker discovery methods often underuse cycling program cell markers as overly general, because both epithelial and immune cells express them. However, when these cycling markers are combined with broad markers specific to all immune cells, we can unambiguously categorise a cell as immune-cycling.

To formalise this strategy, one could employ a marker selection model like logistic regression restricting the model to groups of cells that belong to the same broad cell type. Then, iterate this approach across multiple layers of hierarchy. However, this isolated approach hinders information sharing across hierarchical levels and could yield suboptimal results. To address this limitation, I have restructured the standard logistic regression model to operate within the cell hierarchy framework, aiming to identify cell type markers at each hierarchical level \pcref{fig:multimodal-tree-math}.

\figuretextwidth{multimodal-tree-math.pdf}{multimodal-tree-math}{Hierarchical logistic regression conceptual description}{The hierarchical logistic regression model utilises a tree that  represents the hierarchical relationships among different cell types. Each node in this tree corresponds to a specific cell type, and edges signify subclass relationships. Mathematically, the likelihood conditions the probability of a cell belonging to a specific type based on its hierarchical ancestors, leveraging an array of weight vectors corresponding to each hierarchical layer. This structure introduces conditional dependencies between cell types, allowing contextual cell-type marker predictions}

\subsubsection*{Multi-class logistic regression}

Consider a Bayesian formulation of the logistic regression model for predicting cell types. Let $y_i$ represent the cell type label for cell $i$, which can assume one of $K$ possible classes, $\{1, 2, \ldots, K\}$. Define $\mathbf{x}_i$ as the $g$-dimensional vector of gene expression levels for cell $i$. The likelihood function for this model is given by:

\begin{equation}
    \label{eq:logistic-regression-likelihood}
    p(y_i = k | \mathbf{x}_i, \mathbf{w}) = \frac{\exp(\mathbf{w}_{k}^T \mathbf{x}_i)}{\sum_{k=1}^{K} \exp(\mathbf{w}_{k}^T \mathbf{x}_i)}
\end{equation}

Here, $\mathbf{w}_k$ is the weight vector corresponding to class $k$. To promote sparsity, we employ a zero-centred Laplace prior for the weights, characterised by scale parameter $\lambda$:

\begin{equation}
    \label{eq:logistic-regression-prior}
    p(w_{kg}) \sim \text{Laplace}(0, \lambda),\quad\text{i.i.d. for all } g 
\end{equation}

We fit this model to an annotated \ac{scRNAseq} dataset using the \ac{VI} method, as detailed in \cref{sec:bayesian-intro}. This provides an approximation of the posterior distribution $p(\mathbf{w}_k | \mathbf{X}, \mathbf{y})$. The magnitude of each $w_{k,g}$ thereby affords a straightforward interpretation of gene $g$'s importance in classifying cell type $k$.

\subsubsection*{Graph structure of annotated single-cell data}

The graphical structure of single-cell data annotation hierarchies can be mathematically represented as a tree $T = (V, E)$, where $V$ is the set of vertices and $E$ is the set of edges. Each vertex $v \in V$ represents a particular cell type annotation, and each edge $(u, v) \in E$ represents a subclass relationship between cell types $u$ and $v$. 

Let $r$ be the root node, representing the most general cell type encompassing all cells. Nodes directly connected to $r$ represent broad cell types; for example these could be ``Epithelial cells'', ``Immune cells'' or ``Stromal cells''. The set of all nodes connected to the root is denoted as $L_1$ and represents the first layer of the hierarchy. 

Mathematically, a hierarchical layer $L_i$ consists of nodes that are equidistant from the root node $r$. That is, for any node $v$ in $L_i$, the shortest path from $r$ to $v$ has length $i$:

\begin{equation}
    L_i = \{v \in V : d(r, v) = i\}
\end{equation}

where $d(r,v)$ denotes the shortest path from $r$ to $v$ in tree $T$. 

For convenience, let's also define a function $f_{\text{p}}$ \pcref{fig:multimodal-tree-math} that retreives the parent $u$ of the child node $v$:

\begin{equation}
    f_{\text{p}}(v) = u, \text{ where } (u, v) \in E
\end{equation}

The $f_{\text{s}}$ function \pcref{fig:multimodal-tree-math} returns the set of siblings of node $v$ (including $v$ itself), that lie in the same layer as $v$ node, and belong to the same parent node $u$:

\begin{equation}
    f_{\text{s}}(v) = \{w \in V : f_{\text{p}}(w) = f_{\text{p}}(v)\}
\end{equation}

Taking an element in $L_3$, and iteratively applying $f_{\text{p}}$ until we reach $L_1$  would return it's hierachy ``CD8+ T cells'' $\rightarrow$ ``T cells'' $\rightarrow$ ``Immune cells''. Similarly, applying $f_{\text{s}}$ to this element would return all other cell types in that belong to the same parent node ``T cells'', such as ``CD4+ T cells'' and ``NK cells''.

\subsubsection*{Hierarchical extension of logistic regression}

Now that we have defined the hierarchical structure of the cell type annotation, we can extend the logistic regression model to incorporate this information \pcref{fig:multimodal-tree-math}. Similar to standard logistic regression, $\mathbf{x}_i$ as the $g$-dimensional vector of gene expression levels for cell $i$. Let $\mathbf{y}_i = [y_{i,1}, y_{i,2}, \ldots, y_{i,H}]$ be the set of labels for cell $i$, corresponding to each of the $H$ hierarchical layers. Similarly, the weight vector for layer $L_h$ is denoted as $\mathbf{w}_{L_h}$, where $h \in {1, 2, \ldots, H}$.

The probability of cell $i$ belonging to class $k$ at layer $L_h$ is now dependent on the hierarchical parents of this cell:

\begin{equation}
    \begin{aligned}
    p(y_{i,h} = k) &= p(y_{i,h} = k | y_{i,h-1} = f_{\text{p}}(k)) \\
    &\phantom{=} \times p(y_{i,h-1} = k | y_{i,h-2} = f_{\text{p}}(f_{\text{p}}(k))) \\
    &\phantom{=} \times \cdots \\
    &\phantom{=} \times p(y_{i,h_1} = f_{\text{p}}^{H-1}(k))
    \end{aligned}
\end{equation}

Notice that the conditional term like $p(y_{i,h} = k | y_{i,h-1} = f_{\text{p}}(k))$ is equivalent to the standard logistic regression model, restricted to the $f_s(k)$, i.e. the siblings of label $k$. 

Expanding the above equation using \cref{eq:logistic-regression-likelihood} yields the likelihood function for the hierarchical logistic regression model:

\begin{equation}
    \begin{aligned}
    p(y_{i,h} = k | \mathbf{x}_{i} \mathbf{w}) &= 
         \frac{\exp(\mathbf{w}_{L_h, k}^T \mathbf{x}_i)}{\sum_{j \in f_s(k)} \exp(\mathbf{w}_{L_h, j}^T \mathbf{x}_i)} \\
        &\phantom{=} \times \frac{\exp(\mathbf{w}_{L_{h-1}, f_p(k)}^T \mathbf{x}_i)}{\sum_{j \in  f_s(f_p(k))} \exp(\mathbf{w}_{L_{h-1}, j}^T \mathbf{x}_i)} \\
        &\phantom{=} \times \cdots \\
        &\phantom{=} \times \frac{\exp(\mathbf{w}_{L_1, f_p^{H-1}(k)}^T \mathbf{x}_i)}{\sum_{j \in L_1} \exp(\mathbf{w}_{L_1, j}^T \mathbf{x}_i)}
    \end{aligned}
\end{equation}

The prior distribution for the weights $\mathbf{w}$ is the same Laplace distribution as in \cref{eq:logistic-regression-prior}. To address the issue of class imbalance, we now adjust the weights using the square root of the number of cells in each group. We employ square root scaling to moderate the penalty on larger groups, thereby maintaining a balanced emphasis across different classes.

This model is fit using \ac{ADVI}, and tha posterior distributions of the weights are used to asses the improtance of a gene to define a given class. 

\figuretextwidth{multimodal-markers.pdf}{multimodal-markers}{Hierarchical cell-type markers for \ac{ISS} panels}{The result of running hierarchical logistic regression to select marker genes for cell typing, employing Breast Cancer \ac{scRNAseq} Atlas \parencite{Wu2021-uq} as input dataset. The analysis was limited to genes from \ac{ISS} oncology and immune panels. The figure displays genes with the highest weights for each hierarhical layer. PVL = perivascular-like cells. CAF = cancer associated fibroblasts}

\subsubsection*{Infered set of markers for \ac{ISS} panels}

I run the aforementioned model on the breast cancer atlas derived from \ac{scRNAseq} data ($\sim$ 30000 genes), as described by \textcite{Wu2021-uq}. The analysis was limited to the subset of genes present in two \ac{ISS} panels (91 + 62). After applying a filter to retain only the top 70\textsuperscript{th} percentile of all gene weights, and manually excluding genes expressed in multiple cell types within each group, the resulting marker set exhibited limited resolution \pcref{fig:multimodal-markers}. Specifically, the markers were able to differentiate only two out of the four hierarchical levels defined in the atlas. At the first level, adequate markers allowed for differentiation among broad cell types: ``Immune", ``Epithelial'', and ``Stromal''. At the second level, the ``Immune" cells resolved further into ``B-cells", ``Myeloid", and ``T-cells". The ``Stromal" category subdivided into a mixed population of ``Fibroblasts + PVL" and ``Endothelial" cells. The analysis did not provide further resolution due to a poor choice of markers during panel design.

\subsection{Cell type assignmment with sparse data}
\label{sec:modalities-celltype}

I overlaid the \ac{ISS} data with nuclei segmentation masks and allocated signals to the closest nuclei. I applied a conservative distance cut-off of 5$\mu$m — approximately double the nuclear radii — to minimise the likelihood of misannotation, resulting in approximately 30\% signal loss.

Cell type classification occurred in two steps in a simple `if-else' algorithm. In the first iteration, nuclei possessing any markers corresponding to the broad categories (``Epithelial", ``Immune", ``Stromal") were assigned accordingly. In the subsequent iteration, nuclei initially classified as ``immune" or ``stromal" underwent further categorisation based on the presence of specific markers shown on \cref{fig:multimodal-markers}. Nuclei lacking proximal markers or exhibiting conflicting assignments received an ``unknown" classification. The annotated maps for the lymph node case appear in \cref{appfig:appendix-applications-LN}.

\figuretextwidth{multimodal-glmm.pdf}{multimodal-glmm}{Modelling clone-specificepxression and composition using \acs{GLMM}}{On of the objectives of this study is to characterise the phenotypic, transcriptional, and compositional differences among cancer clones. I record cell-type composition and \ac{ISS} signal counts, attributing them to cells with known cell types. I then model these data using a \acf{GLMM} that incorporates a clone-specific fixed effect, denoted as $\beta$, and a clone-specific regional random effect, denoted as $\alpha$. Inferred posterior distriubtion of $\beta$ is then used to compute statistics and reason about differences between clones.}

\subsection{\acs{GLMM} for multiregional quantitative analysis of clone-specific differences}
\label{sec:modalities-glmm}

Upon allocating \ac{ISS} signals to individual cells and identifying cell-types, microregions contain several types of information. These include continuous clonal contributions, counts of \ac{IHC}-stained nuclei, \ac{ISS} signals categorised by originating cell type, counts of each cell type, and categorical histological phenotype features \pcref{sec:modalities-multimodal-results}.

The primary objective is to elucidate the phenotypic, transcriptional, and compositional differences between clones. It appeared that breast cancer clones generally do not intermingle, as discussed in \cref{sec:bassis-validation,sec:applications-DICS-growth} and shown on \cref{fig:applications-maps-PBC-P1,fig:applications-maps-DCIS} we restricted our analysis to highly clonal regions based on specific \ac{CCF} thresholds: \ac{CCF}\textsubscript{clone} > 0.7 for P1, and \ac{CCF}\textsubscript{clone} > 0.15 and 0.05 for P2-TN1 and P2-LN1, respectively. The lower thresholds for P2-TN1 and P2-LN1 account for the high levels of non-epithelial cells present in these samples.

Standard statistical tests like Fisher's exact test easily handle categorical data, such as histological annotations. However, these methods are less applicable for count data, which are subject to variations in cell numbers and likely overdispersion (e.g. region-specific expression variation). To address these challenges, I employed custom \acfp{GLMM} designed to account for variable cell numbers and data overdispersion \pcref{fig:multimodal-glmm}.

\subsubsection*{Intrinsic expression in cell types}

To quantify differential expression associated with clonal regions for a specific cell type, we record \ac{ISS} signals corresponding to nuclei classified under that cell type. I model the distribution of observed \ac{ISS} expression signals $Y_{rg}$ for genes $g$ in regions $r$ using a clone-specific expression rate $\beta_{cg}$. This rate is sampled uniformly, encompassing all plausible values (after exponentiation):

\begin{equation}
    \beta_{cg} \sim \text{Uniform}(-10,10)
\end{equation}

Acknowledging the possibility of region-specific variation in expression, let's introduce a random effect variable $\alpha_{rg}$. This variable has a clone-specific hierarchical prior $\sigma_{cg}$, which governs the prior belief regarding the extent of inter-regional variability. A value of 0.05 for this hyperprior anticipates low variability, yet permits deviations if the data suggest otherwise. The binary matrix $A_{rc}$ maps dominant clones to their corresponding regions.

\begin{align}
    \sigma_{cg} &\sim \text{HalfNormal}(0.05) \\
    \alpha_{rg} &\sim \text{Normal}(0, A_{rc}\sigma_{cg})
\end{align}

After adjusting for the number of nuclei of the relevant cell type 
$x_r$, the likelihood becomes:

\begin{equation}
    Y_{rg} \sim \text{Poisson}\left(x_r e^{A_{rc}\beta_{cg} + \alpha_{rg}}\right)
\end{equation}
    
This formulation accommodates both clone- and region-specific variations in gene expression, providing an interpretable statistical model for analysing cell-type specific differential expression \pcref{fig:multimodal-glmm}.

\subsubsection*{Cell-type composition}

To assess differential cellular composition between regions associated with clones, I computed the counts of nuclei categorised as specific cell types. I then modelled the distribution of nuclei counts $Y_{rt}$ attributed to cell type $t$ across regions $r$ as a mixed model. This model incorporates both a clone-specific frequency $\beta_{ct}$ and a region-specific random effect $\alpha_{rt}$, akin to the approach used for \textbf{cell-type specific expression}:

\begin{align}
\beta_{ct} &\sim \text{Uniform}(-10, 10) \\
\sigma_{ct} &\sim \text{HalfNormal}(0.05) \\
\alpha_{rt} &\sim \text{Normal}(0, A\sigma_{ct})
\end{align}

Given the sum-to-one constraint on cell type frequencies, I employed a softmax (logit) link function along with a Multinomial likelihood for the cell types:

\begin{equation}
\lambda_{rt} = A\beta_{ct} + \alpha_{rt}
\end{equation}
\begin{equation}
Y_{rt} \sim \text{Multinomial}\left(n=x_r, p=\frac{e^{\lambda_{rt}}}{1 + \sum_{i=1}^{T-1} e^{\lambda_{rt}}}\right)
\end{equation}

By incorporating these constrains, I ensure that the model accounts for the inherent constraints and random effects in differential cellular composition across regions \pcref{fig:multimodal-glmm}.

\subsubsection*{IHC staining}
\ac{lucy} counted the number of IHC-positive and IHC-negative nuclei in each region using QuPath. To enable the application of the aforementioned \textbf{cell-type composition} model, I treated IHC\textsuperscript{+} and IHC\textsuperscript{-} nuclei as two distinct cell types.

\subsubsection*{Inference and statistical testing}

I implemented the models using the \ac{NumPyro} library and employed \acf{HMC} for parameter inference \pcref{sec:bayesian-intro}. To identify quantiatively differences, I performed pairwise comparisons of clone-specific expression rates $\beta_{cg}$, for each pairs of clones. Examples of posteriors for expression rates appear in \cref{appfig:appendix-applications-PBC-P1}

\ac{HMC} generates empirical posterior distributions with a finite number of samples, limiting our ability to resolve small differences. Specifically, the quantiles are restricted to values of $1/n$, where $n$ is the sample size from the \ac{HMC} run. This limitation becomes particularly challenging when conducting multiple comparisons, as it necessitates the multiple testing correction. 

To enable the calculations of more extreme quantiles, I approximated the posterior distribution of differences in expression rates $\Delta_g = \beta_{1g} - \beta_{2g}$, using the square of the mean divided by the standard deviation, or $\left(\frac{\mu(\Delta)}{\sigma(\Delta)}\right)^2$. Assuming that $\Delta$ is noramlly distributed, this approximation follows a $\chi_{1}^{2}$. Treating the posterior as analogous to the frequentist test statistics, I compute p-value and apply multiple testing corrections to each comparison to maintain a \acf{FDR} below 0.1.

\explain{}{make a plot to exaplain}

P-values computed according to this approach appear on the vulcano plots on \cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-PBC-P2,appfig:appendix-applications-DCIS-expression}. For differential composition and protein expression, I used a similar approach with the softmax-transformed $\beta_{ct}$ values. Examples of $\beta_{ct}$ posteriors and differential composition statistics appear in \cref{fig:applications-maps-LN} and \cref{appfig:appendix-applications-PBC-P1,appfig:appendix-applications-PBC-P2}. 

To maximise statistical power in clone-specific differential expression analysis, we excluded genes with a clone-agnostic average number of detected signals per region less than 0.3.

\subsection{Approach limitations}


\figuretextwidth{multimodal-cancerclonemaps.pdf}{multimodal-cancerclonemaps}{Multimodal data visualisation web-tool at \href{https://www.cancerclonemaps.org/}{cancerclonemaps.org}}{The web tool visualises multimodal data, specifically designed to meet the requirements of the \ac{BaSISS} experiment. This web tool interactively displays microregion borders, histological features, \ac{ISS} and \ac{BaSISS} data, inferred clonal maps, and \ac{ISS}-based cell type locations for all samples analysed in the study from \textcite{Lomakin2022-ks}}

\subsection{Multimodal data visualisation}
\label{sec:modalities-cancerclonemaps}

The challenge of visualising spatial multimodal data, particularly in the project that combines \ac{BaSISS}, \ac{ISS}, and histological data, requires a tailored approach. Current methods such as \ac{TissUUmaps}, \ac{Omero}, \ac{WebAtlas}, and \ac{Napari} fall short in meeting specific visualisation requirements, especially for clone fields inferred from \ac{BaSISS} data.

To address this challenge, we created a specialised web-based tool, available at \href{https://www.cancerclonemaps.org/}{cancerclonemaps.org}. Our tool utilises a Flask backend and incorporates frontend technologies like `D3' for visual elements, `pixi.js' for efficient spot visualisation, `Leaflet' for tissue mapping, `Geoman' for map selection tools, and `Plotly.js' for the diagrams. This stack of technologies enables interactive exploration of our complex data sets, offering a unified platform to investigate histogenomic relationships. Consequently, this tool streamlines exploratory analysis and assists in identifying qualitative patterns across different data types.

\section{Summary}
\label{sec:modalities-summary}

In \cref{sec:chapter-basiss-model}, I detail experimental methods and statistical algorithms for identifying genomically-defined cancer subclones. \cref{sec:chapter-basiss-multimodal} extends this discussion by explaining how to integrate genetic information with spatial data on transcription, proteomics, and histology. By combining these approaches, the assembled toolkit aims to shed light on both the intrinsic cellular properties and the tumour microenvironment. Specifically, it seeks to identify differences between cancer clones in terms of their intrinsic features and their respective habitats.

While some assays discussed here lack multiplexing capability and may appear outdated in the fast-evolving field of spatial genomics, they can still provide valuable insights. When interpreted carefully, the generated data can reveal intricate details about the spatial evolution and ecology of tumours. The following section \pcref{sec:chapter-basiss-applications} demonstrates the utility of these methods through a case study on breast cancer.